<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="keywords" content="Hexo Theme Keep">
    <meta name="description" content="Welcome to my personal blogs">
    <meta name="author" content="Yuxuan Wu">
    
    <title>
        
            BIO312 |
        
        Yuxuan Wu
    </title>
    <link rel="shortcut icon" href="/images/logo.svg">
    
<link rel="stylesheet" href="/css/style.css">

    
<link rel="stylesheet" href="/css/font-awesome.min.css">

    <script id="hexo-configurations">
    let KEEP = window.KEEP || {};
    KEEP.hexo_config = {"hostname":"example.com","root":"/","language":"en","path":"search.xml"};
    KEEP.theme_config = {"toc":{"enable":true,"number":true,"expand_all":false},"style":{"primary_color":"#0066CC","avatar":"/images/avatar.svg","favicon":"/images/logo.svg","article_img_align":"left","left_side_width":"260px","content_max_width":"920px","hover":{"shadow":true,"scale":true},"first_screen":{"enable":true,"background_img":"/images/bg.svg","description":"Keep coding, Keep hungry."},"scroll":{"progress_bar":{"enable":true},"percent":{"enable":true}}},"local_search":{"enable":true,"preload":true},"code_copy":{"enable":true,"style":"default"},"pjax":{"enable":false},"lazyload":{"enable":false},"version":"3.3.1"};
    KEEP.language_ago = {"second":"%s seconds ago","minute":"%s minutes ago","hour":"%s hours ago","day":"%s days age","week":"%s weeks age","month":"%s months age","year":"%s years age"};
  </script>
<!-- hexo injector head_end start -->
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css">

<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/hexo-math@4.0.0/dist/style.css">
<!-- hexo injector head_end end --><meta name="generator" content="Hexo 5.3.0"><style>mjx-container[jax="SVG"] {
  direction: ltr;
}

mjx-container[jax="SVG"] > svg {
  overflow: visible;
}

mjx-container[jax="SVG"] > svg a {
  fill: blue;
  stroke: blue;
}

mjx-container[jax="SVG"][display="true"] {
  display: block;
  text-align: center;
  margin: 1em 0;
}

mjx-container[jax="SVG"][justify="left"] {
  text-align: left;
}

mjx-container[jax="SVG"][justify="right"] {
  text-align: right;
}

g[data-mml-node="merror"] > g {
  fill: red;
  stroke: red;
}

g[data-mml-node="merror"] > rect[data-background] {
  fill: yellow;
  stroke: none;
}

g[data-mml-node="mtable"] > line[data-line] {
  stroke-width: 70px;
  fill: none;
}

g[data-mml-node="mtable"] > rect[data-frame] {
  stroke-width: 70px;
  fill: none;
}

g[data-mml-node="mtable"] > .mjx-dashed {
  stroke-dasharray: 140;
}

g[data-mml-node="mtable"] > .mjx-dotted {
  stroke-linecap: round;
  stroke-dasharray: 0,140;
}

g[data-mml-node="mtable"] > svg {
  overflow: visible;
}

[jax="SVG"] mjx-tool {
  display: inline-block;
  position: relative;
  width: 0;
  height: 0;
}

[jax="SVG"] mjx-tool > mjx-tip {
  position: absolute;
  top: 0;
  left: 0;
}

mjx-tool > mjx-tip {
  display: inline-block;
  padding: .2em;
  border: 1px solid #888;
  font-size: 70%;
  background-color: #F8F8F8;
  color: black;
  box-shadow: 2px 2px 5px #AAAAAA;
}

g[data-mml-node="maction"][data-toggle] {
  cursor: pointer;
}

mjx-status {
  display: block;
  position: fixed;
  left: 1em;
  bottom: 1em;
  min-width: 25%;
  padding: .2em .4em;
  border: 1px solid #888;
  font-size: 90%;
  background-color: #F8F8F8;
  color: black;
}

foreignObject[data-mjx-xml] {
  font-family: initial;
  line-height: normal;
  overflow: visible;
}

.MathJax path {
  stroke-width: 3;
}

mjx-container[display="true"] {
  overflow: auto hidden;
}

mjx-container[display="true"] + br {
  display: none;
}
</style></head>


<body>
<div class="progress-bar-container">
    
        <span class="scroll-progress-bar"></span>
    

    
</div>


<main class="page-container">

    

    <div class="page-main-content">

        <div class="page-main-content-top">
            <header class="header-wrapper">

    <div class="header-content">
        <div class="left">
            <a class="logo-title" href="/">
                Yuxuan Wu
            </a>
        </div>

        <div class="right">
            <div class="pc">
                <ul class="menu-list">
                    
                        <li class="menu-item">
                            <a class=""
                               href="/"
                            >
                                HOME
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/archives"
                            >
                                ARCHIVES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/categories"
                            >
                                CATEGORIES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/tags"
                            >
                                TAGS
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/about"
                            >
                                ABOUT
                            </a>
                        </li>
                    
                    
                        <li class="menu-item search search-popup-trigger">
                            <i class="fas fa-search"></i>
                        </li>
                    
                </ul>
            </div>
            <div class="mobile">
                
                    <div class="icon-item search search-popup-trigger"><i class="fas fa-search"></i></div>
                
                <div class="icon-item menu-bar">
                    <div class="menu-bar-middle"></div>
                </div>
            </div>
        </div>
    </div>

    <div class="header-drawer">
        <ul class="drawer-menu-list">
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/">HOME</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/archives">ARCHIVES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/categories">CATEGORIES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/tags">TAGS</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/about">ABOUT</a>
                </li>
            
        </ul>
    </div>

    <div class="window-mask"></div>

</header>


        </div>

        <div class="page-main-content-middle">

            <div class="main-content normal-code-theme">

                
                    <div class="fade-in-down-animation">
    <div class="article-content-container">

        <div class="article-title">
            <span class="title-hover-animation">BIO312</span>
        </div>

        
            <div class="article-header">
                <div class="avatar">
                    <img src="/images/avatar.svg">
                </div>
                <div class="info">
                    <div class="author">
                        <span class="name">Yuxuan Wu</span>
                        <span class="level">Lv4</span>
                    </div>
                    <div class="meta-info">
                        <div class="article-meta-info">
    <span class="article-date article-meta-item">
        <i class="fas fa-edit"></i> 2021-03-02 19:44:01
    </span>
    
        <span class="article-categories article-meta-item">
            <i class="fas fa-folder"></i>
            <ul>
                
                    <li>
                        <a href="/categories/Biology/">Biology</a>
                    </li>
                
            </ul>
        </span>
    
    
        <span class="article-tags article-meta-item">
            <i class="fas fa-tags"></i>
            <ul>
                
                    <li>
                        <a href="/tags/notes/">notes</a>
                    </li>
                
            </ul>
        </span>
    

    
    
        <span class="article-wordcount article-meta-item">
            <i class="fas fa-file-word"></i> <span>2.1k Words</span>
        </span>
    
    
        <span class="article-min2read article-meta-item">
            <i class="fas fa-clock"></i> <span>13 Mins</span>
        </span>
    
    
        <span class="article-pv article-meta-item">
            <i class="fas fa-eye"></i> <span id="busuanzi_value_page_pv"></span>
        </span>
    
</div>

                    </div>
                </div>
            </div>
        

        <div class="article-content markdown-body">
            <p>Courseworks requirements</p>
<p>Protocol</p>
<p>columns</p>
<p>Expected output</p>
<p>Which columns</p>
<p>Justification for choosing that column </p>
<p>Expected output? Diagrams</p>
<h3 id="Protein-seperation"><a href="#Protein-seperation" class="headerlink" title="Protein seperation"></a>Protein seperation</h3><ol>
<li><strong>SDS-PAGE:</strong> basis-molecular weight</li>
<li><strong>Isoelectric focusing:</strong> electric charge difference</li>
<li><strong>Chromatic methods-useful adjuncts to gel-based approaches</strong><ol>
<li>HPLC - high performance liquid chromatography </li>
<li>Thin-layer chromatography </li>
</ol>
</li>
<li><strong>2D gel electrophoresis</strong><ul>
<li>Powerful gel-based method commonly used to analyze complex samples</li>
<li>To characterize full range of proteins in the sample, not just a few specific proteins</li>
</ul>
</li>
</ol>
<h3 id="SDS-PAGE"><a href="#SDS-PAGE" class="headerlink" title="SDS-PAGE:"></a>SDS-PAGE:</h3><ul>
<li>Proteins are usually mixed with the detergent, SDS and a tracking dye to make the sample</li>
<li>SDS binds to the protein and gives it size-dependent negative charge and gives it size-dependent negative charge and consistent hydrodynamic properties</li>
</ul>
<h3 id="Isoelectric-point-pi"><a href="#Isoelectric-point-pi" class="headerlink" title="Isoelectric point (pi)"></a>Isoelectric point (pi)</h3><ul>
<li>ph at which molecule has net zero charge （determined using computer program for known sequence or empirically ）</li>
</ul>
<h3 id="Isoelectric-Focusing"><a href="#Isoelectric-Focusing" class="headerlink" title="Isoelectric Focusing"></a>Isoelectric Focusing</h3><p>Electrophoresis through polyacrylamide gel in which there is a pH gradient</p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210302212707.png" alt="image-20210302212707320"></p>
<h3 id="2D-gel"><a href="#2D-gel" class="headerlink" title="2D gel"></a>2D gel</h3><p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210302212808.png" alt="image-20210302212808199"></p>
<h3 id="Western-Blot"><a href="#Western-Blot" class="headerlink" title="Western Blot"></a>Western Blot</h3><p>a.k.a: immunoblotting</p>
<ul>
<li>Proteins first electrophoreses by SDS-PAGE</li>
<li>Wet gel is placed against nitrocellulose</li>
<li>In electrophoretic chamber: </li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210302212907.png" alt="image-20210302212906975"></p>
<blockquote>
<p>Ammonium sulfate precipitation is one of the most commonly used methods for protein purification from a solution. In solution, proteins form hydrogen bonds with water molecules through their exposed polar and ionic groups. When high concentrations of small, highly charged ions such as ammonium sulfate are added, these groups compete with the proteins to bind to the water molecules. This removes the water molecules from the protein and decreases its solubility, resulting in precipitation. Critical factors that affect the concentration at which a particular protein will precipitate include: the number and position of polar groups, molecular weight of the protein, pH of the solution, and temperature at which the precipitation is performed. The concentration at which antibodies precipitate varies among species; most rabbit antibodies precipitate with a 40% saturated solution, whereas mouse antibodies require 45-50% saturation.</p>
</blockquote>
<blockquote>
<p><strong>Protocol</strong></p>
<ol>
<li>Allow serum or ascitic fluid to thaw, determine total volume, and centrifuge at 3000g for 30 minutes.</li>
<li>Transfer sample to a beaker containing a stir bar and place on a magnetic stirrer.</li>
<li>While the sample is stirring, slowly add saturated ammonium sulfate to bring the final concentration to 50% saturation.<ol>
<li>Volume of ammonium sulfate needed is equal to the volume of sample.</li>
<li>Adding the ammonium sulfate very slowly ensures that local concentration around the site of addition does not exceed the desired salt concentration.</li>
</ol>
</li>
<li>Once the total volume of ammonium sulfate is added, move beaker to 4°C for 6 hours or overnight.</li>
<li>Transfer to conical tube and centrifuge the precipitate at 3000g for 30 minutes.</li>
<li>Carefully remove and discard supernatant. Invert conical tube and drain well. For serum or ascites, resuspend pellet in 30%-50% of the starting volume in 1XPBS. For monoclonal antibody tissue culture supernatants, resuspend pellet in 10% of the starting volume in 1X PBS.</li>
<li>Transfer antibody solution to dialysis tubing and dialyze versus three changes of 1XPBS/0.08% Sodium Azide. Be sure to allow enough space for expansion of the antibody solution during dialysis. Normally twice the re-suspended volume is sufficient.</li>
<li>Remove antibody solution from the tubing and centrifuge to remove any remaining debris.</li>
<li>Determine the concentration and store at -80°C for long-term storage.</li>
</ol>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210302213429.png" alt="image-20210302213429785"></p>
<h3 id="Protein-identification"><a href="#Protein-identification" class="headerlink" title="Protein identification"></a>Protein identification</h3><h4 id="Edman-degradation"><a href="#Edman-degradation" class="headerlink" title="Edman degradation"></a>Edman degradation</h4><p>Sequence aa in a peptide. </p>
<p>Amino-terminal residue is <strong>labeled</strong> and cleaved from the pipetted <strong>without disrupting the peptide bonds</strong> between other amino acid residues</p>
<h4 id="Mass-Spectrometry"><a href="#Mass-Spectrometry" class="headerlink" title="Mass Spectrometry:"></a>Mass Spectrometry:</h4><p>Analytical techniques that measures the mass-to-charge ratio of charged particles, mass spectrometry is used for determining <strong>masses</strong> of particles and the <strong>elemental composition of a sample</strong> of molecules as well as for <strong>elucidating the chemical structure of molecules</strong> such as peptide</p>
<h3 id="Protein-sequencing"><a href="#Protein-sequencing" class="headerlink" title="Protein sequencing"></a>Protein sequencing</h3><h4 id="Sanger-sequence-protein-identification"><a href="#Sanger-sequence-protein-identification" class="headerlink" title="Sanger sequence (protein identification)"></a>Sanger sequence (protein identification)</h4><h3 id="Cleavage-of-polypeptides-for-analysis"><a href="#Cleavage-of-polypeptides-for-analysis" class="headerlink" title="Cleavage of polypeptides for analysis"></a>Cleavage of polypeptides for analysis</h3><ol>
<li>Strong acid - not sequence specific</li>
<li>Sequence-specific proteolytic enzymes (proteases)</li>
<li>Sequence-specific chemical cleavage (<strong>cyanogen</strong> bromide cleavage at <strong>methonine residues</strong>)</li>
</ol>
<p>Only used in the mutant protein that is not in the databases </p>
<p>FPLC: </p>
<p>HPLC: high pressure </p>
<hr>
<h2 id="CW1"><a href="#CW1" class="headerlink" title="CW1"></a>CW1</h2><p>How would you design a purification regime, step-by-step, to isolate the mutant enzyme X from the liver of mouse “Y”?  Explain the rationale of your purification scheme. [<strong>60 marks</strong>] </p>
<p><strong>Answer</strong>:</p>
<p><code>step1:</code>  Since the provided raw sample is from the liver of mouse “Y”, which means the liver need to be smashed into the liquid format</p>
<p><code>step2:</code> The smashed liver liquid are then processed in a high speed blender, breaking out the cell membranes and releasing the inside cell components.</p>
<p><code>step3:</code>Owing to the fact that our target is to isolate the mutant enzyme, we need to utilize some filter papers to roughly filter out those target contents. </p>
<p><code>step4: </code>Ammonium Sulfate precipitation method can be used to partially purify the mixture. According to the protocols, we need to slowly add saturated ammonium sulfate to bring the final concentration to 50% saturation, which means the total amount of ammonium sulfate should equal with the sample amount. Therefore, the ammonium sulfate should add into the samples gradually, in this case, the ideal percentage should be 10%, 20%…50%. But owing to the different condition, some bias may exist, therefore, 5 columens should be prepared to hold the precipitates under different ammonium sulfate percentage, each columns should be clearly labeled for convenience. Adding the ammonium sulfate very slowly ensures that local concentration around the site of addition does not exceed the desired salt concentration. The insoluble aggregates would be our products. </p>
<p><code>Rational of step4</code>: The whole mechanisms behind this approach is the electrolyte-nonelectrolyte interaction.  In solution, proteins form hydrogen bonds with water molecules through their exposed polar and ionic groups. When high concentrations of small, highly charged ions such as ammonium sulfate are added, these groups compete with the proteins to bind to the water molecules. This removes the water molecules from the protein and decreases its solubility, resulting in precipitation. Critical factors that affect the concentration at which a particular protein will precipitate include: the number and position of polar groups, molecular weight of the protein, pH of the solution, and temperature at which the precipitation is performed.</p>
<p><code>step5:</code> the isolated 5 samples are resuspended in the buffer to be active and soluble again. As suggested in the problems that the presence of enzyme X can be detected with radioassay by the capture of the released 14CO2. In this case, five samples are feed into substrates respectively and the sample whose generated CO2 can be detected by radioassay could be considered as the sample enzyme x. We denoted this sample as sample A. </p>
<p><code>step6:</code>SDS-PAGE should be incorporated to test the sample A’s purity</p>
<p><code>Rational of step6:</code>SDS-PAGE is an analytical technique to separate proteins based on their molecular weight. In SDS-PAGE, the use of sodium dodecyl sulfate (SDS, also known as sodium lauryl sulfate) and polyacrylamide gel largely eliminates the influence of the structure and charge, and proteins are separated solely based on polypeptide chain length. Therefore, it could be used to figure out the samples’ purity.</p>
<p><code>step7</code>:  Dialysis technique should be included to perform purification  for samples containing macromolecules, in this case, enzymes. Some unnecessary salts in the previous steps could be removed.</p>
<p><code>Rational of step7     </code>Dialysis is a separation technique that facilitates the removal of small, unwanted compounds from macromolecules in solution by selective and passive diffusion through a semi-permeable membrane. </p>
<p><code>step8</code>: ion exchange chromatography should be utilized on sample A in the following step. Owing to the fact we have no idea about the pI (isoelectric point) of the protein, we need to include both anion and cation exchanger. In other words, the samples need to conduct the experiments twice.</p>
<p>We should note that the tightly bound ones would be washed away by the high concentrated sodium chloride, the eluted samples should be collected by different tubes</p>
<p>SDS-PAGE should be conducted to figure out the inside components.</p>
<p><code>Rational of step8:</code></p>
<p>The net surface charge of proteins varies according to the surrounding pH. The pH at which a protein has no net charge is called isoelectric point (pI). Above its isoelectric point (pI), a protein will bind to a positively charged anion exchanger. Below its pI, a protein will bind to a negatively charged cation exchanger.</p>
<p>Proteins bind as they are loaded onto a column at low ionic strength. The conditions are then altered so that bound substances are desorbed differentially. Elution is usually performed by increasing salt concentration or changing pH in a gradient. Sodium chloride is usually considered as a suitable candidate to elute the bond ones gradually. By adjusting different concentration sodium chloride solution, we could obtain different protein samples. </p>
<p><code>Step9</code> Repeat the step in step5 and denoted the target sample as sample B.</p>
<p><code>step10:</code> Hydrophobic Interaction Chromatography (HIC) would be utilized on sample B to purify the enzyme X.</p>
<p><code>Rational of step10</code>: it could separate the protein based on the hydrophobicity. In this type of chromatography, hydrophobic groups such as phenyl, octyl, or butyl, are attached to the stationary column. Proteins that pass through the column that have hydrophobic amino acid side chains on their surfaces are able to interact with and bind to the hydrophobic groups on the column. Finally, buffer need to be used to elute the rest proteins.</p>
<p><code>step11</code>: Similarly,  repeat the step in step5 &amp; step6, and denoted the target sample as sample C </p>
<p><code>step12</code>: Gel filtration chromatography should also be included on sample C to filter out the proteins based on their molecules weight. The technique is ideal for the final polishing steps in a purification when sample volumes have been reduced.</p>
<p><code>Rational of step12</code> : The gel filtration chromatography is based on the molecular size and the hydrodynamic volume of the components. The molecules are separated by the differential exclusion or inclusion of solutes as they pass through the stationary phase containing heteroporous cross-linked polymeric gel or beads. Different permeation rates of the solute molecules cause them to sift in the interior of the gel particles. A column of the porous matrix is in equilibrium with the mobile phase for separation of the molecules. Large molecules are entirely excluded from the pores and come first in the effluent. Smaller molecules get distributed between the mobile phase and the outside of the sieve. Then, they pass through the column at a slower rate and appear later in the effluent.</p>
<p><code>step13 </code>: Similarly,  repeat the step in step5 &amp; step6, and denoted the target sample as sample D</p>
<p>Based on the previous 13 steps, we could successfully isolate the mutant enzyme X from the liver of mouse “Y”. For the protein identification, you could turn to 2D gel electrophoresis and mass spectrometry for more details. </p>
<p><code>step14:</code>2D Gel Electrophoresis can be used to characterize the proteins in the remaining samples. </p>
<p><code>Rational behind step14:</code> two dimensional gel electrophoresis could separate proteins based on pI in 1st dimension and operate proteins based on molecular weight in 2nd dimensions</p>
<p><code>step15</code>: Search the online databases, hoping to find the similar results.</p>
<hr>
<p>Your partially purified sample exhibits very strong enzyme X activity in 14C assay but your SDS-PAGE looks like the example shown in the figure below: </p>
<p>What strategies would you use to identify enzyme X from the mixture? Explain how you would carry out the experiment and the reasons of your strategies. [<strong>40 marks</strong>] </p>
<p><strong>Answer:</strong></p>
<p>As learnt from the lecture, there are two widely used methods in protein identification, Edman Degradation and Mass spectrometry.</p>
<p>Edman degradation is the process of purifying protein by sequentially removing one residue at a time from the amino end of a peptide. Phenyl isothiocyanate is added to create a phenylthiocarbamoyl derivative with the N-terminal to remove only one residue at a time. The N-terminal is then cleaved under less harsh acidic conditions, creating a cyclic compound of phenylthiohydantoin PTH-amino acid. This does not damage the protein and leaves two constituents of the peptide.</p>
<p>At the first step, the proteins need to be broken down to peptide fragments by the strong acid which could ensure their terminal would be blocked and cannot be sequenced. High pressure liquid chromatography (HPLC) would be employed to separate the peptides. The amino acid sequencing are then followed to conduct sequence alignment to match the known sequences in the database to identify the protein.</p>
<p>The second choice would be the mass spectrometry. Mass spectrometry measures the mass-to-charge ratio of charged particles, mass spectrometry is used for determining masses of particles and the elemental composition of a sample of molecule as well as for elucidating the chemical structure of molecules such as peptides. </p>
<p>As can be seen from the figure that, four bands, suggesting four proteins, have been got. With either Edman degradation or mass spectrometry, we could utilize the approaches like BLAST to align the sequence stored in database. Since enzyme X in this case still could not be determined, we could use the “process of elimination” to identify the other three proteins.</p>

        </div>

        
            <div class="post-copyright-info">
                <div class="article-copyright-info-container">
    <ul>
        <li>Post title：BIO312</li>
        <li>Post author：Yuxuan Wu</li>
        <li>Create time：2021-03-02 19:44:01</li>
        <li>
            Post link：yuxuanwu17.github.io2021/03/02/2021-03-02-BIO312/
        </li>
        <li>
            Copyright Notice：All articles in this blog are licensed under <a class="license" target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by-nc-sa/4.0/deed.zh">BY-NC-SA</a> unless stating additionally.
        </li>
    </ul>
</div>

            </div>
        

        
            <div class="article-nav">
                
                    <div class="article-prev">
                        <a class="prev"
                           rel="prev"
                           href="/2021/03/10/2021-03-10-biotechnology-presentation/"
                        >
                            <span class="left arrow-icon flex-center">
                              <i class="fas fa-chevron-left"></i>
                            </span>
                            <span class="title flex-center">
                                <span class="post-nav-title-item">biotechnology presentation</span>
                                <span class="post-nav-item">Prev posts</span>
                            </span>
                        </a>
                    </div>
                
                
                    <div class="article-next">
                        <a class="next"
                           rel="next"
                           href="/2021/02/21/m6a_virus/"
                        >
                            <span class="title flex-center">
                                <span class="post-nav-title-item">m6A virus project related papers and notes</span>
                                <span class="post-nav-item">Next posts</span>
                            </span>
                            <span class="right arrow-icon flex-center">
                              <i class="fas fa-chevron-right"></i>
                            </span>
                        </a>
                    </div>
                
            </div>
        

        
    </div>
</div>


                
            </div>

        </div>

        <div class="page-main-content-bottom">
            <footer class="footer">
    <div class="info-container">
        <div class="copyright-info info-item">
            &copy;
            
              <span>2020</span> -
            
            2021 <i class="fas fa-heart icon-animate"></i> <a href="/">Yuxuan Wu</a>
        </div>
        
            <script async  src="//busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
            <div class="website-count info-item">
                
                    <span id="busuanzi_container_site_uv">
                        Visitor Count <span id="busuanzi_value_site_uv"></span>&ensp;
                    </span>
                
                
                    <span id="busuanzi_container_site_pv">
                        Totalview <span id="busuanzi_value_site_pv"></span>
                    </span>
                
            </div>
        
        <div class="theme-info info-item">
            Powered by <a target="_blank" href="https://hexo.io">Hexo</a> | Theme <a class="theme-version" target="_blank" href="https://github.com/XPoet/hexo-theme-keep">Keep v3.3.1</a>
        </div>
    </div>
</footer>

        </div>
    </div>

    
        <div class="post-tools">
            <div class="post-tools-container">
    <ul class="tools-list">
        <!-- TOC aside toggle -->
        
            <li class="tools-item page-aside-toggle">
                <i class="fas fa-outdent"></i>
            </li>
        

        <!-- go comment -->
        
    </ul>
</div>

        </div>
    

    <div class="right-bottom-side-tools">
        <div class="side-tools-container">
    <ul class="side-tools-list">
        <li class="tools-item tool-font-adjust-plus flex-center">
            <i class="fas fa-search-plus"></i>
        </li>

        <li class="tools-item tool-font-adjust-minus flex-center">
            <i class="fas fa-search-minus"></i>
        </li>

        <li class="tools-item tool-expand-width flex-center">
            <i class="fas fa-arrows-alt-h"></i>
        </li>

        <li class="tools-item tool-dark-light-toggle flex-center">
            <i class="fas fa-moon"></i>
        </li>

        <!-- rss -->
        

        

        <li class="tools-item tool-scroll-to-bottom flex-center">
            <i class="fas fa-arrow-down"></i>
        </li>
    </ul>

    <ul class="exposed-tools-list">
        <li class="tools-item tool-toggle-show flex-center">
            <i class="fas fa-cog fa-spin"></i>
        </li>
        
            <li class="tools-item tool-scroll-to-top flex-center">
                <i class="arrow-up fas fa-arrow-up"></i>
                <span class="percent"></span>
            </li>
        
    </ul>
</div>

    </div>

    
        <aside class="page-aside">
            <div class="post-toc-wrap">
    <div class="post-toc">
        <ol class="nav"><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-seperation"><span class="nav-number">1.</span> <span class="nav-text">Protein seperation</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#SDS-PAGE"><span class="nav-number">2.</span> <span class="nav-text">SDS-PAGE:</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Isoelectric-point-pi"><span class="nav-number">3.</span> <span class="nav-text">Isoelectric point (pi)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Isoelectric-Focusing"><span class="nav-number">4.</span> <span class="nav-text">Isoelectric Focusing</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#2D-gel"><span class="nav-number">5.</span> <span class="nav-text">2D gel</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Western-Blot"><span class="nav-number">6.</span> <span class="nav-text">Western Blot</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-identification"><span class="nav-number">7.</span> <span class="nav-text">Protein identification</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#Edman-degradation"><span class="nav-number">7.1.</span> <span class="nav-text">Edman degradation</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Mass-Spectrometry"><span class="nav-number">7.2.</span> <span class="nav-text">Mass Spectrometry:</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-sequencing"><span class="nav-number">8.</span> <span class="nav-text">Protein sequencing</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#Sanger-sequence-protein-identification"><span class="nav-number">8.1.</span> <span class="nav-text">Sanger sequence (protein identification)</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Cleavage-of-polypeptides-for-analysis"><span class="nav-number">9.</span> <span class="nav-text">Cleavage of polypeptides for analysis</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#CW1"><span class="nav-number"></span> <span class="nav-text">CW1</span></a>
    </div>
</div>
        </aside>
    

    .image-viewer-container {
  position: fixed;
  left: 0;
  top: 0;
  width: 100%;
  height: 100%;
  display: flex;
  align-items: center;
  justify-content: center;
  background: rgba(0, 0, 0, 0);
  visibility: hidden;
  z-index: $z-index-8;
  padding: 6%;
  box-sizing: border-box;
  transition-t("visibility, background", "0, 0", "0.3, 0.3", "ease, ease");

  &.active {
    background: rgba(0, 0, 0, 0.5);
    visibility: visible;

    img {
      cursor: zoom-out;
      transform: scale(1);
      padding: 2px;
      background: var(--background-color);
    }
  }

  img {
    max-width: 100%;
    max-height: 100%;
    transform: scale(0);
    transition-t("transform", "0", "0.3", "ease");
  }
}




    
        <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
          <span class="search-input-field-pre">
            <i class="fas fa-keyboard"></i>
          </span>
            <div class="search-input-container">
                <input autocomplete="off"
                       autocorrect="off"
                       autocapitalize="off"
                       placeholder="Search..."
                       spellcheck="false"
                       type="search"
                       class="search-input"
                >
            </div>
            <span class="popup-btn-close">
                <i class="fas fa-times"></i>
            </span>
        </div>
        <div id="search-result">
            <div id="no-result">
                <i class="fas fa-spinner fa-pulse fa-5x fa-fw"></i>
            </div>
        </div>
    </div>
</div>

    

</main>




<script src="/js/utils.js"></script>

<script src="/js/main.js"></script>

<script src="/js/header-shrink.js"></script>

<script src="/js/back2top.js"></script>

<script src="/js/dark-light-toggle.js"></script>



    
<script src="/js/local-search.js"></script>




    
<script src="/js/code-copy.js"></script>





<div class="post-scripts">
    
        
<script src="/js/left-side-toggle.js"></script>

<script src="/js/libs/anime.min.js"></script>

<script src="/js/toc.js"></script>

    
</div>



</body>
</html>
