<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="keywords" content="Hexo Theme Keep">
    <meta name="description" content="Welcome to my personal blogs">
    <meta name="author" content="Yuxuan Wu">
    
    <title>
        
            BIO312 |
        
        Yuxuan Wu
    </title>
    <link rel="shortcut icon" href="/images/logo.svg">
    
<link rel="stylesheet" href="/css/style.css">

    
<link rel="stylesheet" href="/css/font-awesome.min.css">

    <script id="hexo-configurations">
    let KEEP = window.KEEP || {};
    KEEP.hexo_config = {"hostname":"example.com","root":"/","language":"en","path":"search.xml"};
    KEEP.theme_config = {"toc":{"enable":true,"number":true,"expand_all":false},"style":{"primary_color":"#0066CC","avatar":"/images/avatar.svg","favicon":"/images/logo.svg","article_img_align":"left","left_side_width":"260px","content_max_width":"920px","hover":{"shadow":true,"scale":true},"first_screen":{"enable":true,"background_img":"/images/bg.svg","description":"Keep coding, Keep hungry."},"scroll":{"progress_bar":{"enable":true},"percent":{"enable":true}}},"local_search":{"enable":true,"preload":true},"code_copy":{"enable":true,"style":"default"},"pjax":{"enable":false},"lazyload":{"enable":false},"version":"3.3.1"};
    KEEP.language_ago = {"second":"%s seconds ago","minute":"%s minutes ago","hour":"%s hours ago","day":"%s days age","week":"%s weeks age","month":"%s months age","year":"%s years age"};
  </script>
<!-- hexo injector head_end start -->
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css">

<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/hexo-math@4.0.0/dist/style.css">
<!-- hexo injector head_end end --><meta name="generator" content="Hexo 5.3.0"><style>mjx-container[jax="SVG"] {
  direction: ltr;
}

mjx-container[jax="SVG"] > svg {
  overflow: visible;
}

mjx-container[jax="SVG"] > svg a {
  fill: blue;
  stroke: blue;
}

mjx-container[jax="SVG"][display="true"] {
  display: block;
  text-align: center;
  margin: 1em 0;
}

mjx-container[jax="SVG"][justify="left"] {
  text-align: left;
}

mjx-container[jax="SVG"][justify="right"] {
  text-align: right;
}

g[data-mml-node="merror"] > g {
  fill: red;
  stroke: red;
}

g[data-mml-node="merror"] > rect[data-background] {
  fill: yellow;
  stroke: none;
}

g[data-mml-node="mtable"] > line[data-line] {
  stroke-width: 70px;
  fill: none;
}

g[data-mml-node="mtable"] > rect[data-frame] {
  stroke-width: 70px;
  fill: none;
}

g[data-mml-node="mtable"] > .mjx-dashed {
  stroke-dasharray: 140;
}

g[data-mml-node="mtable"] > .mjx-dotted {
  stroke-linecap: round;
  stroke-dasharray: 0,140;
}

g[data-mml-node="mtable"] > svg {
  overflow: visible;
}

[jax="SVG"] mjx-tool {
  display: inline-block;
  position: relative;
  width: 0;
  height: 0;
}

[jax="SVG"] mjx-tool > mjx-tip {
  position: absolute;
  top: 0;
  left: 0;
}

mjx-tool > mjx-tip {
  display: inline-block;
  padding: .2em;
  border: 1px solid #888;
  font-size: 70%;
  background-color: #F8F8F8;
  color: black;
  box-shadow: 2px 2px 5px #AAAAAA;
}

g[data-mml-node="maction"][data-toggle] {
  cursor: pointer;
}

mjx-status {
  display: block;
  position: fixed;
  left: 1em;
  bottom: 1em;
  min-width: 25%;
  padding: .2em .4em;
  border: 1px solid #888;
  font-size: 90%;
  background-color: #F8F8F8;
  color: black;
}

foreignObject[data-mjx-xml] {
  font-family: initial;
  line-height: normal;
  overflow: visible;
}

.MathJax path {
  stroke-width: 3;
}

mjx-container[display="true"] {
  overflow: auto hidden;
}

mjx-container[display="true"] + br {
  display: none;
}
</style></head>


<body>
<div class="progress-bar-container">
    
        <span class="scroll-progress-bar"></span>
    

    
</div>


<main class="page-container">

    

    <div class="page-main-content">

        <div class="page-main-content-top">
            <header class="header-wrapper">

    <div class="header-content">
        <div class="left">
            <a class="logo-title" href="/">
                Yuxuan Wu
            </a>
        </div>

        <div class="right">
            <div class="pc">
                <ul class="menu-list">
                    
                        <li class="menu-item">
                            <a class=""
                               href="/"
                            >
                                HOME
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/archives"
                            >
                                ARCHIVES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/categories"
                            >
                                CATEGORIES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/tags"
                            >
                                TAGS
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/about"
                            >
                                ABOUT
                            </a>
                        </li>
                    
                    
                        <li class="menu-item search search-popup-trigger">
                            <i class="fas fa-search"></i>
                        </li>
                    
                </ul>
            </div>
            <div class="mobile">
                
                    <div class="icon-item search search-popup-trigger"><i class="fas fa-search"></i></div>
                
                <div class="icon-item menu-bar">
                    <div class="menu-bar-middle"></div>
                </div>
            </div>
        </div>
    </div>

    <div class="header-drawer">
        <ul class="drawer-menu-list">
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/">HOME</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/archives">ARCHIVES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/categories">CATEGORIES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/tags">TAGS</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/about">ABOUT</a>
                </li>
            
        </ul>
    </div>

    <div class="window-mask"></div>

</header>


        </div>

        <div class="page-main-content-middle">

            <div class="main-content normal-code-theme">

                
                    <div class="fade-in-down-animation">
    <div class="article-content-container">

        <div class="article-title">
            <span class="title-hover-animation">BIO312</span>
        </div>

        
            <div class="article-header">
                <div class="avatar">
                    <img src="/images/avatar.svg">
                </div>
                <div class="info">
                    <div class="author">
                        <span class="name">Yuxuan Wu</span>
                        <span class="level">Lv8</span>
                    </div>
                    <div class="meta-info">
                        <div class="article-meta-info">
    <span class="article-date article-meta-item">
        <i class="fas fa-edit"></i> 2021-03-02 19:44:01
    </span>
    
        <span class="article-categories article-meta-item">
            <i class="fas fa-folder"></i>
            <ul>
                
                    <li>
                        <a href="/categories/Biology/">Biology</a>
                    </li>
                
            </ul>
        </span>
    
    
        <span class="article-tags article-meta-item">
            <i class="fas fa-tags"></i>
            <ul>
                
                    <li>
                        <a href="/tags/notes/">notes</a>
                    </li>
                
            </ul>
        </span>
    

    
    
        <span class="article-wordcount article-meta-item">
            <i class="fas fa-file-word"></i> <span>3.9k Words</span>
        </span>
    
    
        <span class="article-min2read article-meta-item">
            <i class="fas fa-clock"></i> <span>24 Mins</span>
        </span>
    
    
        <span class="article-pv article-meta-item">
            <i class="fas fa-eye"></i> <span id="busuanzi_value_page_pv"></span>
        </span>
    
</div>

                    </div>
                </div>
            </div>
        

        <div class="article-content markdown-body">
            <p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210521135014.png" alt="image-20210521135014278"></p>
<p>[toc]</p>
<h2 id="Revision-protein-structure-and-methods-of-analysis"><a href="#Revision-protein-structure-and-methods-of-analysis" class="headerlink" title="Revision protein structure and methods of analysis"></a>Revision protein structure and methods of analysis</h2><h3 id="Protein-seperation"><a href="#Protein-seperation" class="headerlink" title="Protein seperation"></a>Protein seperation</h3><ol>
<li><strong>SDS-PAGE:</strong> basis-molecular weight</li>
<li><strong>Isoelectric focusing:</strong> electric charge difference</li>
<li><strong>Chromatic methods-useful adjuncts to gel-based approaches</strong><ol>
<li>HPLC - high performance liquid chromatography </li>
<li>Thin-layer chromatography </li>
</ol>
</li>
<li><strong>2D gel electrophoresis</strong><ul>
<li>Powerful gel-based method commonly used to analyze complex samples</li>
<li>To characterize full range of proteins in the sample, not just a few specific proteins</li>
</ul>
</li>
</ol>
<h3 id="SDS-PAGE"><a href="#SDS-PAGE" class="headerlink" title="SDS-PAGE:"></a>SDS-PAGE:</h3><ul>
<li>Proteins are usually mixed with the detergent, SDS and a tracking dye to make the sample</li>
<li>SDS binds to the protein and gives it size-dependent negative charge and gives it size-dependent negative charge and consistent hydrodynamic properties</li>
</ul>
<h3 id="Isoelectric-point-pi"><a href="#Isoelectric-point-pi" class="headerlink" title="Isoelectric point (pi)"></a>Isoelectric point (pi)</h3><ul>
<li>ph at which molecule has net zero charge （determined using computer program for known sequence or empirically ）</li>
</ul>
<h3 id="Isoelectric-Focusing"><a href="#Isoelectric-Focusing" class="headerlink" title="Isoelectric Focusing"></a>Isoelectric Focusing</h3><p>Electrophoresis through polyacrylamide gel in which there is a pH gradient</p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210302212707.png" alt="image-20210302212707320"></p>
<h3 id="2D-gel"><a href="#2D-gel" class="headerlink" title="2D gel"></a>2D gel</h3><p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210302212808.png" alt="image-20210302212808199"></p>
<h3 id="Western-Blot"><a href="#Western-Blot" class="headerlink" title="Western Blot"></a>Western Blot</h3><p>a.k.a: immunoblotting</p>
<ul>
<li>Proteins first electrophoreses by SDS-PAGE</li>
<li>Wet gel is placed against nitrocellulose</li>
<li>In electrophoretic chamber: </li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210302212907.png" alt="image-20210302212906975"></p>
<blockquote>
<p>Ammonium sulfate precipitation is one of the most commonly used methods for protein purification from a solution. In solution, proteins form hydrogen bonds with water molecules through their exposed polar and ionic groups. When high concentrations of small, highly charged ions such as ammonium sulfate are added, these groups compete with the proteins to bind to the water molecules. This removes the water molecules from the protein and decreases its solubility, resulting in precipitation. Critical factors that affect the concentration at which a particular protein will precipitate include: the number and position of polar groups, molecular weight of the protein, pH of the solution, and temperature at which the precipitation is performed. The concentration at which antibodies precipitate varies among species; most rabbit antibodies precipitate with a 40% saturated solution, whereas mouse antibodies require 45-50% saturation.</p>
</blockquote>
<blockquote>
<p><strong>Protocol</strong></p>
<ol>
<li>Allow serum or ascitic fluid to thaw, determine total volume, and centrifuge at 3000g for 30 minutes.</li>
<li>Transfer sample to a beaker containing a stir bar and place on a magnetic stirrer.</li>
<li>While the sample is stirring, slowly add saturated ammonium sulfate to bring the final concentration to 50% saturation.<ol>
<li>Volume of ammonium sulfate needed is equal to the volume of sample.</li>
<li>Adding the ammonium sulfate very slowly ensures that local concentration around the site of addition does not exceed the desired salt concentration.</li>
</ol>
</li>
<li>Once the total volume of ammonium sulfate is added, move beaker to 4°C for 6 hours or overnight.</li>
<li>Transfer to conical tube and centrifuge the precipitate at 3000g for 30 minutes.</li>
<li>Carefully remove and discard supernatant. Invert conical tube and drain well. For serum or ascites, resuspend pellet in 30%-50% of the starting volume in 1XPBS. For monoclonal antibody tissue culture supernatants, resuspend pellet in 10% of the starting volume in 1X PBS.</li>
<li>Transfer antibody solution to dialysis tubing and dialyze versus three changes of 1XPBS/0.08% Sodium Azide. Be sure to allow enough space for expansion of the antibody solution during dialysis. Normally twice the re-suspended volume is sufficient.</li>
<li>Remove antibody solution from the tubing and centrifuge to remove any remaining debris.</li>
<li>Determine the concentration and store at -80°C for long-term storage.</li>
</ol>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210302213429.png" alt="image-20210302213429785"></p>
<h3 id="Protein-identification"><a href="#Protein-identification" class="headerlink" title="Protein identification"></a>Protein identification</h3><h4 id="Edman-degradation"><a href="#Edman-degradation" class="headerlink" title="Edman degradation"></a>Edman degradation</h4><p>Sequence aa in a peptide. </p>
<p>Amino-terminal residue is <strong>labeled</strong> and cleaved from the pipetted <strong>without disrupting the peptide bonds</strong> between other amino acid residues</p>
<h4 id="Mass-Spectrometry"><a href="#Mass-Spectrometry" class="headerlink" title="Mass Spectrometry:"></a>Mass Spectrometry:</h4><p>Analytical techniques that measures the mass-to-charge ratio of charged particles, mass spectrometry is used for determining <strong>masses</strong> of particles and the <strong>elemental composition of a sample</strong> of molecules as well as for <strong>elucidating the chemical structure of molecules</strong> such as peptide</p>
<h3 id="Protein-sequencing"><a href="#Protein-sequencing" class="headerlink" title="Protein sequencing"></a>Protein sequencing</h3><h4 id="Sanger-sequence-protein-identification"><a href="#Sanger-sequence-protein-identification" class="headerlink" title="Sanger sequence (protein identification)"></a>Sanger sequence (protein identification)</h4><h3 id="Cleavage-of-polypeptides-for-analysis"><a href="#Cleavage-of-polypeptides-for-analysis" class="headerlink" title="Cleavage of polypeptides for analysis"></a>Cleavage of polypeptides for analysis</h3><ol>
<li>Strong acid - not sequence specific</li>
<li>Sequence-specific proteolytic enzymes (proteases)</li>
<li>Sequence-specific chemical cleavage (<strong>cyanogen</strong> bromide cleavage at <strong>methonine residues</strong>)</li>
</ol>
<p>Only used in the mutant protein that is not in the databases </p>
<p>FPLC: </p>
<p>HPLC: high pressure </p>
<h2 id="Microcharacterisation-of-proteins"><a href="#Microcharacterisation-of-proteins" class="headerlink" title="Microcharacterisation of proteins"></a>Microcharacterisation of proteins</h2><h2 id="Advanced-proteomic-techniques"><a href="#Advanced-proteomic-techniques" class="headerlink" title="Advanced proteomic techniques"></a>Advanced proteomic techniques</h2><h3 id="Top-Down-or-Bottom-up-analysis"><a href="#Top-Down-or-Bottom-up-analysis" class="headerlink" title="Top-Down or Bottom-up analysis"></a>Top-Down or Bottom-up analysis</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523141041.png" alt="image-20210523141041633"></p>
<h3 id="Top-down-advantages"><a href="#Top-down-advantages" class="headerlink" title="Top-down advantages"></a>Top-down advantages</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523141131.png" alt="image-20210523141131007"></p>
<h3 id="disadvantages-for-top-down"><a href="#disadvantages-for-top-down" class="headerlink" title="disadvantages for top-down"></a>disadvantages for top-down</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523141200.png" alt="image-20210523141200747"></p>
<h2 id="Lecture-4-Quantitative-Mass-Spec-based-Proteomics"><a href="#Lecture-4-Quantitative-Mass-Spec-based-Proteomics" class="headerlink" title="Lecture 4_Quantitative Mass Spec-based Proteomics"></a>Lecture 4_Quantitative Mass Spec-based Proteomics</h2><ul>
<li><p>Principles of quantitative proteomics</p>
</li>
<li><p>Gel vs. non-gel approaches</p>
</li>
<li><p>Labeling vs. non-labeling approaches</p>
</li>
<li><p>Case study: candidate biomarker generation using label-free approaches</p>
</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523141655.png" alt="image-20210523141655702"></p>
<h3 id="Summary-of-label-methods"><a href="#Summary-of-label-methods" class="headerlink" title="Summary of label methods"></a>Summary of label methods</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523143459.png" alt="image-20210523143459324"></p>
<h3 id="Pros-and-Cons-for-label-free-methods"><a href="#Pros-and-Cons-for-label-free-methods" class="headerlink" title="Pros and Cons for label-free methods"></a>Pros and Cons for label-free methods</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523143106.png" alt="image-20210523143106440"></p>
<h4 id="Methods"><a href="#Methods" class="headerlink" title="Methods"></a>Methods</h4><p>Spectral counting  &amp; Peak-intensity</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523143644.png" alt="image-20210523143644199"></p>
<h3 id="LC-MS-Chromatographic-Alignment"><a href="#LC-MS-Chromatographic-Alignment" class="headerlink" title="LC-MS Chromatographic Alignment"></a>LC-MS Chromatographic Alignment</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523143857.png" alt="image-20210523143857757"></p>
<h2 id="Tutorial-1"><a href="#Tutorial-1" class="headerlink" title="Tutorial 1"></a>Tutorial 1</h2><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525174544.png" alt="image-20210525174544609"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525174744.png" alt="image-20210525174744143"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525174753.png" alt="image-20210525174753723"></p>
<h3 id="The-principle-of-SDS-PAGE"><a href="#The-principle-of-SDS-PAGE" class="headerlink" title="The principle of SDS-PAGE"></a>The principle of SDS-PAGE</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525175334.png" alt="image-20210525175334329"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525175357.png" alt="image-20210525175357744"></p>
<p>![image-20210525175525318](/Users/yuxuan/Library/Application Support/typora-user-images/image-20210525175525318.png)</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525175915.png"></p>
<h3 id="Western-plot"><a href="#Western-plot" class="headerlink" title="Western plot"></a>Western plot</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525181246.png" alt="image-20210525181246074"></p>
<h2 id="Tutorial-2"><a href="#Tutorial-2" class="headerlink" title="Tutorial 2"></a>Tutorial 2</h2><h3 id="Labeling-vs-non-labeling"><a href="#Labeling-vs-non-labeling" class="headerlink" title="Labeling vs non-labeling"></a>Labeling vs non-labeling</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523145719.png" alt="image-20210523145719943"></p>
<p><code>comparison between label &amp; label free</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523160242.png" alt="image-20210523160242108"></p>
<h3 id="Label-free-approaches"><a href="#Label-free-approaches" class="headerlink" title="Label free approaches"></a>Label free approaches</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523151047.png" alt="image-20210523151047928"></p>
<h3 id="pros-and-cons-of-label-free-protein"><a href="#pros-and-cons-of-label-free-protein" class="headerlink" title="pros and cons of label free protein"></a>pros and cons of label free protein</h3><ul>
<li><p>Pros:</p>
<ul>
<li>Technically <code>simple</code><ul>
<li>No complicated (or expensive) labelling or tagging protocol</li>
</ul>
</li>
<li>Can <code>avoid complex pre-separation</code><ul>
<li>direct LC-MS is okay</li>
</ul>
</li>
</ul>
</li>
<li><p>Cons:</p>
<ul>
<li>Still can be <code>expensive</code><ul>
<li>Need plenty of replicates to get statistical power (machine time!)</li>
</ul>
</li>
<li>Need good data analysis software</li>
<li>Not straightforward to validate results from big data sets</li>
<li><code>Low abundance proteins</code> hard to measure accurately</li>
</ul>
</li>
</ul>
<h3 id="Why-does-MS1-intensity-not-tell-us-peptide-abundance-directly"><a href="#Why-does-MS1-intensity-not-tell-us-peptide-abundance-directly" class="headerlink" title="Why does MS1 intensity not tell us peptide abundance directly?"></a>Why does MS1 intensity not tell us peptide abundance directly?</h3><ul>
<li>Peptide sequences <code>ionise to different extents</code><ul>
<li>based on the biophysical properties of a given sequence</li>
</ul>
</li>
<li>Thus, measured ion intensity <code>only weakly correlates to the abundance</code> of the peptide (and hence) protein in the proteome</li>
<li><code>Not simple to extract the total ion abundance</code> for a given peptide in different runs</li>
<li><code>Difficult</code> to ensure that you are <code>comparing the same signal in parallel runs</code></li>
</ul>
<h3 id="Spectral-counting"><a href="#Spectral-counting" class="headerlink" title="Spectral counting"></a>Spectral counting</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523152406.png" alt="image-20210523152406643"></p>
<h3 id="The-normalization-of-spectral-counts"><a href="#The-normalization-of-spectral-counts" class="headerlink" title="The normalization of spectral counts"></a>The normalization of spectral counts</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523152458.png" alt="image-20210523152458935"></p>
<h3 id="Intensity-based-label-free"><a href="#Intensity-based-label-free" class="headerlink" title="Intensity based label free"></a>Intensity based label free</h3><h4 id="Method-summary"><a href="#Method-summary" class="headerlink" title="Method summary:"></a>Method summary:</h4><ul>
<li>Compare MS1 peptide ion abundance across replicates runs;</li>
<li>Specialist software must align different parallel LC-MS runs</li>
<li>Calculate ratios from aligned MS1 data</li>
<li>Requires good software (e.g.: Mascot distiller; Progenesis; MaxQuant; OpenMS; ProteomeDiscoverer; PLGS, many others…)</li>
</ul>
<h4 id="Advantages"><a href="#Advantages" class="headerlink" title="Advantages:"></a>Advantages:</h4><ul>
<li>In theory, should be more accurate than spectral counting – uses real intensity data</li>
<li>No complicated labelling protocols</li>
</ul>
<h4 id="Disadvantages"><a href="#Disadvantages" class="headerlink" title="Disadvantages:"></a>Disadvantages:</h4><ul>
<li>Data processing is fairly CPU intensive</li>
<li>Only works well if experimental system has high technical and biological reproducibility</li>
</ul>
<h4 id="Requirements"><a href="#Requirements" class="headerlink" title="Requirements:"></a>Requirements:</h4><ul>
<li>Very good LC delivery system (must be reproducible)</li>
<li>High resolution mass spec (Orbitrap, Synapt etc…)</li>
<li>Good PC for running software</li>
</ul>
<h4 id="Labelling-quantification"><a href="#Labelling-quantification" class="headerlink" title="Labelling quantification"></a>Labelling quantification</h4><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523160136.png" alt="image-20210523160136172"></p>
<h4 id="Two-approaches"><a href="#Two-approaches" class="headerlink" title="Two approaches"></a>Two approaches</h4><ol>
<li><p>Chemical (in vitro) labeling</p>
<ul>
<li>ICAT (isotope coded affinity tag)</li>
<li>iTRAQ (isobaric tag for relative and absolute quantification)</li>
</ul>
</li>
<li><p>Metabolic [or <em>in vivo</em>] labelling</p>
<ol>
<li>SILAC [stable isotope labelling by amino acids in culture]<ul>
<li>Requires a cell culture, therefore cannot be used for experiments from other kinds of samples e.g. clinical samples</li>
</ul>
</li>
</ol>
</li>
</ol>
<h2 id="Lecture-5-Post-translational-modifications"><a href="#Lecture-5-Post-translational-modifications" class="headerlink" title="Lecture 5_Post-translational modifications"></a>Lecture 5_Post-translational modifications</h2><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523161351.png" alt="image-20210523161351854"></p>
<h3 id="Proteomic-Analysis-for-PTMs"><a href="#Proteomic-Analysis-for-PTMs" class="headerlink" title="Proteomic Analysis for PTMs"></a>Proteomic Analysis for PTMs</h3><p>Key issue in PTM analysis:  modified proteins/peptides often have <strong><code>low abundance</code></strong></p>
<ul>
<li><p>**<code>Enrichment</code>**can be used to increase their concentration in the sample</p>
</li>
<li><p>The most established methodology is <strong>phosphoproteomics</strong></p>
</li>
<li><p>Development of phosphoproteomicshas been driven by the deep interest in signalling research (pharmaapplications)</p>
</li>
<li><p>Phosphopeptides are relatively easy to enrich and mass spectrometric methods have been well established, however, the enrichment outcomes are often <code>sample-dependent</code></p>
</li>
<li><p>The field is currently moving to high-throughput of profiling methods of other PTMs and the analysis of several PTMs in parallel</p>
</li>
</ul>
<h3 id="MS-for-Phosphopeptides"><a href="#MS-for-Phosphopeptides" class="headerlink" title="MS for Phosphopeptides"></a>MS for Phosphopeptides</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523163237.png" alt="image-20210523163237578"></p>
<ul>
<li>Precursor/ Reporter ion scanning</li>
<li>Neutral-Loss-Dependent <mjx-container class="MathJax" jax="SVG"><svg style="vertical-align: -0.05ex" xmlns="http://www.w3.org/2000/svg" width="5.027ex" height="1.645ex" role="img" focusable="false" viewBox="0 -705 2221.9 727"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g data-mml-node="math"><g data-mml-node="mi"><path data-c="1D440" d="M289 629Q289 635 232 637Q208 637 201 638T194 648Q194 649 196 659Q197 662 198 666T199 671T201 676T203 679T207 681T212 683T220 683T232 684Q238 684 262 684T307 683Q386 683 398 683T414 678Q415 674 451 396L487 117L510 154Q534 190 574 254T662 394Q837 673 839 675Q840 676 842 678T846 681L852 683H948Q965 683 988 683T1017 684Q1051 684 1051 673Q1051 668 1048 656T1045 643Q1041 637 1008 637Q968 636 957 634T939 623Q936 618 867 340T797 59Q797 55 798 54T805 50T822 48T855 46H886Q892 37 892 35Q892 19 885 5Q880 0 869 0Q864 0 828 1T736 2Q675 2 644 2T609 1Q592 1 592 11Q592 13 594 25Q598 41 602 43T625 46Q652 46 685 49Q699 52 704 61Q706 65 742 207T813 490T848 631L654 322Q458 10 453 5Q451 4 449 3Q444 0 433 0Q418 0 415 7Q413 11 374 317L335 624L267 354Q200 88 200 79Q206 46 272 46H282Q288 41 289 37T286 19Q282 3 278 1Q274 0 267 0Q265 0 255 0T221 1T157 2Q127 2 95 1T58 0Q43 0 39 2T35 11Q35 13 38 25T43 40Q45 46 65 46Q135 46 154 86Q158 92 223 354T289 629Z"></path></g><g data-mml-node="msup" transform="translate(1051, 0)"><g data-mml-node="mi"><path data-c="1D446" d="M308 24Q367 24 416 76T466 197Q466 260 414 284Q308 311 278 321T236 341Q176 383 176 462Q176 523 208 573T273 648Q302 673 343 688T407 704H418H425Q521 704 564 640Q565 640 577 653T603 682T623 704Q624 704 627 704T632 705Q645 705 645 698T617 577T585 459T569 456Q549 456 549 465Q549 471 550 475Q550 478 551 494T553 520Q553 554 544 579T526 616T501 641Q465 662 419 662Q362 662 313 616T263 510Q263 480 278 458T319 427Q323 425 389 408T456 390Q490 379 522 342T554 242Q554 216 546 186Q541 164 528 137T492 78T426 18T332 -20Q320 -22 298 -22Q199 -22 144 33L134 44L106 13Q83 -14 78 -18T65 -22Q52 -22 52 -14Q52 -11 110 221Q112 227 130 227H143Q149 221 149 216Q149 214 148 207T144 186T142 153Q144 114 160 87T203 47T255 29T308 24Z"></path></g><g data-mml-node="mi" transform="translate(696.6, 363) scale(0.707)"><path data-c="1D45B" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path></g></g></g></g></svg></mjx-container></li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523163411.png" alt="image-20210523163411638"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523163428.png" alt="image-20210523163428035"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523163439.png" alt="image-20210523163439522"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523170011.png" alt="image-20210523170011446"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523170023.png" alt="image-20210523170023740"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523170034.png"></p>
<h2 id="Lecture-09-Folding-in-vivo"><a href="#Lecture-09-Folding-in-vivo" class="headerlink" title="Lecture 09 Folding in vivo"></a>Lecture 09 Folding in vivo</h2><h3 id="Folding-in-vitro-and-in-vivo"><a href="#Folding-in-vitro-and-in-vivo" class="headerlink" title="Folding in vitro and in vivo"></a>Folding in vitro and in vivo</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523171332.png" alt="image-20210523171332137"></p>
<h3 id="Macromolecular-crowding"><a href="#Macromolecular-crowding" class="headerlink" title="Macromolecular crowding"></a>Macromolecular crowding</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523171412.png" alt="image-20210523171412425"></p>
<h3 id="Effects-of-crowding"><a href="#Effects-of-crowding" class="headerlink" title="Effects of crowding"></a>Effects of crowding</h3><p><strong>definition</strong></p>
<p>Molecular crowdingis ageneric term for the condition where a significant volume of a solution, or cytoplasm for example, is occupied with things other than water</p>
<p>Fact:</p>
<p>-association rate (k<strong>a</strong>) increase significantly</p>
<p>-dissociation rate (k<strong>d</strong>) decrease significantly (k<strong>d</strong>=1/k<strong>a</strong>)</p>
<h3 id="Molecular-chaperones"><a href="#Molecular-chaperones" class="headerlink" title="Molecular chaperones"></a>Molecular chaperones</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523171644.png" alt="image-20210523171644917"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523171653.png" alt="image-20210523171653069"></p>
<h2 id="Lecture-10-Folding-in-vitro"><a href="#Lecture-10-Folding-in-vitro" class="headerlink" title="Lecture 10 Folding in vitro"></a>Lecture 10 Folding in vitro</h2><h3 id="The-Levinthal-Paradox"><a href="#The-Levinthal-Paradox" class="headerlink" title="The Levinthal Paradox"></a>The Levinthal Paradox</h3><p><code>Conclusion:</code> Protein folding is not random, must have pathways.</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525215304.png" alt="image-20210525215304583"></p>
<h3 id="Equilibrium-unfolding"><a href="#Equilibrium-unfolding" class="headerlink" title="Equilibrium unfolding"></a>Equilibrium unfolding</h3><ul>
<li><p>switch off part of the interactions in the native protein under different denaturing conditions such as chemical denaturants, low pH, high salt and high temperature</p>
</li>
<li><p>understand which types of native structure can be preserved by the remaining interactions</p>
</li>
</ul>
<h3 id="Anfinsen-Experiment"><a href="#Anfinsen-Experiment" class="headerlink" title="Anfinsen Experiment"></a>Anfinsen Experiment</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525215803.png" alt="image-20210525215803427"></p>
<blockquote>
<p>Conclusion: All the information necessary for folding the peptide chain into its native structure is contained in the primary amino acid sequence of the peptide.</p>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525215922.png" alt="image-20210525215922335"></p>
<blockquote>
<p> Conclusion: The native form of a protein has the thermodynamically most stable structure.</p>
</blockquote>
<h3 id="Molten-Globule-State-MG"><a href="#Molten-Globule-State-MG" class="headerlink" title="Molten Globule State (MG)"></a>Molten Globule State (MG)</h3><ul>
<li>It is an intermediate of the folding transition U→MG→F</li>
<li>It is a compact globule, yet expanded over a native radius</li>
<li>Native-like secondary structure, can be measured by CD and NMR proton exchange rate</li>
<li>It has a slowly fluctuating tertiary structure which gives no detectable near UV CD signal and gives quenched fluorescence signal with broadened NMR chemical peaks</li>
<li>Non-specific assembly of secondary structure and hydrophobic interactions, which allows ANS to bind and gives an enhanced ANS fluorescence </li>
<li>MG is about a 10 % increase in size than the native state</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525220949.png" alt="image-20210525220948934"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525221041.png" alt="image-20210525221041197"></p>
<h3 id="Protein-melting-thermal-shift"><a href="#Protein-melting-thermal-shift" class="headerlink" title="Protein melting (thermal shift)"></a>Protein melting (thermal shift)</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525221117.png" alt="image-20210525221117616"></p>
<h2 id="Lecture-11-SPR"><a href="#Lecture-11-SPR" class="headerlink" title="Lecture 11 SPR"></a>Lecture 11 SPR</h2><h2 id="Lecture-12-Crystallography"><a href="#Lecture-12-Crystallography" class="headerlink" title="Lecture 12 Crystallography"></a>Lecture 12 Crystallography</h2><h3 id="Whole-process"><a href="#Whole-process" class="headerlink" title="Whole process"></a>Whole process</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524083431.png" alt="image-20210524083431460"></p>
<h3 id="X-ray-diffraction-of-a-crystal"><a href="#X-ray-diffraction-of-a-crystal" class="headerlink" title="X-ray diffraction of a crystal"></a>X-ray diffraction of a crystal</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524083522.png" alt="image-20210524083522318"></p>
<h3 id="Raw-data"><a href="#Raw-data" class="headerlink" title="Raw data"></a>Raw data</h3><p>Every atom in a unit cell contributes to every reflection in the diffraction pattern.</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524083728.png" alt="image-20210524083727937"></p>
<p>Each diffraction spot (reflection) contains information on the position of every atom!</p>
<p><strong>The degree of order in the crystal determines the quality of the diffraction data and ultimately the quality of the final atomic model</strong></p>
<p>“low resolution”</p>
<p>“high resolution”</p>
<h3 id="From-diffraction-to-electron-density-map"><a href="#From-diffraction-to-electron-density-map" class="headerlink" title="From diffraction to electron density map"></a>From diffraction to electron density map</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524084221.png" alt="image-20210524084221744"></p>
<h3 id="Solving-the-phase-problem"><a href="#Solving-the-phase-problem" class="headerlink" title="Solving the phase problem"></a>Solving the phase problem</h3><ul>
<li>The diffraction data <code>does not give the phase angle </code> that is needed to calculate the electron density map.</li>
<li>Have to <code>get the phase angle through other methods</code>. <ul>
<li>Isomorphous Replacement: <code>Insert a heavy metal atom into crystal protein</code>, and <code>locate in diffraction pattern</code> and in the cell. Use the location of metal ion to find the phase angle for the other protein atoms.</li>
</ul>
</li>
<li>Requirements:<ul>
<li>Add atom with same unit cell size.</li>
<li>Cannot disturb protein structure.</li>
</ul>
</li>
</ul>
<h3 id="Phase-problem"><a href="#Phase-problem" class="headerlink" title="Phase problem"></a>Phase problem</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524085508.png" alt="image-20210524085508442"> </p>
<h3 id="Resolution"><a href="#Resolution" class="headerlink" title="Resolution"></a>Resolution</h3><blockquote>
<p>Resolution:</p>
<p>another measure of how good your model is.</p>
</blockquote>
<ul>
<li>Resolution gives the size of the smallest molecule you can see or resolve.  </li>
<li>Dependent on the amount of data ultimately phased and used in structure determination.</li>
</ul>
<h3 id="Limitations-of-the-phase-problem"><a href="#Limitations-of-the-phase-problem" class="headerlink" title="Limitations of the phase problem"></a>Limitations of the phase problem</h3><ul>
<li><p>Crystallizing Protein:</p>
<ul>
<li>Fragile</li>
<li>Requires a crystal with shortest side 0.2 mm</li>
</ul>
</li>
<li><p>Flaws of Crystallization:</p>
<ul>
<li>Disorder in Unit Cell</li>
<li>Vibrations of molecules</li>
<li>Distortion in Crystallization</li>
</ul>
</li>
</ul>
<h3 id="Steps-of-protein-X-ray-crystallography"><a href="#Steps-of-protein-X-ray-crystallography" class="headerlink" title="Steps of protein X-ray crystallography"></a>Steps of protein X-ray crystallography</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524085954.png" alt="image-20210524085954600"></p>
<h3 id="R-factor-measuring-convergence"><a href="#R-factor-measuring-convergence" class="headerlink" title="R-factor: measuring convergence"></a>R-factor: measuring convergence</h3><ul>
<li><p>To compare the generated electron density map and your model, you have to use the R-factor (Reliability Factor)</p>
</li>
<li><p>Measurement of the agreement between the crystallographic model and the experimental X-ray diffraction ie. how well the refined structure predicts the observed data</p>
</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524090443.png" alt="image-20210524090443676"></p>
<h3 id="Occupancy"><a href="#Occupancy" class="headerlink" title="Occupancy"></a>Occupancy</h3><p>The occupancy “nj” of atom “j”: </p>
<blockquote>
<p>a measurement of the fraction of molecules in the crystal in which atom j actually occupies the position specified in the model. </p>
</blockquote>
<ul>
<li><p>If all molecules in the crystal are precisely identical, then occupancies for all atoms are 1.00. </p>
</li>
<li><p>Occupancy is necessary because occasionally two or more distinct conformations are observed for a small region like a surface side chain. For example, if the <strong>two conformations occur with equal frequency, then atoms involved receive occupancies of 0.5 in each of their two possible positions</strong>. </p>
</li>
<li><p>So, “occupancies” is an estimates of the frequency of alternative conformations, giving some additional information about the dynamics of the protein molecule. </p>
</li>
</ul>
<h3 id="Redundancy"><a href="#Redundancy" class="headerlink" title="Redundancy"></a>Redundancy</h3><p>Redundancy = (number of measured reflections) / (number of unique reflections).</p>
<ul>
<li><p>The calculation gives the <strong>average number of independent measurements of each reflection</strong> in a crystallographic data set. </p>
</li>
<li><p>Because several independent measurements are done for each reflection. </p>
</li>
<li><p>To improve accuracy in measuring reflection intensities, data collection strategies are intentionally designed to take advantage of symmetry and overlap to give redundancy of measurement. </p>
</li>
<li><p>Such statistical parameters as standard deviation are used to measure agreement among the repeated measurements.</p>
</li>
</ul>
<h3 id="Resolution-1"><a href="#Resolution-1" class="headerlink" title="Resolution"></a>Resolution</h3><ul>
<li><p>Reflect the quality of the image.</p>
</li>
<li><p>In X-ray crystallography, “2-Å model” means that the model takes into account diffraction from sets of equivalent, parallel planes of atoms spaced as closely as 2 Å in the unit cell. </p>
</li>
</ul>
<h3 id="“B-Factor-Temperature-factor-”"><a href="#“B-Factor-Temperature-factor-”" class="headerlink" title="“B-Factor (Temperature factor)”"></a>“B-Factor (Temperature factor)”</h3><ul>
<li><p>B-factor can be thought of as a measure of <code>how much an atom oscillates or vibrates</code> around the position specified in the model.</p>
</li>
<li><p>Atoms at <code>side-chain termini</code> are expected to <code>exhibit more freedom of movement</code> than main-chain atoms, and this movement amounts to spreading each atom over a small region of space. </p>
</li>
<li><p>Diffraction is affected by this variation in atomic position.</p>
</li>
<li><p>From the <code>temperature factors</code> computed during refinement, we learn which atoms in the molecule have the most freedom of movement, and we gain some insight into the dynamics of our largely static model. </p>
</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524174928.png" alt="image-20210524174928076"></p>
<h3 id="Two-types-of-solvent-ordered-and-disordered"><a href="#Two-types-of-solvent-ordered-and-disordered" class="headerlink" title="Two types of solvent: ordered and disordered"></a>Two types of solvent: ordered and disordered</h3><ul>
<li><p><code>Ordered water</code> molecules show up as discrete blobs of electron density in contact with the protein or with other ordered water molecules</p>
</li>
<li><p><code>Disordered water</code> regions show up as featureless (flat) electron density</p>
</li>
</ul>
<p><strong>Position of N and O atoms in Gln(and Asn) side chain</strong> </p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524174908.png" alt="image-20210524174908366"></p>
<h2 id="Lecture-13-14-NMR"><a href="#Lecture-13-14-NMR" class="headerlink" title="Lecture 13  14 NMR"></a>Lecture 13  14 NMR</h2><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525184805.png" alt="image-20210525184805350"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525184839.png" alt="image-20210525184839551"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525191917.png" alt="image-20210525191917854"></p>
<h3 id="demo-in-class"><a href="#demo-in-class" class="headerlink" title="demo in class"></a>demo in class</h3><p><code>answer a:</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525193314.png" alt="image-20210525193314205"></p>
<p><code>answer b:</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525193910.png" alt="image-20210525193910236"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525194221.png" alt="image-20210525194221673"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525194936.png" alt="image-20210525194936354"></p>
<h3 id="Types-of-triple-resonance"><a href="#Types-of-triple-resonance" class="headerlink" title="Types of triple resonance"></a>Types of triple resonance</h3><p><code>Nomenclature: </code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526105913.png" alt="image-20210526105913534"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526110441.png" alt="image-20210526110441526"></p>
<h3 id="Protein-dynamics"><a href="#Protein-dynamics" class="headerlink" title="Protein dynamics"></a>Protein dynamics</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526111144.png" alt="image-20210526111144037"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526111414.png" alt="image-20210526111414430"></p>
<h3 id="NMR-shows-change-in-configuration"><a href="#NMR-shows-change-in-configuration" class="headerlink" title="NMR: shows change in configuration"></a>NMR: shows change in configuration</h3><p><code>mutations can effect folding and stability </code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526113844.png" alt="image-20210526113844230"></p>
<h3 id="Binding-constants"><a href="#Binding-constants" class="headerlink" title="Binding constants"></a>Binding constants</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526113918.png" alt="image-20210526113918581"></p>
<h2 id="Tutorial-on-NMR"><a href="#Tutorial-on-NMR" class="headerlink" title="Tutorial on NMR"></a>Tutorial on NMR</h2><h3 id="Question-1"><a href="#Question-1" class="headerlink" title="Question 1"></a>Question 1</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526114257.png" alt="image-20210526114257882"></p>
<p><code>answer</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526114513.png" alt="image-20210526114513266"></p>
<h3 id="Question-2"><a href="#Question-2" class="headerlink" title="Question 2"></a>Question 2</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526115416.png" alt="image-20210526115416672"></p>
<p><code>answer</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526115618.png" alt="image-20210526115618544"></p>
<h3 id="Question-3"><a href="#Question-3" class="headerlink" title="Question 3"></a>Question 3</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526120145.png" alt="image-20210526120145040"></p>
<p><code>answer</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526120446.png" alt="image-20210526120446037"></p>
<h2 id="Lecture-15-CD"><a href="#Lecture-15-CD" class="headerlink" title="Lecture 15 CD"></a>Lecture 15 CD</h2><h3 id="Circular-Polarized-Light"><a href="#Circular-Polarized-Light" class="headerlink" title="Circular Polarized Light"></a>Circular Polarized Light</h3><p>Nearly all molecules synthesized by living organisms are <code>optically active</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524212852.png" alt="image-20210524212852153"></p>
<h3 id="Circular-Polarized-Light-1"><a href="#Circular-Polarized-Light-1" class="headerlink" title="Circular Polarized Light"></a>Circular Polarized Light</h3><p><code>Ellipticity</code></p>
<p>Plane polarized light is the sum of **left **and **right **circularly polarized light. </p>
<ul>
<li><p>When plane polarized light passes through a solution containing an optically active substance the left and right circularly polarized components of the plane-polarized light are <code>absorbed by different amounts. </code></p>
</li>
<li><p>When these components are recombined they appear as elliptically polarized light. The ellipticityis defined as Q, the angle of polarization and is measured in degrees, (deg) or millidegrees, (mdeg).</p>
</li>
</ul>
<h3 id="CD-signal-of-proteins"><a href="#CD-signal-of-proteins" class="headerlink" title="CD signal of proteins"></a>CD signal of proteins</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524213120.png" alt="image-20210524213120909"></p>
<h3 id="Standard-curves"><a href="#Standard-curves" class="headerlink" title="Standard curves"></a>Standard curves</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524213353.png" alt="image-20210524213353329"></p>
<h3 id="Using-CD-to-Monitor-tertiary-structure"><a href="#Using-CD-to-Monitor-tertiary-structure" class="headerlink" title="Using CD to Monitor tertiary structure"></a>Using CD to Monitor tertiary structure</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524213554.png" alt="image-20210524213554780"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524213608.png" alt="image-20210524213608760"></p>
<h3 id="Summary-of-CD"><a href="#Summary-of-CD" class="headerlink" title="Summary of CD"></a>Summary of CD</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524214916.png" alt="image-20210524214916555"></p>
<h3 id="Demo-in-the-class"><a href="#Demo-in-the-class" class="headerlink" title="Demo in the class"></a>Demo in the class</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525202814.png" alt="image-20210525202813881"></p>
<p>A: <mjx-container class="MathJax" jax="SVG"><svg style="vertical-align: -0.025ex" xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g data-mml-node="math"><g data-mml-node="mi"><path data-c="1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></g></g></g></svg></mjx-container> helix </p>
<p>B: y has more structure compared to x since absence of ATP, the kinase is a random coil and with the inclusion of ATP, protein turn to a secondary structure. </p>
<p>x: background high </p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525203347.PNG" alt="IMG_0420"></p>
<h2 id="Tutorial-for-final-paper"><a href="#Tutorial-for-final-paper" class="headerlink" title="Tutorial for final paper"></a><code>Tutorial for final paper</code></h2><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524221949.png" alt="image-20210524221949150"></p>
<p>What does <mjx-container class="MathJax" jax="SVG"><svg style="vertical-align: -0.025ex" xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g data-mml-node="math"><g data-mml-node="mi"><path data-c="1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></g></g></g></svg></mjx-container>-helix A shows a high protection at &lt;1 ms during refolding imply? </p>
<blockquote>
<p>That’s the part where molecular condense and collapse and shield from the water</p>
</blockquote>
<p>What does the low protection readout of <mjx-container class="MathJax" jax="SVG"><svg style="vertical-align: -0.439ex" xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g data-mml-node="math"><g data-mml-node="mi"><path data-c="1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></g></g></g></svg></mjx-container> strands-1,-2 and -3 &lt;1 ms imply ?</p>
<blockquote>
<p>They are completely exposed to the water.</p>
</blockquote>
<p>Sequence of the folding process</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525083611.png" alt="image-20210525083610873"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525195556.png" alt="image-20210525195556857"></p>
<p>How do you use gel filtration to determine a protein is dimer or monomer?</p>
<blockquote>
<p>SDS-PAGE denature the protein, the structure would be destroyed. If this protein is a dimer, it will divid into two proteins.</p>
<p>Assume we have a kinase of 20KD from SDS-PAGE (monomer). 40KD(dimer). A set of marker run at GF column. Add protein to the marker and run run the gel again. Marker would stick to the protein. If protein came up at the same location, it will be a monomer, if out in the 40, then will be the dimer</p>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525201634.png" alt="image-20210525201634649"></p>
<h3 id="Concepts-of-thermal-shift-CD"><a href="#Concepts-of-thermal-shift-CD" class="headerlink" title="Concepts of thermal shift + CD"></a>Concepts of thermal shift + CD</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525203347.PNG" alt="IMG_0420"></p>
<h3 id="concepts-of-SPR-kon-kof-kd"><a href="#concepts-of-SPR-kon-kof-kd" class="headerlink" title="concepts of SPR: kon; kof; kd"></a>concepts of SPR: kon; kof; kd</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525205400.png" alt="image-20210525205400376"></p>
<hr>
<h2 id="CW1"><a href="#CW1" class="headerlink" title="CW1"></a>CW1</h2><h3 id="Requirements-1"><a href="#Requirements-1" class="headerlink" title="Requirements"></a>Requirements</h3><p>Courseworks requirements</p>
<p>Protocol</p>
<p>columns</p>
<p>Expected output</p>
<p>Which columns</p>
<p>Justification for choosing that column </p>
<p>Expected output? Diagrams</p>
<hr>
<p>How would you design a purification regime, step-by-step, to isolate the mutant enzyme X from the liver of mouse “Y”?  Explain the rationale of your purification scheme. [<strong>60 marks</strong>] </p>
<p><strong>Answer</strong>:</p>
<p><code>step1:</code>  Since the provided raw sample is from the liver of mouse “Y”, which means the liver need to be smashed into the liquid format</p>
<p><code>step2:</code> The smashed liver liquid are then processed in a high speed blender, breaking out the cell membranes and releasing the inside cell components.</p>
<p><code>step3:</code>Owing to the fact that our target is to isolate the mutant enzyme, we need to utilize some filter papers to roughly filter out those target contents. </p>
<p><code>step4: </code>Ammonium Sulfate precipitation method can be used to partially purify the mixture. According to the protocols, we need to slowly add saturated ammonium sulfate to bring the final concentration to 50% saturation, which means the total amount of ammonium sulfate should equal with the sample amount. Therefore, the ammonium sulfate should add into the samples gradually, in this case, the ideal percentage should be 10%, 20%…50%. But owing to the different condition, some bias may exist, therefore, 5 columens should be prepared to hold the precipitates under different ammonium sulfate percentage, each columns should be clearly labeled for convenience. Adding the ammonium sulfate very slowly ensures that local concentration around the site of addition does not exceed the desired salt concentration. The insoluble aggregates would be our products. </p>
<p><code>Rational of step4</code>: The whole mechanisms behind this approach is the electrolyte-nonelectrolyte interaction.  In solution, proteins form hydrogen bonds with water molecules through their exposed polar and ionic groups. When high concentrations of small, highly charged ions such as ammonium sulfate are added, these groups compete with the proteins to bind to the water molecules. This removes the water molecules from the protein and decreases its solubility, resulting in precipitation. Critical factors that affect the concentration at which a particular protein will precipitate include: the number and position of polar groups, molecular weight of the protein, pH of the solution, and temperature at which the precipitation is performed.</p>
<p><code>step5:</code> the isolated 5 samples are resuspended in the buffer to be active and soluble again. As suggested in the problems that the presence of enzyme X can be detected with radioassay by the capture of the released 14CO2. In this case, five samples are feed into substrates respectively and the sample whose generated CO2 can be detected by radioassay could be considered as the sample enzyme x. We denoted this sample as sample A. </p>
<p><code>step6:</code>SDS-PAGE should be incorporated to test the sample A’s purity</p>
<p><code>Rational of step6:</code>SDS-PAGE is an analytical technique to separate proteins based on their molecular weight. In SDS-PAGE, the use of sodium dodecyl sulfate (SDS, also known as sodium lauryl sulfate) and polyacrylamide gel largely eliminates the influence of the structure and charge, and proteins are separated solely based on polypeptide chain length. Therefore, it could be used to figure out the samples’ purity.</p>
<p><code>step7</code>:  Dialysis technique should be included to perform purification  for samples containing macromolecules, in this case, enzymes. Some unnecessary salts in the previous steps could be removed.</p>
<p><code>Rational of step7     </code>Dialysis is a separation technique that facilitates the removal of small, unwanted compounds from macromolecules in solution by selective and passive diffusion through a semi-permeable membrane. </p>
<p><code>step8</code>: ion exchange chromatography should be utilized on sample A in the following step. Owing to the fact we have no idea about the pI (isoelectric point) of the protein, we need to include both anion and cation exchanger. In other words, the samples need to conduct the experiments twice.</p>
<p>We should note that the tightly bound ones would be washed away by the high concentrated sodium chloride, the eluted samples should be collected by different tubes</p>
<p>SDS-PAGE should be conducted to figure out the inside components.</p>
<p><code>Rational of step8:</code></p>
<p>The net surface charge of proteins varies according to the surrounding pH. The pH at which a protein has no net charge is called isoelectric point (pI). Above its isoelectric point (pI), a protein will bind to a positively charged anion exchanger. Below its pI, a protein will bind to a negatively charged cation exchanger.</p>
<p>Proteins bind as they are loaded onto a column at low ionic strength. The conditions are then altered so that bound substances are desorbed differentially. Elution is usually performed by increasing salt concentration or changing pH in a gradient. Sodium chloride is usually considered as a suitable candidate to elute the bond ones gradually. By adjusting different concentration sodium chloride solution, we could obtain different protein samples. </p>
<p><code>Step9</code> Repeat the step in step5 and denoted the target sample as sample B.</p>
<p><code>step10:</code> Hydrophobic Interaction Chromatography (HIC) would be utilized on sample B to purify the enzyme X.</p>
<p><code>Rational of step10</code>: it could separate the protein based on the hydrophobicity. In this type of chromatography, hydrophobic groups such as phenyl, octyl, or butyl, are attached to the stationary column. Proteins that pass through the column that have hydrophobic amino acid side chains on their surfaces are able to interact with and bind to the hydrophobic groups on the column. Finally, buffer need to be used to elute the rest proteins.</p>
<p><code>step11</code>: Similarly,  repeat the step in step5 &amp; step6, and denoted the target sample as sample C </p>
<p><code>step12</code>: Gel filtration chromatography should also be included on sample C to filter out the proteins based on their molecules weight. The technique is ideal for the final polishing steps in a purification when sample volumes have been reduced.</p>
<p><code>Rational of step12</code> : The gel filtration chromatography is based on the molecular size and the hydrodynamic volume of the components. The molecules are separated by the differential exclusion or inclusion of solutes as they pass through the stationary phase containing heteroporous cross-linked polymeric gel or beads. Different permeation rates of the solute molecules cause them to sift in the interior of the gel particles. A column of the porous matrix is in equilibrium with the mobile phase for separation of the molecules. Large molecules are entirely excluded from the pores and come first in the effluent. Smaller molecules get distributed between the mobile phase and the outside of the sieve. Then, they pass through the column at a slower rate and appear later in the effluent.</p>
<p><code>step13 </code>: Similarly,  repeat the step in step5 &amp; step6, and denoted the target sample as sample D</p>
<p>Based on the previous 13 steps, we could successfully isolate the mutant enzyme X from the liver of mouse “Y”. For the protein identification, you could turn to 2D gel electrophoresis and mass spectrometry for more details. </p>
<p><code>step14:</code>2D Gel Electrophoresis can be used to characterize the proteins in the remaining samples. </p>
<p><code>Rational behind step14:</code> two dimensional gel electrophoresis could separate proteins based on pI in 1st dimension and operate proteins based on molecular weight in 2nd dimensions</p>
<p><code>step15</code>: Search the online databases, hoping to find the similar results.</p>
<hr>
<p>Your partially purified sample exhibits very strong enzyme X activity in 14C assay but your SDS-PAGE looks like the example shown in the figure below: </p>
<p>What strategies would you use to identify enzyme X from the mixture? Explain how you would carry out the experiment and the reasons of your strategies. [<strong>40 marks</strong>] </p>
<p><strong>Answer:</strong></p>
<p>As learnt from the lecture, there are two widely used methods in protein identification, Edman Degradation and Mass spectrometry.</p>
<p>Edman degradation is the process of purifying protein by sequentially removing one residue at a time from the amino end of a peptide. Phenyl isothiocyanate is added to create a phenylthiocarbamoyl derivative with the N-terminal to remove only one residue at a time. The N-terminal is then cleaved under less harsh acidic conditions, creating a cyclic compound of phenylthiohydantoin PTH-amino acid. This does not damage the protein and leaves two constituents of the peptide.</p>
<p>At the first step, the proteins need to be broken down to peptide fragments by the strong acid which could ensure their terminal would be blocked and cannot be sequenced. High pressure liquid chromatography (HPLC) would be employed to separate the peptides. The amino acid sequencing are then followed to conduct sequence alignment to match the known sequences in the database to identify the protein.</p>
<p>The second choice would be the mass spectrometry. Mass spectrometry measures the mass-to-charge ratio of charged particles, mass spectrometry is used for determining masses of particles and the elemental composition of a sample of molecule as well as for elucidating the chemical structure of molecules such as peptides. </p>
<p>As can be seen from the figure that, four bands, suggesting four proteins, have been got. With either Edman degradation or mass spectrometry, we could utilize the approaches like BLAST to align the sequence stored in database. Since enzyme X in this case still could not be determined, we could use the “process of elimination” to identify the other three proteins.</p>

        </div>

        
            <div class="post-copyright-info">
                <div class="article-copyright-info-container">
    <ul>
        <li>Post title：BIO312</li>
        <li>Post author：Yuxuan Wu</li>
        <li>Create time：2021-03-02 19:44:01</li>
        <li>
            Post link：yuxuanwu17.github.io2021/03/02/2021-03-02-BIO312/
        </li>
        <li>
            Copyright Notice：All articles in this blog are licensed under <a class="license" target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by-nc-sa/4.0/deed.zh">BY-NC-SA</a> unless stating additionally.
        </li>
    </ul>
</div>

            </div>
        

        
            <div class="article-nav">
                
                    <div class="article-prev">
                        <a class="prev"
                           rel="prev"
                           href="/2021/03/10/2021-03-10-BIO312-cw1/"
                        >
                            <span class="left arrow-icon flex-center">
                              <i class="fas fa-chevron-left"></i>
                            </span>
                            <span class="title flex-center">
                                <span class="post-nav-title-item">BIO312 cw1</span>
                                <span class="post-nav-item">Prev posts</span>
                            </span>
                        </a>
                    </div>
                
                
                    <div class="article-next">
                        <a class="next"
                           rel="next"
                           href="/2021/02/21/m6a_virus/"
                        >
                            <span class="title flex-center">
                                <span class="post-nav-title-item">m6A virus project related papers and notes</span>
                                <span class="post-nav-item">Next posts</span>
                            </span>
                            <span class="right arrow-icon flex-center">
                              <i class="fas fa-chevron-right"></i>
                            </span>
                        </a>
                    </div>
                
            </div>
        

        
    </div>
</div>


                
            </div>

        </div>

        <div class="page-main-content-bottom">
            <footer class="footer">
    <div class="info-container">
        <div class="copyright-info info-item">
            &copy;
            
              <span>2020</span> -
            
            2021 <i class="fas fa-heart icon-animate"></i> <a href="/">Yuxuan Wu</a>
        </div>
        
            <script async  src="//busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
            <div class="website-count info-item">
                
                    <span id="busuanzi_container_site_uv">
                        Visitor Count <span id="busuanzi_value_site_uv"></span>&ensp;
                    </span>
                
                
                    <span id="busuanzi_container_site_pv">
                        Totalview <span id="busuanzi_value_site_pv"></span>
                    </span>
                
            </div>
        
        <div class="theme-info info-item">
            Powered by <a target="_blank" href="https://hexo.io">Hexo</a> | Theme <a class="theme-version" target="_blank" href="https://github.com/XPoet/hexo-theme-keep">Keep v3.3.1</a>
        </div>
    </div>
</footer>

        </div>
    </div>

    
        <div class="post-tools">
            <div class="post-tools-container">
    <ul class="tools-list">
        <!-- TOC aside toggle -->
        
            <li class="tools-item page-aside-toggle">
                <i class="fas fa-outdent"></i>
            </li>
        

        <!-- go comment -->
        
    </ul>
</div>

        </div>
    

    <div class="right-bottom-side-tools">
        <div class="side-tools-container">
    <ul class="side-tools-list">
        <li class="tools-item tool-font-adjust-plus flex-center">
            <i class="fas fa-search-plus"></i>
        </li>

        <li class="tools-item tool-font-adjust-minus flex-center">
            <i class="fas fa-search-minus"></i>
        </li>

        <li class="tools-item tool-expand-width flex-center">
            <i class="fas fa-arrows-alt-h"></i>
        </li>

        <li class="tools-item tool-dark-light-toggle flex-center">
            <i class="fas fa-moon"></i>
        </li>

        <!-- rss -->
        

        

        <li class="tools-item tool-scroll-to-bottom flex-center">
            <i class="fas fa-arrow-down"></i>
        </li>
    </ul>

    <ul class="exposed-tools-list">
        <li class="tools-item tool-toggle-show flex-center">
            <i class="fas fa-cog fa-spin"></i>
        </li>
        
            <li class="tools-item tool-scroll-to-top flex-center">
                <i class="arrow-up fas fa-arrow-up"></i>
                <span class="percent"></span>
            </li>
        
    </ul>
</div>

    </div>

    
        <aside class="page-aside">
            <div class="post-toc-wrap">
    <div class="post-toc">
        <ol class="nav"><li class="nav-item nav-level-2"><a class="nav-link" href="#Revision-protein-structure-and-methods-of-analysis"><span class="nav-number">1.</span> <span class="nav-text">Revision protein structure and methods of analysis</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-seperation"><span class="nav-number">1.1.</span> <span class="nav-text">Protein seperation</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#SDS-PAGE"><span class="nav-number">1.2.</span> <span class="nav-text">SDS-PAGE:</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Isoelectric-point-pi"><span class="nav-number">1.3.</span> <span class="nav-text">Isoelectric point (pi)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Isoelectric-Focusing"><span class="nav-number">1.4.</span> <span class="nav-text">Isoelectric Focusing</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#2D-gel"><span class="nav-number">1.5.</span> <span class="nav-text">2D gel</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Western-Blot"><span class="nav-number">1.6.</span> <span class="nav-text">Western Blot</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-identification"><span class="nav-number">1.7.</span> <span class="nav-text">Protein identification</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#Edman-degradation"><span class="nav-number">1.7.1.</span> <span class="nav-text">Edman degradation</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Mass-Spectrometry"><span class="nav-number">1.7.2.</span> <span class="nav-text">Mass Spectrometry:</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-sequencing"><span class="nav-number">1.8.</span> <span class="nav-text">Protein sequencing</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#Sanger-sequence-protein-identification"><span class="nav-number">1.8.1.</span> <span class="nav-text">Sanger sequence (protein identification)</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Cleavage-of-polypeptides-for-analysis"><span class="nav-number">1.9.</span> <span class="nav-text">Cleavage of polypeptides for analysis</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Microcharacterisation-of-proteins"><span class="nav-number">2.</span> <span class="nav-text">Microcharacterisation of proteins</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Advanced-proteomic-techniques"><span class="nav-number">3.</span> <span class="nav-text">Advanced proteomic techniques</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Top-Down-or-Bottom-up-analysis"><span class="nav-number">3.1.</span> <span class="nav-text">Top-Down or Bottom-up analysis</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Top-down-advantages"><span class="nav-number">3.2.</span> <span class="nav-text">Top-down advantages</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#disadvantages-for-top-down"><span class="nav-number">3.3.</span> <span class="nav-text">disadvantages for top-down</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-4-Quantitative-Mass-Spec-based-Proteomics"><span class="nav-number">4.</span> <span class="nav-text">Lecture 4_Quantitative Mass Spec-based Proteomics</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Summary-of-label-methods"><span class="nav-number">4.1.</span> <span class="nav-text">Summary of label methods</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Pros-and-Cons-for-label-free-methods"><span class="nav-number">4.2.</span> <span class="nav-text">Pros and Cons for label-free methods</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#Methods"><span class="nav-number">4.2.1.</span> <span class="nav-text">Methods</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#LC-MS-Chromatographic-Alignment"><span class="nav-number">4.3.</span> <span class="nav-text">LC-MS Chromatographic Alignment</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Tutorial-1"><span class="nav-number">5.</span> <span class="nav-text">Tutorial 1</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#The-principle-of-SDS-PAGE"><span class="nav-number">5.1.</span> <span class="nav-text">The principle of SDS-PAGE</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Western-plot"><span class="nav-number">5.2.</span> <span class="nav-text">Western plot</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Tutorial-2"><span class="nav-number">6.</span> <span class="nav-text">Tutorial 2</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Labeling-vs-non-labeling"><span class="nav-number">6.1.</span> <span class="nav-text">Labeling vs non-labeling</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Label-free-approaches"><span class="nav-number">6.2.</span> <span class="nav-text">Label free approaches</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#pros-and-cons-of-label-free-protein"><span class="nav-number">6.3.</span> <span class="nav-text">pros and cons of label free protein</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Why-does-MS1-intensity-not-tell-us-peptide-abundance-directly"><span class="nav-number">6.4.</span> <span class="nav-text">Why does MS1 intensity not tell us peptide abundance directly?</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Spectral-counting"><span class="nav-number">6.5.</span> <span class="nav-text">Spectral counting</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#The-normalization-of-spectral-counts"><span class="nav-number">6.6.</span> <span class="nav-text">The normalization of spectral counts</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Intensity-based-label-free"><span class="nav-number">6.7.</span> <span class="nav-text">Intensity based label free</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#Method-summary"><span class="nav-number">6.7.1.</span> <span class="nav-text">Method summary:</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Advantages"><span class="nav-number">6.7.2.</span> <span class="nav-text">Advantages:</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Disadvantages"><span class="nav-number">6.7.3.</span> <span class="nav-text">Disadvantages:</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Requirements"><span class="nav-number">6.7.4.</span> <span class="nav-text">Requirements:</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Labelling-quantification"><span class="nav-number">6.7.5.</span> <span class="nav-text">Labelling quantification</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Two-approaches"><span class="nav-number">6.7.6.</span> <span class="nav-text">Two approaches</span></a></li></ol></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-5-Post-translational-modifications"><span class="nav-number">7.</span> <span class="nav-text">Lecture 5_Post-translational modifications</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Proteomic-Analysis-for-PTMs"><span class="nav-number">7.1.</span> <span class="nav-text">Proteomic Analysis for PTMs</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#MS-for-Phosphopeptides"><span class="nav-number">7.2.</span> <span class="nav-text">MS for Phosphopeptides</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-09-Folding-in-vivo"><span class="nav-number">8.</span> <span class="nav-text">Lecture 09 Folding in vivo</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Folding-in-vitro-and-in-vivo"><span class="nav-number">8.1.</span> <span class="nav-text">Folding in vitro and in vivo</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Macromolecular-crowding"><span class="nav-number">8.2.</span> <span class="nav-text">Macromolecular crowding</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Effects-of-crowding"><span class="nav-number">8.3.</span> <span class="nav-text">Effects of crowding</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Molecular-chaperones"><span class="nav-number">8.4.</span> <span class="nav-text">Molecular chaperones</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-10-Folding-in-vitro"><span class="nav-number">9.</span> <span class="nav-text">Lecture 10 Folding in vitro</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#The-Levinthal-Paradox"><span class="nav-number">9.1.</span> <span class="nav-text">The Levinthal Paradox</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Equilibrium-unfolding"><span class="nav-number">9.2.</span> <span class="nav-text">Equilibrium unfolding</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Anfinsen-Experiment"><span class="nav-number">9.3.</span> <span class="nav-text">Anfinsen Experiment</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Molten-Globule-State-MG"><span class="nav-number">9.4.</span> <span class="nav-text">Molten Globule State (MG)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-melting-thermal-shift"><span class="nav-number">9.5.</span> <span class="nav-text">Protein melting (thermal shift)</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-11-SPR"><span class="nav-number">10.</span> <span class="nav-text">Lecture 11 SPR</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-12-Crystallography"><span class="nav-number">11.</span> <span class="nav-text">Lecture 12 Crystallography</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Whole-process"><span class="nav-number">11.1.</span> <span class="nav-text">Whole process</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#X-ray-diffraction-of-a-crystal"><span class="nav-number">11.2.</span> <span class="nav-text">X-ray diffraction of a crystal</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Raw-data"><span class="nav-number">11.3.</span> <span class="nav-text">Raw data</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#From-diffraction-to-electron-density-map"><span class="nav-number">11.4.</span> <span class="nav-text">From diffraction to electron density map</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Solving-the-phase-problem"><span class="nav-number">11.5.</span> <span class="nav-text">Solving the phase problem</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Phase-problem"><span class="nav-number">11.6.</span> <span class="nav-text">Phase problem</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Resolution"><span class="nav-number">11.7.</span> <span class="nav-text">Resolution</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Limitations-of-the-phase-problem"><span class="nav-number">11.8.</span> <span class="nav-text">Limitations of the phase problem</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Steps-of-protein-X-ray-crystallography"><span class="nav-number">11.9.</span> <span class="nav-text">Steps of protein X-ray crystallography</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#R-factor-measuring-convergence"><span class="nav-number">11.10.</span> <span class="nav-text">R-factor: measuring convergence</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Occupancy"><span class="nav-number">11.11.</span> <span class="nav-text">Occupancy</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Redundancy"><span class="nav-number">11.12.</span> <span class="nav-text">Redundancy</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Resolution-1"><span class="nav-number">11.13.</span> <span class="nav-text">Resolution</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#%E2%80%9CB-Factor-Temperature-factor-%E2%80%9D"><span class="nav-number">11.14.</span> <span class="nav-text">“B-Factor (Temperature factor)”</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Two-types-of-solvent-ordered-and-disordered"><span class="nav-number">11.15.</span> <span class="nav-text">Two types of solvent: ordered and disordered</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-13-14-NMR"><span class="nav-number">12.</span> <span class="nav-text">Lecture 13  14 NMR</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#demo-in-class"><span class="nav-number">12.1.</span> <span class="nav-text">demo in class</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Types-of-triple-resonance"><span class="nav-number">12.2.</span> <span class="nav-text">Types of triple resonance</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-dynamics"><span class="nav-number">12.3.</span> <span class="nav-text">Protein dynamics</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#NMR-shows-change-in-configuration"><span class="nav-number">12.4.</span> <span class="nav-text">NMR: shows change in configuration</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Binding-constants"><span class="nav-number">12.5.</span> <span class="nav-text">Binding constants</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Tutorial-on-NMR"><span class="nav-number">13.</span> <span class="nav-text">Tutorial on NMR</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Question-1"><span class="nav-number">13.1.</span> <span class="nav-text">Question 1</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Question-2"><span class="nav-number">13.2.</span> <span class="nav-text">Question 2</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Question-3"><span class="nav-number">13.3.</span> <span class="nav-text">Question 3</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-15-CD"><span class="nav-number">14.</span> <span class="nav-text">Lecture 15 CD</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Circular-Polarized-Light"><span class="nav-number">14.1.</span> <span class="nav-text">Circular Polarized Light</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Circular-Polarized-Light-1"><span class="nav-number">14.2.</span> <span class="nav-text">Circular Polarized Light</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#CD-signal-of-proteins"><span class="nav-number">14.3.</span> <span class="nav-text">CD signal of proteins</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Standard-curves"><span class="nav-number">14.4.</span> <span class="nav-text">Standard curves</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Using-CD-to-Monitor-tertiary-structure"><span class="nav-number">14.5.</span> <span class="nav-text">Using CD to Monitor tertiary structure</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Summary-of-CD"><span class="nav-number">14.6.</span> <span class="nav-text">Summary of CD</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Demo-in-the-class"><span class="nav-number">14.7.</span> <span class="nav-text">Demo in the class</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Tutorial-for-final-paper"><span class="nav-number">15.</span> <span class="nav-text">Tutorial for final paper</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Concepts-of-thermal-shift-CD"><span class="nav-number">15.1.</span> <span class="nav-text">Concepts of thermal shift + CD</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#concepts-of-SPR-kon-kof-kd"><span class="nav-number">15.2.</span> <span class="nav-text">concepts of SPR: kon; kof; kd</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#CW1"><span class="nav-number">16.</span> <span class="nav-text">CW1</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Requirements-1"><span class="nav-number">16.1.</span> <span class="nav-text">Requirements</span></a></li></ol></li></ol>
    </div>
</div>
        </aside>
    

    <div class="image-viewer-container">
    <img src="">
</div>


    
        <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
          <span class="search-input-field-pre">
            <i class="fas fa-keyboard"></i>
          </span>
            <div class="search-input-container">
                <input autocomplete="off"
                       autocorrect="off"
                       autocapitalize="off"
                       placeholder="Search..."
                       spellcheck="false"
                       type="search"
                       class="search-input"
                >
            </div>
            <span class="popup-btn-close">
                <i class="fas fa-times"></i>
            </span>
        </div>
        <div id="search-result">
            <div id="no-result">
                <i class="fas fa-spinner fa-pulse fa-5x fa-fw"></i>
            </div>
        </div>
    </div>
</div>

    

</main>




<script src="/js/utils.js"></script>

<script src="/js/main.js"></script>

<script src="/js/header-shrink.js"></script>

<script src="/js/back2top.js"></script>

<script src="/js/dark-light-toggle.js"></script>



    
<script src="/js/local-search.js"></script>




    
<script src="/js/code-copy.js"></script>





<div class="post-scripts">
    
        
<script src="/js/left-side-toggle.js"></script>

<script src="/js/libs/anime.min.js"></script>

<script src="/js/toc.js"></script>

    
</div>



</body>
</html>
