<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="keywords" content="Hexo Theme Keep">
    <meta name="description" content="Welcome to my personal blogs">
    <meta name="author" content="Yuxuan Wu">
    
    <title>
        
            Data preparation for multi-instance learning in m6a sequence data |
        
        Yuxuan Wu
    </title>
    <link rel="shortcut icon" href="/images/logo.svg">
    
<link rel="stylesheet" href="/css/style.css">

    
<link rel="stylesheet" href="/css/font-awesome.min.css">

    <script id="hexo-configurations">
    let KEEP = window.KEEP || {};
    KEEP.hexo_config = {"hostname":"example.com","root":"/","language":"en","path":"search.xml"};
    KEEP.theme_config = {"toc":{"enable":true,"number":true,"expand_all":false},"style":{"primary_color":"#0066CC","avatar":"/images/avatar.svg","favicon":"/images/logo.svg","article_img_align":"left","left_side_width":"260px","content_max_width":"920px","hover":{"shadow":true,"scale":true},"first_screen":{"enable":true,"background_img":"/images/bg.svg","description":"Keep coding, Keep hungry."},"scroll":{"progress_bar":{"enable":true},"percent":{"enable":true}}},"local_search":{"enable":true,"preload":true},"code_copy":{"enable":true,"style":"default"},"pjax":{"enable":false},"lazyload":{"enable":false},"version":"3.3.1"};
    KEEP.language_ago = {"second":"%s seconds ago","minute":"%s minutes ago","hour":"%s hours ago","day":"%s days age","week":"%s weeks age","month":"%s months age","year":"%s years age"};
  </script>
<!-- hexo injector head_end start -->
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css">

<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/hexo-math@4.0.0/dist/style.css">
<!-- hexo injector head_end end --><meta name="generator" content="Hexo 5.3.0"></head>


<body>
<div class="progress-bar-container">
    
        <span class="scroll-progress-bar"></span>
    

    
</div>


<main class="page-container">

    

    <div class="page-main-content">

        <div class="page-main-content-top">
            <header class="header-wrapper">

    <div class="header-content">
        <div class="left">
            <a class="logo-title" href="/">
                Yuxuan Wu
            </a>
        </div>

        <div class="right">
            <div class="pc">
                <ul class="menu-list">
                    
                        <li class="menu-item">
                            <a class=""
                               href="/"
                            >
                                HOME
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/archives"
                            >
                                ARCHIVES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/categories"
                            >
                                CATEGORIES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/tags"
                            >
                                TAGS
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/about"
                            >
                                ABOUT
                            </a>
                        </li>
                    
                    
                        <li class="menu-item search search-popup-trigger">
                            <i class="fas fa-search"></i>
                        </li>
                    
                </ul>
            </div>
            <div class="mobile">
                
                    <div class="icon-item search search-popup-trigger"><i class="fas fa-search"></i></div>
                
                <div class="icon-item menu-bar">
                    <div class="menu-bar-middle"></div>
                </div>
            </div>
        </div>
    </div>

    <div class="header-drawer">
        <ul class="drawer-menu-list">
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/">HOME</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/archives">ARCHIVES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/categories">CATEGORIES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/tags">TAGS</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/about">ABOUT</a>
                </li>
            
        </ul>
    </div>

    <div class="window-mask"></div>

</header>


        </div>

        <div class="page-main-content-middle">

            <div class="main-content normal-code-theme">

                
                    <div class="fade-in-down-animation">
    <div class="article-content-container">

        <div class="article-title">
            <span class="title-hover-animation">Data preparation for multi-instance learning in m6a sequence data</span>
        </div>

        
            <div class="article-header">
                <div class="avatar">
                    <img src="/images/avatar.svg">
                </div>
                <div class="info">
                    <div class="author">
                        <span class="name">Yuxuan Wu</span>
                        <span class="level">Lv4</span>
                    </div>
                    <div class="meta-info">
                        <div class="article-meta-info">
    <span class="article-date article-meta-item">
        <i class="fas fa-edit"></i> 2021-01-28 14:22:26
    </span>
    
        <span class="article-categories article-meta-item">
            <i class="fas fa-folder"></i>
            <ul>
                
                    <li>
                        <a href="/categories/FYP/">FYP</a>
                    </li>
                
            </ul>
        </span>
    
    
        <span class="article-tags article-meta-item">
            <i class="fas fa-tags"></i>
            <ul>
                
                    <li>
                        <a href="/tags/R/">R</a>
                    </li>
                
                    <li>
                        | <a href="/tags/RNA-modification/">RNA modification</a>
                    </li>
                
                    <li>
                        | <a href="/tags/Multi-instance-learning/">Multi-instance learning</a>
                    </li>
                
            </ul>
        </span>
    

    
    
        <span class="article-wordcount article-meta-item">
            <i class="fas fa-file-word"></i> <span>1.8k Words</span>
        </span>
    
    
        <span class="article-min2read article-meta-item">
            <i class="fas fa-clock"></i> <span>11 Mins</span>
        </span>
    
    
        <span class="article-pv article-meta-item">
            <i class="fas fa-eye"></i> <span id="busuanzi_value_page_pv"></span>
        </span>
    
</div>

                    </div>
                </div>
            </div>
        

        <div class="article-content markdown-body">
            <p>Sample preparation for the multi-instance learning.</p>
<p>The whole script was finished by R</p>
<h3 id="Load-the-necessary-libraries-and-import-the-data"><a href="#Load-the-necessary-libraries-and-import-the-data" class="headerlink" title="Load the necessary libraries and import the data"></a>Load the necessary libraries and import the data</h3><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br></pre></td><td class="code"><pre><span class="line">library(GenomicRanges)</span><br><span class="line">library(Biostrings)</span><br><span class="line">library(BSgenome.Hsapiens.UCSC.hg19)</span><br><span class="line">library(dplyr)</span><br><span class="line">library(permute)</span><br><span class="line">library(ggplot2)</span><br></pre></td></tr></table></figure>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line"># whether to use the parallel calculation</span><br><span class="line"># library(doParallel)</span><br><span class="line"># cl &lt;- makePSOCKcluster(5)</span><br><span class="line"># registerDoParallel(cl)</span><br></pre></td></tr></table></figure>

<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br></pre></td><td class="code"><pre><span class="line"># KSHV，RSV，WNV，ZIKV，hMPV</span><br><span class="line"># HCV, DENV, HIV-1 </span><br><span class="line">knitr::opts_knit$set(root.dir &#x3D; &#39;&#x2F;data&#x2F;Virus_m6A_driver&#39;)</span><br><span class="line">target_gene &lt;- &quot;HIV-1&quot;</span><br><span class="line"></span><br><span class="line">if (!dir.exists(paste0(&#39;&#x2F;home&#x2F;yuxuan&#x2F;FYP&#x2F;Virus_init_codes&#x2F;processed_data&#x2F;&#39;,target_gene)))&#123;</span><br><span class="line">  dir.create(paste0(&#39;&#x2F;home&#x2F;yuxuan&#x2F;FYP&#x2F;Virus_init_codes&#x2F;processed_data&#x2F;&#39;,target_gene))</span><br><span class="line">&#125;</span><br><span class="line">target_dir &lt;- paste0(&#39;&#x2F;home&#x2F;yuxuan&#x2F;FYP&#x2F;Virus_init_codes&#x2F;processed_data&#x2F;&#39;,target_gene,&quot;&#x2F;&quot;)</span><br></pre></td></tr></table></figure>
<h4 id="Sample-preparation"><a href="#Sample-preparation" class="headerlink" title="Sample preparation:"></a>Sample preparation:</h4><ul>
<li>In this case, HIV-1 virus would be selected as our target</li>
</ul>
<p><strong>Read the pos and neg samples</strong></p>
<ul>
<li>It is in Grange format, with standard names, ranges and strands. The other columns are meta-columns, storing some features of data.</li>
</ul>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">pos_dir &lt;- paste0(&#39;&#x2F;data&#x2F;Virus_m6A_driver&#x2F;&#39;,target_gene,&#39;_positive.rds&#39;)</span><br><span class="line">pos_data &lt;- readRDS(pos_dir)</span><br><span class="line">neg_dir &lt;- paste0(&#39;&#x2F;data&#x2F;Virus_m6A_driver&#x2F;&#39;,target_gene,&#39;_negative.rds&#39;)</span><br><span class="line">neg_data &lt;- readRDS(neg_dir)</span><br></pre></td></tr></table></figure>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">head(pos_data)</span><br></pre></td></tr></table></figure>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting@master/image-20210128092240985.42ls9b2xba00.png" alt="image-20210128092240985"></p>
<hr>
<h2 id="R-How-to-segment-the-sequence-data-for-multi-instance-learning"><a href="#R-How-to-segment-the-sequence-data-for-multi-instance-learning" class="headerlink" title="R: How to segment the sequence data for multi-instance learning"></a>R: How to segment the sequence data for multi-instance learning</h2><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">summary(width(pos_data))</span><br><span class="line">quantile(width(pos_data), 0.8)</span><br></pre></td></tr></table></figure>
<p>Min.  1st Qu.   Median    Mean        3rd Qu.    Max.<br> 51          76     101              1537         201         147584  </p>
<p>80%<br>422</p>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">summary(width(neg_data))</span><br></pre></td></tr></table></figure>
<p>   Min. 1st Qu.  Median    Mean  3rd Qu.    Max.<br>      5     146           205       205         264         405 </p>
<blockquote>
<p>We can see that the maximum length for the positive data could be 147,584, while the maximum length for the negative sample could be 405, which is far more close</p>
</blockquote>
<blockquote>
<p>The unequal length of the samples could lead to significant bias to the results;</p>
<p>We therefore should try to cut the positive sequence length to minimize the bias.</p>
<p>We notice that the 80% sample in this virus have a roughly similar width with the positive data, so in the following sample, I would still insist on this </p>
</blockquote>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line"># change the seqnames to negative+number (neg1,neg2...)</span><br><span class="line">names(neg_data) &#x3D; paste(&#39;neg&#39;, c(1:length(neg_data)), sep&#x3D;&#39;&#39;)</span><br><span class="line"></span><br><span class="line"># used to set difference to exclude the selected names  </span><br></pre></td></tr></table></figure>


<h4 id="Prepare-to-do-the-loop"><a href="#Prepare-to-do-the-loop" class="headerlink" title="Prepare to do the loop"></a>Prepare to do the loop</h4><hr>
<p><strong>Noted that</strong>:</p>
<ul>
<li><p>Considering the wide range of the peak width, which might lead to the calculation inefficiency, I therefore do not incorporate the padding technique to fill the gap by zero to a fixed length.  </p>
</li>
<li><p>But we need to ensure the length between positive and negative sample to be the same.</p>
</li>
<li><p>That is the reason I made a lot of effort to do the following loop, to obtain the different positive and negative samples, and their length are matched accordingly.</p>
</li>
</ul>
<h5 id="Prepare-the-processed-positive-data-valid-pos"><a href="#Prepare-the-processed-positive-data-valid-pos" class="headerlink" title="Prepare the processed positive data (valid_pos)"></a>Prepare the processed positive data (valid_pos)</h5><ol>
<li>Ensure the processed positive data sample are smaller than the maximum length of negative sample</li>
<li>Sort the value in decreasing order</li>
</ol>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">valid_pos &lt;- pos_data[width(pos_data) &lt;= <span class="built_in">max</span>(width(neg_data)),]</span><br><span class="line">sorted_valid_pos &lt;- valid_pos[order(width(valid_pos), decreasing=<span class="built_in">T</span>), ]</span><br></pre></td></tr></table></figure>
<hr>
<h5 id="Prepare-the-processed-negative-data"><a href="#Prepare-the-processed-negative-data" class="headerlink" title="Prepare the processed negative data"></a>Prepare the processed negative data</h5><figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">valid_neg &lt;- GRanges()</span><br><span class="line">exclude_names &lt;- <span class="built_in">c</span>() <span class="comment"># (save the already finished data name to this)</span></span><br></pre></td></tr></table></figure>


<hr>
<h4 id="Enter-the-for-loop"><a href="#Enter-the-for-loop" class="headerlink" title="Enter the for loop"></a>Enter the for loop</h4><p><strong>Goal:</strong> find the valid negative sample</p>
<ul>
<li>Obtain the unique width of sorted valid positive data, and use this as the loop variable ==w== </li>
<li>Find the temporary positive <strong>(tmp_pos)</strong> and negative samples <strong>(tmp_neg)</strong> based on the selected width ==w== in previous step </li>
<li>Return the selected number of tmp_pos and tmp_neg (<strong>num_pos/num_neg</strong>)</li>
</ul>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br></pre></td><td class="code"><pre><span class="line">unique_width_pos &lt;- unique(width(sorted_valid_pos))</span><br><span class="line"><span class="keyword">for</span> (w <span class="keyword">in</span> unique_width_pos) &#123;</span><br><span class="line">  tmp_pos &lt;- sorted_valid_pos[width(sorted_valid_pos)==w,]</span><br><span class="line">  tmp_neg &lt;- neg_data[width(neg_data)==w,]</span><br><span class="line">  num_pos &lt;- <span class="built_in">length</span>(tmp_pos)</span><br><span class="line">  num_neg &lt;- <span class="built_in">length</span>(tmp_neg)</span><br><span class="line">  <span class="comment">## ... (not_finished)</span></span><br></pre></td></tr></table></figure>


<hr>
<h4 id="Two-situation-after-the-selected-temporary-positive-and-negative-sample"><a href="#Two-situation-after-the-selected-temporary-positive-and-negative-sample" class="headerlink" title="Two situation after the selected temporary positive and negative sample"></a>Two situation after the selected temporary positive and negative sample</h4><ol>
<li><strong>Temporary negative sample &gt;&gt; tmp_pos (negative sample size is larger, resize the negative sample directly) (neg&gt;&gt;pos)</strong></li>
</ol>
<ul>
<li>Return the names of selected temporary negative sample (neg1, neg2…)</li>
<li>Use the returned name to compare with the names in exclude names </li>
<li>Sample the names selected and return the picked names based on the number of temporary samples</li>
<li>Add the picked names to the “exclude names”</li>
<li>The next for loop is simply to extract the picked name sample and append to the final valid_neg sample</li>
</ul>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line"><span class="keyword">if</span> (num_neg &gt;= num_pos) &#123;</span><br><span class="line">  valid_names &lt;- <span class="built_in">names</span>(tmp_neg)</span><br><span class="line">  valid_names &lt;- setdiff(valid_names, exclude_names)</span><br><span class="line">  picked_names &lt;- sample(valid_names, num_pos)</span><br><span class="line">  exclude_names &lt;- <span class="built_in">c</span>(exclude_names, picked_names)</span><br><span class="line">  <span class="keyword">for</span> (n <span class="keyword">in</span> picked_names) &#123;</span><br><span class="line">    valid_neg &lt;- append(valid_neg, tmp_neg[<span class="built_in">names</span>(tmp_neg)==n,])</span><br><span class="line">  &#125;</span><br></pre></td></tr></table></figure>


<ol start="2">
<li><strong>pos&gt;&gt;neg</strong><ul>
<li>We have to increase the number of negative data.</li>
<li>The negative data could be obtained via cutting other longer sequence</li>
</ul>
</li>
</ol>
<ul>
<li>Append all the current temporary negative sample to the final valid negative sample</li>
<li>Add to the exclude names, to update the exclude_names variable</li>
<li>Find the number of sample difference between the tmp_pos &amp; tmp_neg and assign to num_res</li>
<li>Find the original negative sample ahead (not tmp_neg), with the width larger than in this loop ==w== </li>
<li>Same with the previous step, check whether the name is in the exclude names and select the picked names and add to the exclude names</li>
<li><strong>the most important concepts are stated as follows (a for loop in picked names)</strong>:<ul>
<li>I need to find the longer negative sequence to cut down segments and attached to the temporary negative samples</li>
<li>Find the matched neg_data from the picked names called tmp_gr </li>
<li>Calculate the width difference (width_diff) between the tmp_gr from original he neg_data and ==w== in the loop.</li>
<li>Use sample function to randomly select a point, and consider this as the start and remove part of the sequence in the longer sequence</li>
<li>Similarly, to minimize the influence, we would also cut another part from the behind.</li>
</ul>
</li>
</ul>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br></pre></td><td class="code"><pre><span class="line">  &#125;else &#123;</span><br><span class="line">    valid_neg &lt;- append(valid_neg, tmp_neg)</span><br><span class="line">    exclude_names &lt;- c(exclude_names, names(tmp_neg))</span><br><span class="line">    num_res &lt;- num_pos - num_neg</span><br><span class="line">    valid_names &lt;- names(neg_data[width(neg_data) &gt; w,])</span><br><span class="line">    print(valid_names)</span><br><span class="line">    valid_names &lt;- setdiff(valid_names, exclude_names)</span><br><span class="line">    picked_names &lt;- sample(valid_names, num_res)</span><br><span class="line">    exclude_names &lt;- c(exclude_names, picked_names)</span><br><span class="line">    for (n in picked_names) &#123;</span><br><span class="line">      tmp_gr &lt;- neg_data[names(neg_data) &#x3D;&#x3D; n, ]</span><br><span class="line">      width_diff &lt;- width(tmp_gr) - w</span><br><span class="line">      start_remove &lt;- sample(c(1:width_diff), 1)</span><br><span class="line">      tmp_gr &lt;- resize(tmp_gr, width(tmp_gr) - start_remove, fix&#x3D;&#39;start&#39;)</span><br><span class="line">      tmp_gr &lt;- resize(tmp_gr, w, fix&#x3D;&#39;end&#39;)</span><br><span class="line">      names(tmp_gr) &lt;- n</span><br><span class="line">      valid_neg &lt;- append(valid_neg, tmp_gr)</span><br><span class="line">    &#125;</span><br><span class="line">  &#125;</span><br><span class="line">&#125;</span><br></pre></td></tr></table></figure>
<p>For now we finished the preparation of valid negative sample </p>
<hr>
<h3 id="Assign-the-labels-and-check-the-results"><a href="#Assign-the-labels-and-check-the-results" class="headerlink" title="Assign the labels and check the results"></a>Assign the labels and check the results</h3><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">pos_df &lt;- data.frame(&#39;width&#39;&#x3D;width(valid_pos), </span><br><span class="line">                     &#39;label&#39;&#x3D;rep(T, length(valid_pos)))</span><br><span class="line">summary(pos_df)</span><br></pre></td></tr></table></figure>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">neg_df &lt;- data.frame(&#39;width&#39;&#x3D;width(valid_neg), </span><br><span class="line">                     &#39;label&#39;&#x3D;rep(F, length(valid_neg)))</span><br><span class="line">summary(neg_df)</span><br></pre></td></tr></table></figure>
<figure class="highlight plain"><figcaption><span>echo</span></figcaption><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">ggplot(rbind(pos_df, neg_df), aes(x&#x3D;(width), color&#x3D;label, fill&#x3D;label)) +</span><br><span class="line">  geom_histogram(aes(y&#x3D;..density..), color&#x3D;&#39;black&#39;, fill&#x3D;&#39;white&#39;, bins&#x3D;30) +</span><br><span class="line">  geom_density(alpha&#x3D;0.4) + labs(x&#x3D;&#39;Peak Width&#39;, y&#x3D;&#39;Density&#39;) + ggtitle(&#39;Peak Width Distribution for 5hmC data&#39;)</span><br><span class="line"></span><br></pre></td></tr></table></figure>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting@master/image-20210128140706795.6ol0ajyw9640.png" alt="image-20210128140706795"></p>
<p>The correct way here is TRUE and False sample should be the same, and we can see the distribution of sample length</p>
<h3 id="Save-the-data"><a href="#Save-the-data" class="headerlink" title="Save the data"></a>Save the data</h3><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">saveRDS(valid_pos, paste(target_dir, &#39;picked_pos_gr.rds&#39;, sep&#x3D;&#39;&#39;))</span><br><span class="line">saveRDS(valid_neg, paste(target_dir, &#39;picked_neg_gr.rds&#39;, sep&#x3D;&#39;&#39;))</span><br></pre></td></tr></table></figure>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">pos_data &lt;- readRDS(paste(target_dir, &#39;picked_pos_gr.rds&#39;, sep&#x3D;&#39;&#39;))</span><br><span class="line">neg_data &lt;- readRDS(paste(target_dir, &#39;picked_neg_gr.rds&#39;, sep&#x3D;&#39;&#39;))</span><br><span class="line">pos_data$label &#x3D; rep(T, length(pos_data))</span><br><span class="line">neg_data$label &#x3D; rep(F, length(pos_data))</span><br></pre></td></tr></table></figure>
<h3 id="Split-the-data-8-1-1"><a href="#Split-the-data-8-1-1" class="headerlink" title="Split the data (8:1:1)"></a>Split the data (8:1:1)</h3><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br></pre></td><td class="code"><pre><span class="line">pos_data &lt;- pos_data[order(width(pos_data)),]</span><br><span class="line">neg_data &lt;- neg_data[order(width(neg_data)),]</span><br><span class="line">split_idx &lt;- sample(1:3, size&#x3D;length(pos_data), prob&#x3D;c(0.8, 0.1, 0.1), replace&#x3D;T)</span><br><span class="line">train_pos &lt;- pos_data[split_idx&#x3D;&#x3D;1,]</span><br><span class="line">train_neg &lt;- neg_data[split_idx&#x3D;&#x3D;1,]</span><br><span class="line">valid_pos &lt;- pos_data[split_idx&#x3D;&#x3D;2,]</span><br><span class="line">valid_neg &lt;- neg_data[split_idx&#x3D;&#x3D;2,]</span><br><span class="line">test_pos &lt;- pos_data[split_idx&#x3D;&#x3D;3,]</span><br><span class="line">test_neg &lt;- neg_data[split_idx&#x3D;&#x3D;3,]</span><br></pre></td></tr></table></figure>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">train_data &lt;- append(train_pos, train_neg)</span><br><span class="line">valid_data &lt;- append(valid_pos, valid_neg)</span><br><span class="line">test_data &lt;- append(test_pos, test_neg)</span><br></pre></td></tr></table></figure>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">saveRDS(train_data, paste(target_dir, &#39;train_data.rds&#39;, sep&#x3D;&#39;&#39;))</span><br><span class="line">saveRDS(valid_data, paste(target_dir, &#39;valid_data.rds&#39;, sep&#x3D;&#39;&#39;))</span><br><span class="line">saveRDS(test_data, paste(target_dir, &#39;test_data.rds&#39;, sep&#x3D;&#39;&#39;))</span><br></pre></td></tr></table></figure>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">train_data &lt;- readRDS(paste(target_dir, &#39;train_data.rds&#39;, sep&#x3D;&#39;&#39;))</span><br><span class="line">valid_data &lt;- readRDS(paste(target_dir, &#39;valid_data.rds&#39;, sep&#x3D;&#39;&#39;))</span><br><span class="line">test_data &lt;- readRDS(paste(target_dir, &#39;test_data.rds&#39;, sep&#x3D;&#39;&#39;))</span><br></pre></td></tr></table></figure>
<h3 id="String-to-token-function-to-transform-the-AGCT-to-token-format-1-2-3-4"><a href="#String-to-token-function-to-transform-the-AGCT-to-token-format-1-2-3-4" class="headerlink" title="String to token function to transform the AGCT to token format (1,2,3,4)"></a>String to token function to transform the AGCT to token format (1,2,3,4)</h3><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br></pre></td><td class="code"><pre><span class="line">str2token &lt;- function(str_sequence) &#123;</span><br><span class="line">  splitted_seq &lt;- strsplit(str_sequence, &#39;&#39;)</span><br><span class="line">  token_seq &lt;- vector(mode&#x3D;&#39;list&#39;, length&#x3D;length(splitted_seq))</span><br><span class="line">  </span><br><span class="line">  for (idx in c(1:length(splitted_seq)))&#123;</span><br><span class="line">    token_seq[idx] &#x3D; list(as.integer(unlist(splitted_seq[idx]) &#x3D;&#x3D; &#39;A&#39;) +</span><br><span class="line">      2 * as.integer(unlist(splitted_seq[idx]) &#x3D;&#x3D; &#39;C&#39;) +</span><br><span class="line">      3 * as.integer(unlist(splitted_seq[idx]) &#x3D;&#x3D; &#39;G&#39;) +</span><br><span class="line">      4 * as.integer(unlist(splitted_seq[idx]) &#x3D;&#x3D; &#39;U&#39;))</span><br><span class="line">  &#125;</span><br><span class="line">  return(token_seq)</span><br><span class="line">&#125;</span><br></pre></td></tr></table></figure>
<h3 id="Transform-the-Grange-format-to-the-sequence-format"><a href="#Transform-the-Grange-format-to-the-sequence-format" class="headerlink" title="Transform the Grange format to the sequence format"></a>Transform the Grange format to the sequence format</h3><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br></pre></td><td class="code"><pre><span class="line">train_label &lt;- train_data$label &#x3D;&#x3D; T</span><br><span class="line">train_seq &lt;- RNAStringSet(DNAStringSet(Views(Hsapiens, train_data)))</span><br><span class="line">train_seq &lt;- as.character(train_seq)</span><br><span class="line">train_seq &lt;- str2token(train_seq)</span><br><span class="line">train_shuffle_idx &lt;- shuffle(length(train_label))</span><br><span class="line">train_seq &lt;- train_seq[train_shuffle_idx]</span><br><span class="line">train_label &lt;- train_label[train_shuffle_idx]</span><br></pre></td></tr></table></figure>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br></pre></td><td class="code"><pre><span class="line">valid_label &lt;- valid_data$label &#x3D;&#x3D; T</span><br><span class="line">valid_seq &lt;- RNAStringSet(DNAStringSet(Views(Hsapiens, valid_data)))</span><br><span class="line">valid_seq &lt;- as.character(valid_seq)</span><br><span class="line">valid_seq &lt;- str2token(valid_seq)</span><br><span class="line">valid_shuffle_idx &lt;- shuffle(length(valid_label))</span><br><span class="line">valid_seq &lt;- valid_seq[valid_shuffle_idx]</span><br><span class="line">valid_label &lt;- valid_label[valid_shuffle_idx]</span><br></pre></td></tr></table></figure>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line">## str2token transfer the character to numerical format</span><br><span class="line">test_label &lt;- test_data$label &#x3D;&#x3D; T</span><br><span class="line">test_seq &lt;- RNAStringSet(DNAStringSet(Views(Hsapiens, test_data)))</span><br><span class="line">test_seq &lt;- as.character(test_seq)</span><br><span class="line">test_seq &lt;- str2token(test_seq)</span><br><span class="line">test_shuffle_idx &lt;- shuffle(length(test_label))</span><br><span class="line">test_seq &lt;- test_seq[test_shuffle_idx]</span><br><span class="line">test_label &lt;- test_label[test_shuffle_idx]</span><br></pre></td></tr></table></figure>
<h3 id="Save-with-the-rds-format"><a href="#Save-with-the-rds-format" class="headerlink" title="Save with the rds format"></a>Save with the rds format</h3><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br></pre></td><td class="code"><pre><span class="line">saveRDS(train_seq, paste(target_dir, &#39;train_seq.rds&#39;, sep&#x3D;&#39;&#39;))</span><br><span class="line">saveRDS(valid_seq, paste(target_dir, &#39;valid_seq.rds&#39;, sep&#x3D;&#39;&#39;))</span><br><span class="line">saveRDS(test_seq, paste(target_dir, &#39;test_seq.rds&#39;, sep&#x3D;&#39;&#39;))</span><br><span class="line"></span><br><span class="line">saveRDS(train_label, paste(target_dir, &#39;train_label.rds&#39;, sep&#x3D;&#39;&#39;))</span><br><span class="line">saveRDS(valid_label, paste(target_dir, &#39;valid_label.rds&#39;, sep&#x3D;&#39;&#39;))</span><br><span class="line">saveRDS(test_label, paste(target_dir, &#39;test_label.rds&#39;, sep&#x3D;&#39;&#39;))</span><br></pre></td></tr></table></figure>


<h3 id="Some-functions"><a href="#Some-functions" class="headerlink" title="Some functions:"></a>Some functions:</h3><p>- <code>Views()</code>: for extracting sequences underneath GRanges on <code>BSgenome</code> object.</p>
<p>- <code>DNAStringSet()</code>: for the conversion of the BSgenome views into a set of DNA strings so that we could calculate their sequence content.</p>
<p>- <code>letterFrequency()</code>: for the calculation of the proportion / frequency of nucleotides within XStringSet.</p>
<p>- <code>setdiff(x,y)</code>: find the different element in vector x and vector y, but they could only retain the vector in X </p>
<p>- <code>sample(x, n)</code>: X is the whole samples, and n is the number of sample to be selected </p>
<p>- <code>resize(x, width, fix=&quot;start&quot;, use.names=TRUE, ...)</code>: the width of the resized or flanking regions; fix= is an anchor </p>

        </div>

        
            <div class="post-copyright-info">
                <div class="article-copyright-info-container">
    <ul>
        <li>Post title：Data preparation for multi-instance learning in m6a sequence data</li>
        <li>Post author：Yuxuan Wu</li>
        <li>Create time：2021-01-28 14:22:26</li>
        <li>
            Post link：yuxuanwu17.github.io2021/01/28/multi-instance-learning-in-m6a-sequence-data/
        </li>
        <li>
            Copyright Notice：All articles in this blog are licensed under <a class="license" target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by-nc-sa/4.0/deed.zh">BY-NC-SA</a> unless stating additionally.
        </li>
    </ul>
</div>

            </div>
        

        
            <div class="article-nav">
                
                    <div class="article-prev">
                        <a class="prev"
                           rel="prev"
                           href="/2021/01/28/matlab_links/"
                        >
                            <span class="left arrow-icon flex-center">
                              <i class="fas fa-chevron-left"></i>
                            </span>
                            <span class="title flex-center">
                                <span class="post-nav-title-item">matlab相关的知识</span>
                                <span class="post-nav-item">Prev posts</span>
                            </span>
                        </a>
                    </div>
                
                
                    <div class="article-next">
                        <a class="next"
                           rel="next"
                           href="/2021/01/27/meta-analysis/"
                        >
                            <span class="title flex-center">
                                <span class="post-nav-title-item">Meta analysis for knockdown of METTL3 or METTL14 affecting N6-methyladenosine methylation level</span>
                                <span class="post-nav-item">Next posts</span>
                            </span>
                            <span class="right arrow-icon flex-center">
                              <i class="fas fa-chevron-right"></i>
                            </span>
                        </a>
                    </div>
                
            </div>
        

        
    </div>
</div>


                
            </div>

        </div>

        <div class="page-main-content-bottom">
            <footer class="footer">
    <div class="info-container">
        <div class="copyright-info info-item">
            &copy;
            
              <span>2020</span> -
            
            2021 <i class="fas fa-heart icon-animate"></i> <a href="/">Yuxuan Wu</a>
        </div>
        
            <script async  src="//busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
            <div class="website-count info-item">
                
                    <span id="busuanzi_container_site_uv">
                        Visitor Count <span id="busuanzi_value_site_uv"></span>&ensp;
                    </span>
                
                
                    <span id="busuanzi_container_site_pv">
                        Totalview <span id="busuanzi_value_site_pv"></span>
                    </span>
                
            </div>
        
        <div class="theme-info info-item">
            Powered by <a target="_blank" href="https://hexo.io">Hexo</a> | Theme <a class="theme-version" target="_blank" href="https://github.com/XPoet/hexo-theme-keep">Keep v3.3.1</a>
        </div>
    </div>
</footer>

        </div>
    </div>

    
        <div class="post-tools">
            <div class="post-tools-container">
    <ul class="tools-list">
        <!-- TOC aside toggle -->
        
            <li class="tools-item page-aside-toggle">
                <i class="fas fa-outdent"></i>
            </li>
        

        <!-- go comment -->
        
    </ul>
</div>

        </div>
    

    <div class="right-bottom-side-tools">
        <div class="side-tools-container">
    <ul class="side-tools-list">
        <li class="tools-item tool-font-adjust-plus flex-center">
            <i class="fas fa-search-plus"></i>
        </li>

        <li class="tools-item tool-font-adjust-minus flex-center">
            <i class="fas fa-search-minus"></i>
        </li>

        <li class="tools-item tool-expand-width flex-center">
            <i class="fas fa-arrows-alt-h"></i>
        </li>

        <li class="tools-item tool-dark-light-toggle flex-center">
            <i class="fas fa-moon"></i>
        </li>

        <!-- rss -->
        

        

        <li class="tools-item tool-scroll-to-bottom flex-center">
            <i class="fas fa-arrow-down"></i>
        </li>
    </ul>

    <ul class="exposed-tools-list">
        <li class="tools-item tool-toggle-show flex-center">
            <i class="fas fa-cog fa-spin"></i>
        </li>
        
            <li class="tools-item tool-scroll-to-top flex-center">
                <i class="arrow-up fas fa-arrow-up"></i>
                <span class="percent"></span>
            </li>
        
    </ul>
</div>

    </div>

    
        <aside class="page-aside">
            <div class="post-toc-wrap">
    <div class="post-toc">
        <ol class="nav"><li class="nav-item nav-level-3"><a class="nav-link" href="#Load-the-necessary-libraries-and-import-the-data"><span class="nav-number">1.</span> <span class="nav-text">Load the necessary libraries and import the data</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#Sample-preparation"><span class="nav-number">1.1.</span> <span class="nav-text">Sample preparation:</span></a></li></ol></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#R-How-to-segment-the-sequence-data-for-multi-instance-learning"><span class="nav-number"></span> <span class="nav-text">R: How to segment the sequence data for multi-instance learning</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#Prepare-to-do-the-loop"><span class="nav-number">0.1.</span> <span class="nav-text">Prepare to do the loop</span></a><ol class="nav-child"><li class="nav-item nav-level-5"><a class="nav-link" href="#Prepare-the-processed-positive-data-valid-pos"><span class="nav-number">0.1.1.</span> <span class="nav-text">Prepare the processed positive data (valid_pos)</span></a></li><li class="nav-item nav-level-5"><a class="nav-link" href="#Prepare-the-processed-negative-data"><span class="nav-number">0.1.2.</span> <span class="nav-text">Prepare the processed negative data</span></a></li></ol></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Enter-the-for-loop"><span class="nav-number">0.2.</span> <span class="nav-text">Enter the for loop</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Two-situation-after-the-selected-temporary-positive-and-negative-sample"><span class="nav-number">0.3.</span> <span class="nav-text">Two situation after the selected temporary positive and negative sample</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Assign-the-labels-and-check-the-results"><span class="nav-number">1.</span> <span class="nav-text">Assign the labels and check the results</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Save-the-data"><span class="nav-number">2.</span> <span class="nav-text">Save the data</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Split-the-data-8-1-1"><span class="nav-number">3.</span> <span class="nav-text">Split the data (8:1:1)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#String-to-token-function-to-transform-the-AGCT-to-token-format-1-2-3-4"><span class="nav-number">4.</span> <span class="nav-text">String to token function to transform the AGCT to token format (1,2,3,4)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Transform-the-Grange-format-to-the-sequence-format"><span class="nav-number">5.</span> <span class="nav-text">Transform the Grange format to the sequence format</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Save-with-the-rds-format"><span class="nav-number">6.</span> <span class="nav-text">Save with the rds format</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Some-functions"><span class="nav-number">7.</span> <span class="nav-text">Some functions:</span></a></li></ol>
    </div>
</div>
        </aside>
    

    <div class="image-viewer-container">
    <img src="">
</div>


    
        <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
          <span class="search-input-field-pre">
            <i class="fas fa-keyboard"></i>
          </span>
            <div class="search-input-container">
                <input autocomplete="off"
                       autocorrect="off"
                       autocapitalize="off"
                       placeholder="Search..."
                       spellcheck="false"
                       type="search"
                       class="search-input"
                >
            </div>
            <span class="popup-btn-close">
                <i class="fas fa-times"></i>
            </span>
        </div>
        <div id="search-result">
            <div id="no-result">
                <i class="fas fa-spinner fa-pulse fa-5x fa-fw"></i>
            </div>
        </div>
    </div>
</div>

    

</main>




<script src="/js/utils.js"></script>

<script src="/js/main.js"></script>

<script src="/js/header-shrink.js"></script>

<script src="/js/back2top.js"></script>

<script src="/js/dark-light-toggle.js"></script>



    
<script src="/js/local-search.js"></script>




    
<script src="/js/code-copy.js"></script>





<div class="post-scripts">
    
        
<script src="/js/left-side-toggle.js"></script>

<script src="/js/libs/anime.min.js"></script>

<script src="/js/toc.js"></script>

    
</div>



</body>
</html>
