<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="keywords" content="Hexo Theme Keep">
    <meta name="description" content="Welcome to my personal blogs">
    <meta name="author" content="Yuxuan Wu">
    
    <title>
        
            BIO312 Wang Mu&#39;s CW |
        
        Yuxuan Wu
    </title>
    <link rel="shortcut icon" href="/images/logo.svg">
    
<link rel="stylesheet" href="/css/style.css">

    
<link rel="stylesheet" href="/css/font-awesome.min.css">

    <script id="hexo-configurations">
    let KEEP = window.KEEP || {};
    KEEP.hexo_config = {"hostname":"example.com","root":"/","language":"en","path":"search.xml"};
    KEEP.theme_config = {"toc":{"enable":true,"number":true,"expand_all":false},"style":{"primary_color":"#0066CC","avatar":"/images/avatar.svg","favicon":"/images/logo.svg","article_img_align":"left","left_side_width":"260px","content_max_width":"920px","hover":{"shadow":true,"scale":true},"first_screen":{"enable":true,"background_img":"/images/bg.svg","description":"Keep coding, Keep hungry."},"scroll":{"progress_bar":{"enable":true},"percent":{"enable":true}}},"local_search":{"enable":true,"preload":true},"code_copy":{"enable":true,"style":"default"},"pjax":{"enable":false},"lazyload":{"enable":false},"version":"3.3.1"};
    KEEP.language_ago = {"second":"%s seconds ago","minute":"%s minutes ago","hour":"%s hours ago","day":"%s days age","week":"%s weeks age","month":"%s months age","year":"%s years age"};
  </script>
<!-- hexo injector head_end start -->
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css">

<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/hexo-math@4.0.0/dist/style.css">
<!-- hexo injector head_end end --><meta name="generator" content="Hexo 5.3.0"></head>


<body>
<div class="progress-bar-container">
    
        <span class="scroll-progress-bar"></span>
    

    
</div>


<main class="page-container">

    

    <div class="page-main-content">

        <div class="page-main-content-top">
            <header class="header-wrapper">

    <div class="header-content">
        <div class="left">
            <a class="logo-title" href="/">
                Yuxuan Wu
            </a>
        </div>

        <div class="right">
            <div class="pc">
                <ul class="menu-list">
                    
                        <li class="menu-item">
                            <a class=""
                               href="/"
                            >
                                HOME
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/archives"
                            >
                                ARCHIVES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/categories"
                            >
                                CATEGORIES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/tags"
                            >
                                TAGS
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/about"
                            >
                                ABOUT
                            </a>
                        </li>
                    
                    
                        <li class="menu-item search search-popup-trigger">
                            <i class="fas fa-search"></i>
                        </li>
                    
                </ul>
            </div>
            <div class="mobile">
                
                    <div class="icon-item search search-popup-trigger"><i class="fas fa-search"></i></div>
                
                <div class="icon-item menu-bar">
                    <div class="menu-bar-middle"></div>
                </div>
            </div>
        </div>
    </div>

    <div class="header-drawer">
        <ul class="drawer-menu-list">
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/">HOME</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/archives">ARCHIVES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/categories">CATEGORIES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/tags">TAGS</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/about">ABOUT</a>
                </li>
            
        </ul>
    </div>

    <div class="window-mask"></div>

</header>


        </div>

        <div class="page-main-content-middle">

            <div class="main-content normal-code-theme">

                
                    <div class="fade-in-down-animation">
    <div class="article-content-container">

        <div class="article-title">
            <span class="title-hover-animation">BIO312 Wang Mu&#39;s CW</span>
        </div>

        
            <div class="article-header">
                <div class="avatar">
                    <img src="/images/avatar.svg">
                </div>
                <div class="info">
                    <div class="author">
                        <span class="name">Yuxuan Wu</span>
                        <span class="level">Lv13</span>
                    </div>
                    <div class="meta-info">
                        <div class="article-meta-info">
    <span class="article-date article-meta-item">
        <i class="fas fa-edit"></i> 2021-03-10 05:50:03
    </span>
    
        <span class="article-categories article-meta-item">
            <i class="fas fa-folder"></i>
            <ul>
                
                    <li>
                        <a href="/categories/Biology/">Biology</a>
                    </li>
                
            </ul>
        </span>
    
    
        <span class="article-tags article-meta-item">
            <i class="fas fa-tags"></i>
            <ul>
                
                    <li>
                        <a href="/tags/notes/">notes</a>
                    </li>
                
            </ul>
        </span>
    

    
    
        <span class="article-wordcount article-meta-item">
            <i class="fas fa-file-word"></i> <span>2.6k Words</span>
        </span>
    
    
        <span class="article-min2read article-meta-item">
            <i class="fas fa-clock"></i> <span>15 Mins</span>
        </span>
    
    
        <span class="article-pv article-meta-item">
            <i class="fas fa-eye"></i> <span id="busuanzi_value_page_pv"></span>
        </span>
    
</div>

                    </div>
                </div>
            </div>
        

        <div class="article-content markdown-body">
            <ul>
<li><input disabled="" type="checkbox"> 研究tutorial2 的Tandem MS-Sequence Confirmation<ul>
<li><input disabled="" type="checkbox"> 看视频</li>
</ul>
</li>
<li><input disabled="" type="checkbox"> 还是需要研究Mass spectrometry</li>
</ul>
<h2 id="Proteomics-workflow"><a href="#Proteomics-workflow" class="headerlink" title="Proteomics workflow"></a>Proteomics workflow</h2><p>1.Extract proteins from sample</p>
<p>2.Separate proteins (various techniques)</p>
<p>3.Digest proteins into <strong>peptides</strong></p>
<p>4.Separate peptides (various techniques)</p>
<p>5.Analyse peptides by mass spectrometry (MS)</p>
<p> e.g. known masses of peptides or masses of peptide fragments</p>
<p>6.Database search to identify peptides (and hence protein)</p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210510213403.png" alt="image-20210510213403656"></p>
<p><strong>Proteomics</strong></p>
<ol>
<li>Complex mixtures</li>
<li>Partial sequence analysis</li>
<li>Emphasis in identification by database matching</li>
<li>Systems biology</li>
<li><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210511154351.png" alt="image-20210511154351114"></li>
</ol>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210510213558.png" alt="image-20210510213558015"></p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210510213704.png" alt="image-20210510213704538"></p>
<h2 id="Mass-Spectrometry"><a href="#Mass-Spectrometry" class="headerlink" title="Mass Spectrometry"></a>Mass Spectrometry</h2><p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210511155353.png" alt="image-20210511155353628"></p>
<h3 id="Mass-Spectrometry-based-Protein-Identification"><a href="#Mass-Spectrometry-based-Protein-Identification" class="headerlink" title="Mass Spectrometry-based Protein Identification"></a>Mass Spectrometry-based Protein Identification</h3><ul>
<li>A mass spectrum of the resulting digest products produces a “peptide map”or a “peptide fingerprint”.</li>
</ul>
<blockquote>
<p>The measured masses can be compared to theoretical peptide maps derived from database sequences for identification. There are a few choices of mass analysis that can be selected from this point, depending on available instrumentation and other factors. The resulting peptide fragments can be subjected to <code>MALDI-MS</code> or ESI-MSanalysis.</p>
</blockquote>
<ul>
<li>For example, a small aliquot of the digest solution can be directly analyzed by MALDI-MS to obtain a peptide map. The resulting sequence coverage (relative to the entire protein sequence) displayed from the total number of tryptic peptides observed in the MALDI mass spectrum can be quite high, i.e., greater than 80% of the sequence, although it can vary considerably depending on the protein, sample amount, etc. The measured molecular weights of the peptide fragments along with the specificity of the enzyme employed can be searched and compared against protein sequence databasesusing a number of searching algorithms available either commercially or on the Internet.</li>
</ul>
<h2 id="cw1"><a href="#cw1" class="headerlink" title="cw1"></a>cw1</h2><h3 id="Task1"><a href="#Task1" class="headerlink" title="Task1"></a>Task1</h3><p>What is a proteome and how is the link between an organism’s genome and a proteome? (10 marks)</p>
<blockquote>
<p>Proteome represents the proteins catalogue of all proteins in a cell tissue or body fluid. Transcriptome, which is the RNA catalogue of all mRNA expressed in a cell or tissue,  is the link between an organisms genome and proteome.</p>
</blockquote>
<h3 id="Task2"><a href="#Task2" class="headerlink" title="Task2"></a>Task2</h3><p>An investigator brought a purified peptide to the Proteomics Core Lab for <code>MALDI-MS analysis</code> for quality control purpose. The peptide was dissolved in a buffer containing 10mM Tris-HCl, pH 7.5, 250mM NaCl, and trace amount of NP40. The theoretical molecular weight of the peptide should be 1571.5Da. However, the MALDI-MS spectrum showed two major peaks, one at m/z=1572.6, and another at m/z=1594.6. <em>Did the investigator have the right peptide</em>? If yes, is the peptide relatively pure? Explain why. (10 marks)</p>
<p>Theoretical: 1571</p>
<p>Results: 1572.6; 1594.6</p>
<blockquote>
<p>The investigator could have the right, pure peptide, but require further sequence analysis confirmation. The possible reasons for the situation could from two aspects. One is the difference in digestion efficiency, some peptide may suffer from different lysing rate and therefore the different peak; another one could be the situation that some peptide might be ionized during MALDI process, leading to the different major peaks. Based on the previous analysis, we could not jump to the conclusion, further analysis would be required. </p>
</blockquote>
<p><strong>MALDI-MS</strong>:</p>
<p>==Matrix-Assisted Laser Desorption/Ionization (MALDI)==</p>
<blockquote>
<p>可能的答案：</p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210511161610.png" alt="image-20210511161610177"></p>
</blockquote>
<p> <img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210511160231.png" alt="image-20210511160231258"></p>
<p> <img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210511160342.png" alt="image-20210511160342659"></p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210511160929.png" alt="image-20210511160929119"></p>
<h3 id="Task3"><a href="#Task3" class="headerlink" title="Task3"></a>Task3</h3><p>BRCA1 is a breast cancer related protein. However, in many breast cancer <code>biomarker discovery studies</code>, BRCA1 cannot be identified by general shotgun <code>LC/MS-based proteomics approaches</code> which use <code>RP-LC</code> to separate tryptic peptides. Its abundance in cancer cells should be sufficient enough to be detected by the currently available mass spectrometers such as Orbitrap FusionTM. <strong><em>Please describe the most likely reason for the failure of its detection (please keep your answers within one-page limit). (20 marks)</em></strong></p>
<p><strong><em>Why LC/MS-based proteomics approaches which use RP-LC to separate tryptic peptides failed?</em></strong></p>
<p>==RP-LC:== reversed phase</p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210511165137.png" alt="image-20210511165137759"></p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210511165252.png" alt="image-20210511165252077"></p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210511170237.png" alt="image-20210511170237116"></p>
<blockquote>
<p>Bottom-Up: sequence tryptic fragments of larger proteins</p>
<ul>
<li>Easier to analyze larger or hydrophobic proteins</li>
<li>Potential for MALDI MS/MS</li>
<li>Analysis of proteins from their derived peptides in complex biological systems</li>
<li>LC-MS/MS (generally OrbitrapMS) is the technique of choice</li>
</ul>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210513133608.png" alt="image-20210513133608349"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210513133724.png" alt="image-20210513133724701"></p>
<p>网络上的消息</p>
<blockquote>
<p>Phowever, identification of clinical biomarkers remains one of the most challenging applications [33]. Current biomarkers or biomarker candidates struggle with <strong>limited reliability and proper validation as well as with limited sensitivity and specificity</strong> [32].</p>
<p>There are several important factors to be considered in the search for a biomarker [54]. First of all, </p>
<ul>
<li><p>tissues as samples are not easily accessible, and, as they are composed of different cells, they are also very heterogeneous, an issue being addressed with labour intensive laser capture microdissection.</p>
</li>
<li><p>its complexity, time consumption, the need for educating the personnel, the costs associated with purchase of equipment, and last but not least, the availability of tissue samples. In clinical diagnosis there has always been a tendency to use most easily accessible specimens for diagnostic procedures, such as blood, saliva, urine, and faeces. </p>
</li>
<li><p>dynamic complexity and are dependent on patient and environmental characteristics. </p>
</li>
<li><p>collecting the proper number of patients for biomarker validation can also pose a problem. </p>
</li>
<li><p>And finally, it is becoming obvious that a single biomarker is not enough for accurate screening or diagnostic purposes; </p>
</li>
<li><p>rather a panel of proteins will be necessary</p>
</li>
</ul>
</blockquote>
<blockquote>
<p>Global LC-MS/MS-based proteomics suffers inherent problems of:</p>
<p><strong>quantification accuracy</strong> and <strong>reproducibility</strong> (i.e., missing data in samples with low-abundance proteins that fail to be identified) in large-scale studies due to the stochastic sampling nature of shotgun MS/MS</p>
</blockquote>
<blockquote>
<p><strong>Reversed-Phase Liquid Chromatography</strong></p>
<p>Reversed-Phase Liquid Chromatography (RPLC) is one of the High Pressure Liquid Chromatography or “HPLC” approaches, which are now for commercial reasons often alluded to as High <em>Performance</em> Liquid Chromatography. </p>
<p>RPLC is especially useful in the separation and purification of proteins. To that effect columns are used which are packed with, e.g., porous glass beads that are made <strong>hydrophobic</strong> by treating them with octadecyl (C18) groups.</p>
<p> In water such C18-coated beads bind hydrophobically to all but the most hydrophilic proteins.</p>
<p> For the subsequent elution of the bound proteins one <em>decreases</em> the hydrophobizing capability of the aqueous medium by the admixture of <strong>water-miscible organic solvents with much lower AB values</strong> than the 51 mJ/m2 of water, such as methanol, ethanol, ethylene glycol, or acetonitrile.</p>
</blockquote>
<p>正儿八经的answer</p>
<blockquote>
<p>RP-LC, the abbreviation of Reversed-Phase Liquid Chromatography, is especially useful in the separation and purification of proteins. The hydrophobic porous beads with octadecyl (C18) groups could bind hydrophobically to all but the most hydrophilic proteins, enabling the separation of proteins. </p>
<p>As mentioned in the questions that BRCA1could not be identified by general shotgun LC/MS-based proteomics approaches which use RP-LC to separate tryptic peptides, but work in the mass spectrometers. Therefore, I could hypothesis that the most likely reason could related to the hydrophobicity of BRCA1 biomarker.  </p>
<p>Current biomarkers identification struggled with limited reliability and proper validation as well as with limited sensitivity and specificity. Since biomarkers are highly dependent on environmental characteristics, which could be easily influenced by surrounding environment</p>
</blockquote>
<hr>
<p>建议使用的table</p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210512213855.png" alt="image-20210512213855882"></p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210512214022.png" alt="image-20210512214022555"></p>
<p>Peak: molecular weight to the ions</p>
<blockquote>
<p>为什么会有peak 高低</p>
</blockquote>
<ol>
<li>Digestion efficiency for some peptide lysing</li>
<li>Ionize of each protein </li>
</ol>
<p>可能是问题2的答案</p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210512220229.png" alt="image-20210512220229106"></p>
<p>y ions 从右往左数，需要先加上一个18，最后加上1</p>
<p>b ions 从左往右数，需要先加上1，最后加上18</p>
<h2 id="cw2"><a href="#cw2" class="headerlink" title="cw2"></a>cw2</h2><ol>
<li>A redox protein can form many <strong>disulfide bonds</strong> that are regulatory to its activity. Identification of the <strong>disulfide bonds</strong> and cysteine residues involved in forming them could be a challenging task. <code>Mutational analysis</code> (e.g. changing cys to ala) has been the most investigators’ choice. But this method can be very labor-intensive. With the development in ==mass spectrometry==, a mass spec-based disulfide bond determination has become more and more popular. For protein identification using MS, disulfide bonds are normally reduced first and then alkylated (to prevent reforming of disulfide bonds). <strong>However, this procedure will not allow for identification of the cysteines involved in disulfide bond formation.</strong> Propose a method to <code>identify the cysteines involved in disulfide bond </code>formation using <code>mass spec</code> as the analytical tool. (20 marks)</li>
</ol>
<hr>
<ul>
<li>Mass spectrometry </li>
<li>Identify the cysteines involved in disulfide bond </li>
</ul>
<hr>
<blockquote>
<p>For example, simple isotope-coded affinity tag (ICAT)-based approaches (<a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4076987/#B33" >33<i class="fas fa-external-link-alt"></i></a>, <a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4076987/#B91" >91<i class="fas fa-external-link-alt"></i></a>) that pinpoint cysteinyl residues within complex (impure) protein samples are based on splitting a protein sample and labeling half with a light sulfhydryl-reactive ICAT reagent and the other half with a heavy, stable isotope-labeled sulfhydryl-reactive ICAT reagent, mixing, proteolytically digesting, and then analyzing by mass spectrometry. This results in readily identifiable <em>pairs</em> of mass spectral peaks in single-stage mass spectra that are separated by the mass difference between the light and heavy ICAT tags. Mass mapping and, if necessary, analysis of the tagged peptides by tandem mass spectrometry (MS/MS) can reveal the identity of the labeled cysteine residue(s). (More complex ICAT-based schemes for relative quantification are described next.)</p>
</blockquote>
<p>Isotope-coded affinity tag (ICAT)-based approaches could be applied to identify the cysteinyl residues involved in disulfide bond formation using mass spec. </p>
<p>The ICAT approach is achieved by splitting a protein sample and labeling half with a light sulfhydryl-reactive ICAT reagent and the other half with a heavy, stable isotope-labeled sulfhydryl-reactive ICAT reagent, mixing, proteolytically digesting, and then analyzing by mass spectrometry. </p>
<p>This results in readily identifiable pairs of mass spectral peaks in single-stage mass spectra that are separated by the mass difference between the light and heavy ICAT tags. </p>
<p>Mass mapping and analysis of the tagged peptides by tandem mass spectrometry (MS/MS) can reveal the identity of the labeled cysteine residue(s).</p>
<hr>
<ol start="2">
<li>You have just joined a biotechnology company and have been told to work on the BSc protein, which is a regulatory protein of therapeutic interest. Specifically, your task is to discover proteins whose cellular levels are influenced by the BSc protein. Describe a mass spec-based proteomic approaches you would use to identify such proteins [20 marks]. </li>
</ol>
<hr>
<ul>
<li>BSc protein (a regulatory protein of therapeutic interest)</li>
<li>discover proteins whose <code>cellular levels</code> are influenced by the BSc protein</li>
<li>mass spec-based proteomic approaches <code>identify</code></li>
</ul>
<hr>
<p>Answer; </p>
<p>From an MS-based proteomics perspective, all these challenges in cellular levels could be reflected by the quantification of the relative abundance of peptides in different samples. In this case, two technologies including isotope labeling and label-free quantification were used. Consequently, we could compare the abundance between control group and BSc protein influenced group</p>
<p>iTRAQ (isobaric Tags for Relative and Absolute Quantitation), one of the Stable-isotope Labeled Methods, uses up to eight isobaric tags that react with primary amine groups of peptides. During MS analysis, the tags are fragmented into reporter groups of different mass for each tag. The intensity of the different reporter groups is then used to derive the relative abundance of the corresponding peptides and proteins in the mixture.</p>
<p>Besides stable isotope labeling, so-called “label-free” quantification is increasingly used. The basic idea here is to align separate LC-MS/MS runs of peptide mixtures and to calculate differences in intensities of the same peptides detected in each run. Although quantification using this methodology is less accurate than methods using isotope labels, it may be simpler than isotope-based methods and makes cell types accessible that are difficult or impossible to label with amino acids.</p>
<p>The next part would be the data analysis, steps like quality control or normalization might be used for better performance and accurate identification. </p>
<blockquote>
<p>From an MS perspective, all these challenges necessitate the quantification of the relative abundance of peptides in different samples. Comparing two protein complexes or large assemblies, such as the nucleolus, requires knowing the relative abundance of peptides derived from proteins present in two preparations, whereas elucidation of signal response requires measuring changes in the levels of phosphopeptides.Several approaches have been developed to tackle this problem.</p>
<p>Mostly, we will focus here on two technologies that are frequently used and that can be combined seamlessly with the analysis pipeline described: isotope labeling and label-free quantification. </p>
<hr>
<p>In one of the metabolic labeling approaches, proteins fully incorporate “heavy”, nonradioactive isotope-containing amino acids (stable isotope labeling with amino acids in cell culture [SILAC]; <a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928005/#bib55" >Ong et al., 2002<i class="fas fa-external-link-alt"></i></a>) and are analyzed by high resolution MS. To this end, arginine and/or lysine that is labeled with 13C atoms and/or 15N atoms is fed to cells, which integrate these amino acids into all proteins in the course of several cell doublings. </p>
<p>Digestion of these proteins with trypsin, which cuts after arginine or lysine, leads to peptides with a heavy amino acid at their C terminus. The heavy labeled proteome remains distinguishable from the “light” or normal labeled control proteome and the two can be combined at the level of cells or directly after lysis. </p>
<p>This prevents differences in sample preparation from influencing quantification accuracy. The resulting mixture contains SILAC peptide pairs recognizable by having the exact mass difference between the heavy and normal amino acids. The relative intensity of the peaks reflects the relative abundance of the proteins in the mixture.</p>
<p>Although SILAC was originally developed for work on cell lines, it has been extended to include microorganisms and entire mice and even to the very accurate quantification of the levels of thousands of proteins in human tumor biopsies (<a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928005/#bib40" >Krüger et al., 2008<i class="fas fa-external-link-alt"></i></a>; <a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928005/#bib28" >Geiger et al., 2010<i class="fas fa-external-link-alt"></i></a>).</p>
<hr>
<p>The alternative to metabolic labeling is chemical modification of peptides by stable isotope–containing tags. The best-known strategy to this end is called iTRAQ. It uses up to eight isobaric tags that react with primary amine groups of peptides. During MS analysis, the tags are fragmented into reporter groups of different mass for each tag. The intensity of the different reporter groups is then used to derive the relative abundance of the corresponding peptides and proteins in the starting mixture (<a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928005/#bib62" >Ross et al., 2004<i class="fas fa-external-link-alt"></i></a>).</p>
<hr>
<p>Besides stable isotope labeling, so-called “label-free” quantification is increasingly used (<a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928005/#bib52" >Old et al., 2005<i class="fas fa-external-link-alt"></i></a>). The basic idea here is to align separate LC-MS/MS runs of peptide mixtures and to calculate differences in intensities of the same peptides detected in each run. Although quantification using this methodology is less accurate than methods using isotope labels, it may be simpler than isotope-based methods and makes cell types accessible that are difficult or impossible to label with amino acids (<a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928005/#bib48" >Malmström et al., 2009<i class="fas fa-external-link-alt"></i></a>; <a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928005/#bib46" >Luber et al., 2010<i class="fas fa-external-link-alt"></i></a>).</p>
<p>In addition to these two, a number of variations and alternative techniques have been developed that are reviewed in detail elsewhere (<a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928005/#bib54" >Ong and Mann, 2005<i class="fas fa-external-link-alt"></i></a>; <a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928005/#bib5" >Bantscheff et al., 2007<i class="fas fa-external-link-alt"></i></a>; <a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928005/#bib78" >Wilm, 2009<i class="fas fa-external-link-alt"></i></a>). As an example, chemical dimethyl labeling of peptides can be performed economically with isotopically light or heavy reagents (<a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928005/#bib32" >Hsu et al., 2003<i class="fas fa-external-link-alt"></i></a>; <a class="link"   target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928005/#bib10" >Boersema et al., 2009<i class="fas fa-external-link-alt"></i></a>). As usual with chemical techniques in proteomics, it is important to ensure that reactions go to completion. Last not least, if quantification of only a selected subset of proteins is desired, peptides of these proteins can be targeted by a technique called multiple reaction monitoring, or MRM. This requires specialized “triple quadrupole” mass spectrometers, which consist of a selection quadrupole for the precursor ion, a collision cell quadrupole, and a selection quadrupole for the fragments. They are set to exclusively monitor predetermined precursor-to-fragment transitions in rapid succession. In this way the presence and—if an isotope-labeled synthetic peptide analogue is used, the quantity of selected peptides—can be monitored</p>
</blockquote>
<h2 id="Past-exams"><a href="#Past-exams" class="headerlink" title="Past exams"></a>Past exams</h2><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083438.png" alt="image-20210607083438721"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083456.png" alt="image-20210607083456124"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083508.png" alt="image-20210607083508423"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083519.png" alt="image-20210607083519162"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083533.png"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083612.png" alt="image-20210607083542617"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083606.png" alt="image-20210607083551275"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083600.png" alt="image-20210607083600897"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083703.png" alt="image-20210607083702954"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083713.png" alt="image-20210607083713247"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083743.png" alt="image-20210607083743494"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083751.png" alt="image-20210607083751472"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083800.png" alt="image-20210607083759980"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083812.png" alt="image-20210607083812759"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083820.png" alt="image-20210607083820806"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083829.png" alt="image-20210607083829377"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083841.png" alt="image-20210607083841863"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607083850.png" alt="image-20210607083850383"></p>

        </div>

        
            <div class="post-copyright-info">
                <div class="article-copyright-info-container">
    <ul>
        <li>Post title：BIO312 Wang Mu&#39;s CW</li>
        <li>Post author：Yuxuan Wu</li>
        <li>Create time：2021-03-10 05:50:03</li>
        <li>
            Post link：yuxuanwu17.github.io2021/03/10/BIO312 Wang Mu&#39;s CW/
        </li>
        <li>
            Copyright Notice：All articles in this blog are licensed under <a class="license" target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by-nc-sa/4.0/deed.zh">BY-NC-SA</a> unless stating additionally.
        </li>
    </ul>
</div>

            </div>
        

        
            <div class="article-nav">
                
                    <div class="article-prev">
                        <a class="prev"
                           rel="prev"
                           href="/2021/03/10/2021-03-10-BIO304-biotechnology-presentation/"
                        >
                            <span class="left arrow-icon flex-center">
                              <i class="fas fa-chevron-left"></i>
                            </span>
                            <span class="title flex-center">
                                <span class="post-nav-title-item">biotechnology presentation</span>
                                <span class="post-nav-item">Prev posts</span>
                            </span>
                        </a>
                    </div>
                
                
                    <div class="article-next">
                        <a class="next"
                           rel="next"
                           href="/2021/03/02/2021-03-02-BIO312-Wangs-part/"
                        >
                            <span class="title flex-center">
                                <span class="post-nav-title-item">BIO312 Wang&#39;s part</span>
                                <span class="post-nav-item">Next posts</span>
                            </span>
                            <span class="right arrow-icon flex-center">
                              <i class="fas fa-chevron-right"></i>
                            </span>
                        </a>
                    </div>
                
            </div>
        

        
    </div>
</div>


                
            </div>

        </div>

        <div class="page-main-content-bottom">
            <footer class="footer">
    <div class="info-container">
        <div class="copyright-info info-item">
            &copy;
            
              <span>2020</span> -
            
            2022 <i class="fas fa-heart icon-animate"></i> <a href="/">Yuxuan Wu</a>
        </div>
        
            <script async  src="//busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
            <div class="website-count info-item">
                
                    <span id="busuanzi_container_site_uv">
                        Visitor Count <span id="busuanzi_value_site_uv"></span>&ensp;
                    </span>
                
                
                    <span id="busuanzi_container_site_pv">
                        Totalview <span id="busuanzi_value_site_pv"></span>
                    </span>
                
            </div>
        
        <div class="theme-info info-item">
            Powered by <a target="_blank" href="https://hexo.io">Hexo</a> | Theme <a class="theme-version" target="_blank" href="https://github.com/XPoet/hexo-theme-keep">Keep v3.3.1</a>
        </div>
    </div>
</footer>

        </div>
    </div>

    
        <div class="post-tools">
            <div class="post-tools-container">
    <ul class="tools-list">
        <!-- TOC aside toggle -->
        
            <li class="tools-item page-aside-toggle">
                <i class="fas fa-outdent"></i>
            </li>
        

        <!-- go comment -->
        
    </ul>
</div>

        </div>
    

    <div class="right-bottom-side-tools">
        <div class="side-tools-container">
    <ul class="side-tools-list">
        <li class="tools-item tool-font-adjust-plus flex-center">
            <i class="fas fa-search-plus"></i>
        </li>

        <li class="tools-item tool-font-adjust-minus flex-center">
            <i class="fas fa-search-minus"></i>
        </li>

        <li class="tools-item tool-expand-width flex-center">
            <i class="fas fa-arrows-alt-h"></i>
        </li>

        <li class="tools-item tool-dark-light-toggle flex-center">
            <i class="fas fa-moon"></i>
        </li>

        <!-- rss -->
        

        

        <li class="tools-item tool-scroll-to-bottom flex-center">
            <i class="fas fa-arrow-down"></i>
        </li>
    </ul>

    <ul class="exposed-tools-list">
        <li class="tools-item tool-toggle-show flex-center">
            <i class="fas fa-cog fa-spin"></i>
        </li>
        
            <li class="tools-item tool-scroll-to-top flex-center">
                <i class="arrow-up fas fa-arrow-up"></i>
                <span class="percent"></span>
            </li>
        
    </ul>
</div>

    </div>

    
        <aside class="page-aside">
            <div class="post-toc-wrap">
    <div class="post-toc">
        <ol class="nav"><li class="nav-item nav-level-2"><a class="nav-link" href="#Proteomics-workflow"><span class="nav-number">1.</span> <span class="nav-text">Proteomics workflow</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Mass-Spectrometry"><span class="nav-number">2.</span> <span class="nav-text">Mass Spectrometry</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Mass-Spectrometry-based-Protein-Identification"><span class="nav-number">2.1.</span> <span class="nav-text">Mass Spectrometry-based Protein Identification</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#cw1"><span class="nav-number">3.</span> <span class="nav-text">cw1</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Task1"><span class="nav-number">3.1.</span> <span class="nav-text">Task1</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Task2"><span class="nav-number">3.2.</span> <span class="nav-text">Task2</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Task3"><span class="nav-number">3.3.</span> <span class="nav-text">Task3</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#cw2"><span class="nav-number">4.</span> <span class="nav-text">cw2</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Past-exams"><span class="nav-number">5.</span> <span class="nav-text">Past exams</span></a></li></ol>
    </div>
</div>
        </aside>
    

    <div class="image-viewer-container">
    <img src="">
</div>


    
        <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
          <span class="search-input-field-pre">
            <i class="fas fa-keyboard"></i>
          </span>
            <div class="search-input-container">
                <input autocomplete="off"
                       autocorrect="off"
                       autocapitalize="off"
                       placeholder="Search..."
                       spellcheck="false"
                       type="search"
                       class="search-input"
                >
            </div>
            <span class="popup-btn-close">
                <i class="fas fa-times"></i>
            </span>
        </div>
        <div id="search-result">
            <div id="no-result">
                <i class="fas fa-spinner fa-pulse fa-5x fa-fw"></i>
            </div>
        </div>
    </div>
</div>

    

</main>




<script src="/js/utils.js"></script>

<script src="/js/main.js"></script>

<script src="/js/header-shrink.js"></script>

<script src="/js/back2top.js"></script>

<script src="/js/dark-light-toggle.js"></script>



    
<script src="/js/local-search.js"></script>




    
<script src="/js/code-copy.js"></script>





<div class="post-scripts">
    
        
<script src="/js/left-side-toggle.js"></script>

<script src="/js/libs/anime.min.js"></script>

<script src="/js/toc.js"></script>

    
</div>



</body>
</html>
