<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="keywords" content="Hexo Theme Keep">
    <meta name="description" content="Welcome to my personal blogs">
    <meta name="author" content="Yuxuan Wu">
    
    <title>
        
            BIO312 Wang&#39;s part |
        
        Yuxuan Wu
    </title>
    <link rel="shortcut icon" href="/images/logo.svg">
    
<link rel="stylesheet" href="/css/style.css">

    
<link rel="stylesheet" href="/css/font-awesome.min.css">

    <script id="hexo-configurations">
    let KEEP = window.KEEP || {};
    KEEP.hexo_config = {"hostname":"example.com","root":"/","language":"en","path":"search.xml"};
    KEEP.theme_config = {"toc":{"enable":true,"number":true,"expand_all":false},"style":{"primary_color":"#0066CC","avatar":"/images/avatar.svg","favicon":"/images/logo.svg","article_img_align":"left","left_side_width":"260px","content_max_width":"920px","hover":{"shadow":true,"scale":true},"first_screen":{"enable":true,"background_img":"/images/bg.svg","description":"Keep coding, Keep hungry."},"scroll":{"progress_bar":{"enable":true},"percent":{"enable":true}}},"local_search":{"enable":true,"preload":true},"code_copy":{"enable":true,"style":"default"},"pjax":{"enable":false},"lazyload":{"enable":false},"version":"3.3.1"};
    KEEP.language_ago = {"second":"%s seconds ago","minute":"%s minutes ago","hour":"%s hours ago","day":"%s days age","week":"%s weeks age","month":"%s months age","year":"%s years age"};
  </script>
<!-- hexo injector head_end start -->
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css">

<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/hexo-math@4.0.0/dist/style.css">
<!-- hexo injector head_end end --><meta name="generator" content="Hexo 5.3.0"><style>mjx-container[jax="SVG"] {
  direction: ltr;
}

mjx-container[jax="SVG"] > svg {
  overflow: visible;
}

mjx-container[jax="SVG"] > svg a {
  fill: blue;
  stroke: blue;
}

mjx-container[jax="SVG"][display="true"] {
  display: block;
  text-align: center;
  margin: 1em 0;
}

mjx-container[jax="SVG"][justify="left"] {
  text-align: left;
}

mjx-container[jax="SVG"][justify="right"] {
  text-align: right;
}

g[data-mml-node="merror"] > g {
  fill: red;
  stroke: red;
}

g[data-mml-node="merror"] > rect[data-background] {
  fill: yellow;
  stroke: none;
}

g[data-mml-node="mtable"] > line[data-line] {
  stroke-width: 70px;
  fill: none;
}

g[data-mml-node="mtable"] > rect[data-frame] {
  stroke-width: 70px;
  fill: none;
}

g[data-mml-node="mtable"] > .mjx-dashed {
  stroke-dasharray: 140;
}

g[data-mml-node="mtable"] > .mjx-dotted {
  stroke-linecap: round;
  stroke-dasharray: 0,140;
}

g[data-mml-node="mtable"] > svg {
  overflow: visible;
}

[jax="SVG"] mjx-tool {
  display: inline-block;
  position: relative;
  width: 0;
  height: 0;
}

[jax="SVG"] mjx-tool > mjx-tip {
  position: absolute;
  top: 0;
  left: 0;
}

mjx-tool > mjx-tip {
  display: inline-block;
  padding: .2em;
  border: 1px solid #888;
  font-size: 70%;
  background-color: #F8F8F8;
  color: black;
  box-shadow: 2px 2px 5px #AAAAAA;
}

g[data-mml-node="maction"][data-toggle] {
  cursor: pointer;
}

mjx-status {
  display: block;
  position: fixed;
  left: 1em;
  bottom: 1em;
  min-width: 25%;
  padding: .2em .4em;
  border: 1px solid #888;
  font-size: 90%;
  background-color: #F8F8F8;
  color: black;
}

foreignObject[data-mjx-xml] {
  font-family: initial;
  line-height: normal;
  overflow: visible;
}

.MathJax path {
  stroke-width: 3;
}

mjx-container[display="true"] {
  overflow: auto hidden;
}

mjx-container[display="true"] + br {
  display: none;
}
</style></head>


<body>
<div class="progress-bar-container">
    
        <span class="scroll-progress-bar"></span>
    

    
</div>


<main class="page-container">

    

    <div class="page-main-content">

        <div class="page-main-content-top">
            <header class="header-wrapper">

    <div class="header-content">
        <div class="left">
            <a class="logo-title" href="/">
                Yuxuan Wu
            </a>
        </div>

        <div class="right">
            <div class="pc">
                <ul class="menu-list">
                    
                        <li class="menu-item">
                            <a class=""
                               href="/"
                            >
                                HOME
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/archives"
                            >
                                ARCHIVES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/categories"
                            >
                                CATEGORIES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/tags"
                            >
                                TAGS
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/about"
                            >
                                ABOUT
                            </a>
                        </li>
                    
                    
                        <li class="menu-item search search-popup-trigger">
                            <i class="fas fa-search"></i>
                        </li>
                    
                </ul>
            </div>
            <div class="mobile">
                
                    <div class="icon-item search search-popup-trigger"><i class="fas fa-search"></i></div>
                
                <div class="icon-item menu-bar">
                    <div class="menu-bar-middle"></div>
                </div>
            </div>
        </div>
    </div>

    <div class="header-drawer">
        <ul class="drawer-menu-list">
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/">HOME</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/archives">ARCHIVES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/categories">CATEGORIES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/tags">TAGS</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/about">ABOUT</a>
                </li>
            
        </ul>
    </div>

    <div class="window-mask"></div>

</header>


        </div>

        <div class="page-main-content-middle">

            <div class="main-content normal-code-theme">

                
                    <div class="fade-in-down-animation">
    <div class="article-content-container">

        <div class="article-title">
            <span class="title-hover-animation">BIO312 Wang&#39;s part</span>
        </div>

        
            <div class="article-header">
                <div class="avatar">
                    <img src="/images/avatar.svg">
                </div>
                <div class="info">
                    <div class="author">
                        <span class="name">Yuxuan Wu</span>
                        <span class="level">Lv11</span>
                    </div>
                    <div class="meta-info">
                        <div class="article-meta-info">
    <span class="article-date article-meta-item">
        <i class="fas fa-edit"></i> 2021-03-02 19:44:01
    </span>
    
        <span class="article-categories article-meta-item">
            <i class="fas fa-folder"></i>
            <ul>
                
                    <li>
                        <a href="/categories/Biology/">Biology</a>
                    </li>
                
            </ul>
        </span>
    
    
        <span class="article-tags article-meta-item">
            <i class="fas fa-tags"></i>
            <ul>
                
                    <li>
                        <a href="/tags/notes/">notes</a>
                    </li>
                
            </ul>
        </span>
    

    
    
        <span class="article-wordcount article-meta-item">
            <i class="fas fa-file-word"></i> <span>1.3k Words</span>
        </span>
    
    
        <span class="article-min2read article-meta-item">
            <i class="fas fa-clock"></i> <span>7 Mins</span>
        </span>
    
    
        <span class="article-pv article-meta-item">
            <i class="fas fa-eye"></i> <span id="busuanzi_value_page_pv"></span>
        </span>
    
</div>

                    </div>
                </div>
            </div>
        

        <div class="article-content markdown-body">
            <h2 id="Revision-protein-structure-and-methods-of-analysis"><a href="#Revision-protein-structure-and-methods-of-analysis" class="headerlink" title="Revision protein structure and methods of analysis"></a>Revision protein structure and methods of analysis</h2><h3 id="Techniques-for-protein-analysis"><a href="#Techniques-for-protein-analysis" class="headerlink" title="Techniques for protein analysis"></a>Techniques for protein analysis</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210604203452.png" alt="image-20210604203452223"></p>
<h3 id="Protein-seperation"><a href="#Protein-seperation" class="headerlink" title="Protein seperation"></a>Protein seperation</h3><ol>
<li><strong>SDS-PAGE:</strong> basis-molecular weight</li>
<li><strong>Isoelectric focusing:</strong> electric charge difference</li>
<li><strong>Chromatic methods-useful adjuncts to gel-based approaches</strong><ol>
<li>HPLC - high performance liquid chromatography </li>
<li>Thin-layer chromatography </li>
</ol>
</li>
<li><strong>2D gel electrophoresis</strong><ul>
<li>Powerful gel-based method commonly used to analyze complex samples</li>
<li>To characterize full range of proteins in the sample, not just a few specific proteins</li>
</ul>
</li>
</ol>
<h3 id="SDS-PAGE"><a href="#SDS-PAGE" class="headerlink" title="SDS-PAGE:"></a>SDS-PAGE:</h3><ul>
<li>Proteins are usually mixed with the detergent, SDS and a tracking dye to make the sample</li>
<li>SDS binds to the protein and gives it size-dependent negative charge and consistent hydrodynamic properties</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605140749.png" alt="image-20210605140749138"></p>
<h3 id="Isoelectric-point-pi"><a href="#Isoelectric-point-pi" class="headerlink" title="Isoelectric point (pi)"></a>Isoelectric point (pi)</h3><ul>
<li>ph at which molecule has net zero charge （determined using computer program for known sequence or empirically ）</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210604203621.png" alt="image-20210604203621625"></p>
<h3 id="Isoelectric-Focusing"><a href="#Isoelectric-Focusing" class="headerlink" title="Isoelectric Focusing"></a>Isoelectric Focusing</h3><p>Electrophoresis through polyacrylamide gel in which there is a pH gradient</p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210302212707.png" alt="image-20210302212707320"></p>
<h3 id="2D-gel"><a href="#2D-gel" class="headerlink" title="2D gel"></a>2D gel</h3><p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210302212808.png" alt="image-20210302212808199"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605141133.png" alt="image-20210605141133752"></p>
<h3 id="Strength-and-weakness-of-the-2D-gel"><a href="#Strength-and-weakness-of-the-2D-gel" class="headerlink" title="Strength and weakness of the 2D gel"></a>Strength and weakness of the 2D gel</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605141238.png" alt="image-20210605141238352"></p>
<h3 id="Western-Blot"><a href="#Western-Blot" class="headerlink" title="Western Blot"></a>Western Blot</h3><p>a.k.a: immunoblotting</p>
<ul>
<li>Proteins first electrophoreses by SDS-PAGE</li>
<li>Wet gel is placed against <strong>nitrocellulose</strong></li>
<li>In electrophoretic chamber: </li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210604211133.png" alt="image-20210604211133453"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210604212705.png" alt="image-20210604212705256"></p>
<h3 id="Salting-out-precipitation-in-protein-fraction"><a href="#Salting-out-precipitation-in-protein-fraction" class="headerlink" title="Salting out (precipitation in protein fraction)"></a>Salting out (precipitation in protein fraction)</h3><p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210302212907.png" alt="image-20210302212906975"></p>
<blockquote>
<p>Ammonium sulfate precipitation is one of the most commonly used methods for protein purification from a solution. </p>
<p>In solution, proteins form hydrogen bonds with water molecules through their exposed polar and ionic groups. When high concentrations of small, highly charged ions such as ammonium sulfate are added, these groups compete with the proteins to bind to the water molecules. </p>
<p>This removes the water molecules from the protein and decreases its solubility, resulting in precipitation. </p>
<p>Critical factors that affect the concentration at which a particular protein will precipitate include: the number and position of polar groups, molecular weight of the protein, pH of the solution, and temperature at which the precipitation is performed.</p>
<p>The concentration at which antibodies precipitate varies among species; most rabbit antibodies precipitate with a 40% saturated solution, whereas mouse antibodies require 45-50% saturation.</p>
</blockquote>
<blockquote>
<p><strong>Protocol</strong></p>
<ol>
<li>Allow serum or ascitic fluid to thaw, determine total volume, and centrifuge at 3000g for 30 minutes.</li>
<li>Transfer sample to a beaker containing a stir bar and place on a magnetic stirrer.</li>
<li>While the sample is stirring, slowly add saturated ammonium sulfate to bring the final concentration to 50% saturation.<ol>
<li>Volume of ammonium sulfate needed is equal to the volume of sample.</li>
<li>Adding the ammonium sulfate very slowly ensures that local concentration around the site of addition does not exceed the desired salt concentration.</li>
</ol>
</li>
<li>Once the total volume of ammonium sulfate is added, move beaker to 4°C for 6 hours or overnight.</li>
<li>Transfer to conical tube and centrifuge the precipitate at 3000g for 30 minutes.</li>
<li>Carefully remove and discard supernatant. Invert conical tube and drain well. For serum or ascites, resuspend pellet in 30%-50% of the starting volume in 1XPBS. For monoclonal antibody tissue culture supernatants, resuspend pellet in 10% of the starting volume in 1X PBS.</li>
<li>Transfer antibody solution to dialysis tubing and dialyze versus three changes of 1XPBS/0.08% Sodium Azide. Be sure to allow enough space for expansion of the antibody solution during dialysis. Normally twice the re-suspended volume is sufficient.</li>
<li>Remove antibody solution from the tubing and centrifuge to remove any remaining debris.</li>
<li>Determine the concentration and store at -80°C for long-term storage.</li>
</ol>
</blockquote>
<h3 id="Centrifugation"><a href="#Centrifugation" class="headerlink" title="Centrifugation"></a>Centrifugation</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210604204108.png"></p>
<h3 id="chromatography"><a href="#chromatography" class="headerlink" title="chromatography"></a>chromatography</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210604204806.png" alt="image-20210604204806332"></p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210302213429.png" alt="image-20210302213429785"></p>
<h3 id="Gel-filtration-Size-exclusion-chromatography"><a href="#Gel-filtration-Size-exclusion-chromatography" class="headerlink" title="Gel filtration/Size exclusion chromatography"></a>Gel filtration/Size exclusion chromatography</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210604210844.png" alt="image-20210604210844294"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605140832.png" alt="image-20210605140832408"></p>
<h3 id="Ion-exchange-chromatography"><a href="#Ion-exchange-chromatography" class="headerlink" title="Ion exchange chromatography"></a>Ion exchange chromatography</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210604211021.png" alt="image-20210604211021712"></p>
<h3 id="Hydrophobic-interaction-chromatography"><a href="#Hydrophobic-interaction-chromatography" class="headerlink" title="Hydrophobic interaction chromatography"></a>Hydrophobic interaction chromatography</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210604211046.png" alt="image-20210604211046644"></p>
<h3 id="Protein-sequencing"><a href="#Protein-sequencing" class="headerlink" title="Protein sequencing"></a>Protein sequencing</h3><p>Sanger sequence (protein identification)</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210604212758.png" alt="image-20210604212758107"></p>
<h3 id="Cleavage-of-polypeptides-for-analysis"><a href="#Cleavage-of-polypeptides-for-analysis" class="headerlink" title="Cleavage of polypeptides for analysis"></a>Cleavage of polypeptides for analysis</h3><ol>
<li>Strong acid - not sequence specific</li>
<li>Sequence-specific proteolytic enzymes (proteases)</li>
<li>Sequence-specific chemical cleavage (<strong>cyanogen</strong> bromide cleavage at <strong>methonine residues</strong>)</li>
</ol>
<p>Only used in the mutant protein that is not in the databases </p>
<p>FPLC: </p>
<p>HPLC: high pressure </p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210604214657.png" alt="image-20210604214657230"></p>
<h3 id="Protein-identification"><a href="#Protein-identification" class="headerlink" title="Protein identification"></a>Protein identification</h3><h4 id="Edman-degradation"><a href="#Edman-degradation" class="headerlink" title="Edman degradation"></a>Edman degradation</h4><p>Sequence aa in a peptide. </p>
<p>Amino-terminal residue is <strong>labeled</strong> and cleaved from the pipetted <strong>without disrupting the peptide bonds</strong> between other amino acid residues</p>
<h4 id="Mass-spectrometry"><a href="#Mass-spectrometry" class="headerlink" title="Mass spectrometry"></a>Mass spectrometry</h4><h2 id="Mass-Spectrometry"><a href="#Mass-Spectrometry" class="headerlink" title="Mass Spectrometry:"></a>Mass Spectrometry:</h2><p>Analytical techniques that measures the mass-to-charge ratio of charged particles, mass spectrometry is used for determining <strong>masses</strong> of particles and the <strong>elemental composition of a sample</strong> of molecules as well as for <strong>elucidating the chemical structure of molecules</strong> such as peptide</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605141318.png" alt="image-20210605141318489"></p>
<h3 id="Workflow"><a href="#Workflow" class="headerlink" title="Workflow"></a>Workflow</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605141353.png" alt="image-20210605141352968"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605141525.png" alt="image-20210605141525465"></p>
<h3 id="Types-of-MS-data"><a href="#Types-of-MS-data" class="headerlink" title="Types of MS data"></a>Types of MS data</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605141825.png" alt="image-20210605141825840"></p>
<h3 id="3D-structure-determination"><a href="#3D-structure-determination" class="headerlink" title="3D structure determination"></a>3D structure determination</h3><ul>
<li>X-Ray Diffractionneed to crystalize (can be difficult)<ul>
<li><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605142123.png" alt="image-20210605142123677"></li>
</ul>
</li>
<li>NMR small proteins (&lt;25kD)<ul>
<li><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605142133.png" alt="image-20210605142133347"></li>
</ul>
</li>
<li>Electron Micrographpoor resolution<ul>
<li><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605142156.png" alt="image-20210605142156648"></li>
</ul>
</li>
</ul>
<h2 id="Identification-of-proteins-in-complex-mixtures"><a href="#Identification-of-proteins-in-complex-mixtures" class="headerlink" title="Identification of proteins in complex mixtures"></a>Identification of proteins in complex mixtures</h2><h3 id="Proteomics-achievement"><a href="#Proteomics-achievement" class="headerlink" title="Proteomics achievement"></a>Proteomics achievement</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605143214.png" alt="image-20210605143214334"></p>
<h3 id="Protein-Identification-by-Mass-Spectrometry"><a href="#Protein-Identification-by-Mass-Spectrometry" class="headerlink" title="Protein Identification by Mass Spectrometry"></a>Protein Identification by Mass Spectrometry</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605143257.png" alt="image-20210605143257113"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605143712.png" alt="image-20210605143712522"></p>
<h3 id="Mass-Spec-MS-Basics"><a href="#Mass-Spec-MS-Basics" class="headerlink" title="Mass Spec (MS) Basics"></a>Mass Spec (MS) Basics</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605143810.png" alt="image-20210605143810496"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605143917.png" alt="image-20210605143917476"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605143954.png" alt="image-20210605143954296"></p>
<h4 id="MALDI"><a href="#MALDI" class="headerlink" title="MALDI"></a>MALDI</h4><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605143855.png" alt="image-20210605143855901"></p>
<h3 id="Mass-resolution"><a href="#Mass-resolution" class="headerlink" title="Mass resolution"></a>Mass resolution</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605144317.png" alt="image-20210605144317454"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607110709.png" alt="image-20210607110709595"></p>
<h3 id="LC-MS-MS-for-protein-identification"><a href="#LC-MS-MS-for-protein-identification" class="headerlink" title="LC-MS/MS for protein identification"></a>LC-MS/MS for protein identification</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605144533.png" alt="image-20210605144533559"></p>
<h3 id="Mass-spectrometer-is-always-right"><a href="#Mass-spectrometer-is-always-right" class="headerlink" title="Mass spectrometer is always right"></a>Mass spectrometer is always right</h3><h2 id="Advanced-proteomic-techniques"><a href="#Advanced-proteomic-techniques" class="headerlink" title="Advanced proteomic techniques"></a>Advanced proteomic techniques</h2><h3 id="Summary-of-existing-technology"><a href="#Summary-of-existing-technology" class="headerlink" title="Summary of existing technology"></a>Summary of existing technology</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605145325.png" alt="image-20210605145324947"></p>
<h3 id="ICAT"><a href="#ICAT" class="headerlink" title="ICAT"></a>ICAT</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605145359.png" alt="image-20210605145359682"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605145412.png"></p>
<h3 id="Top-Down-or-Bottom-up-analysis"><a href="#Top-Down-or-Bottom-up-analysis" class="headerlink" title="Top-Down or Bottom-up analysis"></a>Top-Down or Bottom-up analysis</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523141041.png" alt="image-20210523141041633"></p>
<h3 id="Top-down-advantages"><a href="#Top-down-advantages" class="headerlink" title="Top-down advantages"></a>Top-down advantages</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523141131.png" alt="image-20210523141131007"></p>
<h3 id="disadvantages-for-top-down"><a href="#disadvantages-for-top-down" class="headerlink" title="disadvantages for top-down"></a>disadvantages for top-down</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523141200.png" alt="image-20210523141200747"></p>
<h2 id="Quantitative-Mass-Spec-based-Proteomics"><a href="#Quantitative-Mass-Spec-based-Proteomics" class="headerlink" title="Quantitative Mass Spec-based Proteomics"></a>Quantitative Mass Spec-based Proteomics</h2><h3 id="Outline"><a href="#Outline" class="headerlink" title="Outline"></a>Outline</h3><ul>
<li><p>Principles of quantitative proteomics</p>
</li>
<li><p>Gel vs. non-gel approaches</p>
</li>
<li><p>Labeling vs. non-labeling approaches</p>
</li>
<li><p>Case study: candidate biomarker generation using label-free approaches</p>
</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523141655.png" alt="image-20210523141655702"></p>
<h3 id="Summary-of-label-methods"><a href="#Summary-of-label-methods" class="headerlink" title="Summary of label methods"></a>Summary of label methods</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523143459.png" alt="image-20210523143459324"></p>
<h3 id="Pros-and-Cons-for-label-free-methods"><a href="#Pros-and-Cons-for-label-free-methods" class="headerlink" title="Pros and Cons for label-free methods"></a>Pros and Cons for label-free methods</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523143106.png" alt="image-20210523143106440"></p>
<h4 id="Methods"><a href="#Methods" class="headerlink" title="Methods"></a>Methods</h4><p>Spectral counting  &amp; Peak-intensity</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523143644.png" alt="image-20210523143644199"></p>
<h3 id="Spectral-counting"><a href="#Spectral-counting" class="headerlink" title="Spectral counting"></a>Spectral counting</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605150028.png" alt="image-20210605150028734"></p>
<h3 id="Peptide-Ion-Intensity-Based-Protein-Quantification"><a href="#Peptide-Ion-Intensity-Based-Protein-Quantification" class="headerlink" title="Peptide Ion Intensity Based Protein Quantification"></a>Peptide Ion Intensity Based Protein Quantification</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605150139.png" alt="image-20210605150139851"></p>
<h3 id="LC-MS-Chromatographic-Alignment"><a href="#LC-MS-Chromatographic-Alignment" class="headerlink" title="LC-MS Chromatographic Alignment"></a>LC-MS Chromatographic Alignment</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523143857.png" alt="image-20210523143857757"></p>
<h3 id="Limitations-of-LC-MS-based-Approach-to-Large-scale-Protein-Profiling"><a href="#Limitations-of-LC-MS-based-Approach-to-Large-scale-Protein-Profiling" class="headerlink" title="Limitations of LC-MS-based Approach to Large-scale Protein Profiling"></a>Limitations of LC-MS-based Approach to Large-scale Protein Profiling</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605150329.png" alt="image-20210605150329515"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605150347.png" alt="image-20210605150347726"></p>
<h3 id="Challenges-in-proteomics"><a href="#Challenges-in-proteomics" class="headerlink" title="Challenges in proteomics"></a>Challenges in proteomics</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605150411.png" alt="image-20210605150411526"></p>
<h2 id="Tutorial-1"><a href="#Tutorial-1" class="headerlink" title="Tutorial 1"></a>Tutorial 1</h2><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525174544.png" alt="image-20210525174544609"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525174744.png" alt="image-20210525174744143"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525174753.png" alt="image-20210525174753723"></p>
<h3 id="Proteomics-workflow"><a href="#Proteomics-workflow" class="headerlink" title="Proteomics workflow"></a>Proteomics workflow</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607095123.png" alt="image-20210607095123034"></p>
<h3 id="Separation-technique"><a href="#Separation-technique" class="headerlink" title="Separation technique"></a>Separation technique</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607095143.png" alt="image-20210607095143133"></p>
<h3 id="Mass-spectrometry-1"><a href="#Mass-spectrometry-1" class="headerlink" title="Mass spectrometry"></a>Mass spectrometry</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607095409.png" alt="image-20210607095409728"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607100124.png" alt="image-20210607100124714"></p>
<h3 id="Technique-summary"><a href="#Technique-summary" class="headerlink" title="Technique summary"></a>Technique summary</h3><h4 id="PMF"><a href="#PMF" class="headerlink" title="PMF"></a>PMF</h4><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607100152.png" alt="image-20210607100152683"></p>
<h3 id="TMS"><a href="#TMS" class="headerlink" title="TMS"></a>TMS</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607100211.png" alt="image-20210607100211529"></p>
<ul>
<li>注意这儿是Tandem MS sequence confirmation</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210607103510.png" alt="image-20210607103510287"></p>
<h3 id="The-principle-of-SDS-PAGE"><a href="#The-principle-of-SDS-PAGE" class="headerlink" title="The principle of SDS-PAGE"></a>The principle of SDS-PAGE</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525175334.png" alt="image-20210525175334329"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525175357.png" alt="image-20210525175357744"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210606150152.png" alt="image-20210525175525318"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525175915.png"></p>
<h3 id="Western-plot"><a href="#Western-plot" class="headerlink" title="Western plot"></a>Western plot</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525181246.png" alt="image-20210525181246074"></p>
<h2 id="Tutorial-2"><a href="#Tutorial-2" class="headerlink" title="Tutorial 2"></a>Tutorial 2</h2><h3 id="Labeling-vs-non-labeling"><a href="#Labeling-vs-non-labeling" class="headerlink" title="Labeling vs non-labeling"></a>Labeling vs non-labeling</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523145719.png" alt="image-20210523145719943"></p>
<p><code>comparison between label &amp; label free</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523160242.png" alt="image-20210523160242108"></p>
<h3 id="Label-free-approaches"><a href="#Label-free-approaches" class="headerlink" title="Label free approaches"></a>Label free approaches</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523151047.png" alt="image-20210523151047928"></p>
<h3 id="pros-and-cons-of-label-free-protein"><a href="#pros-and-cons-of-label-free-protein" class="headerlink" title="pros and cons of label free protein"></a>pros and cons of label free protein</h3><ul>
<li><p>Pros:</p>
<ul>
<li>Technically <code>simple</code><ul>
<li>No complicated (or expensive) labelling or tagging protocol</li>
</ul>
</li>
<li>Can <code>avoid complex pre-separation</code><ul>
<li>direct LC-MS is okay</li>
</ul>
</li>
</ul>
</li>
<li><p>Cons:</p>
<ul>
<li>Still can be <code>expensive</code><ul>
<li>Need plenty of replicates to get statistical power (machine time!)</li>
</ul>
</li>
<li>Need good data analysis software</li>
<li>Not straightforward to validate results from big data sets</li>
<li><code>Low abundance proteins</code> hard to measure accurately</li>
</ul>
</li>
</ul>
<h3 id="Why-does-MS1-intensity-not-tell-us-peptide-abundance-directly"><a href="#Why-does-MS1-intensity-not-tell-us-peptide-abundance-directly" class="headerlink" title="Why does MS1 intensity not tell us peptide abundance directly?"></a>Why does MS1 intensity not tell us peptide abundance directly?</h3><ul>
<li>Peptide sequences <code>ionise to different extents</code><ul>
<li>based on the biophysical properties of a given sequence</li>
</ul>
</li>
<li>Thus, measured ion intensity <code>only weakly correlates to the abundance</code> of the peptide (and hence) protein in the proteome</li>
<li><code>Not simple to extract the total ion abundance</code> for a given peptide in different runs</li>
<li><code>Difficult</code> to ensure that you are <code>comparing the same signal in parallel runs</code></li>
</ul>
<h3 id="Spectral-counting-1"><a href="#Spectral-counting-1" class="headerlink" title="Spectral counting"></a>Spectral counting</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523152406.png" alt="image-20210523152406643"></p>
<h3 id="The-normalization-of-spectral-counts"><a href="#The-normalization-of-spectral-counts" class="headerlink" title="The normalization of spectral counts"></a>The normalization of spectral counts</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523152458.png" alt="image-20210523152458935"></p>
<h3 id="Intensity-based-label-free"><a href="#Intensity-based-label-free" class="headerlink" title="Intensity based label free"></a>Intensity based label free</h3><h4 id="Method-summary"><a href="#Method-summary" class="headerlink" title="Method summary:"></a>Method summary:</h4><ul>
<li>Compare MS1 peptide ion abundance across replicates runs;</li>
<li>Specialist software must align different parallel LC-MS runs</li>
<li>Calculate ratios from aligned MS1 data</li>
<li>Requires good software (e.g.: Mascot distiller; Progenesis; MaxQuant; OpenMS; ProteomeDiscoverer; PLGS, many others…)</li>
</ul>
<h4 id="Advantages"><a href="#Advantages" class="headerlink" title="Advantages:"></a>Advantages:</h4><ul>
<li>In theory, should be more accurate than spectral counting – uses real intensity data</li>
<li>No complicated labelling protocols</li>
</ul>
<h4 id="Disadvantages"><a href="#Disadvantages" class="headerlink" title="Disadvantages:"></a>Disadvantages:</h4><ul>
<li>Data processing is fairly CPU intensive</li>
<li>Only works well if experimental system has high technical and biological reproducibility</li>
</ul>
<h4 id="Requirements"><a href="#Requirements" class="headerlink" title="Requirements:"></a>Requirements:</h4><ul>
<li>Very good LC delivery system (must be reproducible)</li>
<li>High resolution mass spec (Orbitrap, Synapt etc…)</li>
<li>Good PC for running software</li>
</ul>
<h4 id="Labelling-quantification"><a href="#Labelling-quantification" class="headerlink" title="Labelling quantification"></a>Labelling quantification</h4><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523160136.png" alt="image-20210523160136172"></p>
<h4 id="Two-approaches"><a href="#Two-approaches" class="headerlink" title="Two approaches"></a>Two approaches</h4><ol>
<li><p>Chemical (in vitro) labeling</p>
<ul>
<li>ICAT (isotope coded affinity tag)</li>
<li>iTRAQ (isobaric tag for relative and absolute quantification)</li>
</ul>
</li>
<li><p>Metabolic [or <em>in vivo</em>] labelling</p>
<ol>
<li>SILAC [stable isotope labelling by amino acids in culture]<ul>
<li>Requires a cell culture, therefore cannot be used for experiments from other kinds of samples e.g. clinical samples</li>
</ul>
</li>
</ol>
</li>
</ol>
<h2 id="Lecture-5-Post-translational-modifications"><a href="#Lecture-5-Post-translational-modifications" class="headerlink" title="Lecture 5_Post-translational modifications"></a>Lecture 5_Post-translational modifications</h2><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523161351.png" alt="image-20210523161351854"></p>
<h3 id="Proteomic-Analysis-for-PTMs"><a href="#Proteomic-Analysis-for-PTMs" class="headerlink" title="Proteomic Analysis for PTMs"></a>Proteomic Analysis for PTMs</h3><p>Key issue in PTM analysis:  modified proteins/peptides often have <strong><code>low abundance</code></strong></p>
<ul>
<li><p>**<code>Enrichment</code>**can be used to increase their concentration in the sample</p>
</li>
<li><p>The most established methodology is <strong>phosphoproteomics</strong></p>
</li>
<li><p>Development of phosphoproteomicshas been driven by the deep interest in signalling research (pharmaapplications)</p>
</li>
<li><p>Phosphopeptides are relatively easy to enrich and mass spectrometric methods have been well established, however, the enrichment outcomes are often <code>sample-dependent</code></p>
</li>
<li><p>The field is currently moving to high-throughput of profiling methods of other PTMs and the analysis of several PTMs in parallel</p>
</li>
</ul>
<h3 id="MS-for-Phosphopeptides"><a href="#MS-for-Phosphopeptides" class="headerlink" title="MS for Phosphopeptides"></a>MS for Phosphopeptides</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523163237.png" alt="image-20210523163237578"></p>
<ul>
<li>Precursor/ Reporter ion scanning</li>
<li>Neutral-Loss-Dependent <mjx-container class="MathJax" jax="SVG"><svg style="vertical-align: -0.05ex" xmlns="http://www.w3.org/2000/svg" width="5.027ex" height="1.645ex" role="img" focusable="false" viewBox="0 -705 2221.9 727"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g data-mml-node="math"><g data-mml-node="mi"><path data-c="1D440" d="M289 629Q289 635 232 637Q208 637 201 638T194 648Q194 649 196 659Q197 662 198 666T199 671T201 676T203 679T207 681T212 683T220 683T232 684Q238 684 262 684T307 683Q386 683 398 683T414 678Q415 674 451 396L487 117L510 154Q534 190 574 254T662 394Q837 673 839 675Q840 676 842 678T846 681L852 683H948Q965 683 988 683T1017 684Q1051 684 1051 673Q1051 668 1048 656T1045 643Q1041 637 1008 637Q968 636 957 634T939 623Q936 618 867 340T797 59Q797 55 798 54T805 50T822 48T855 46H886Q892 37 892 35Q892 19 885 5Q880 0 869 0Q864 0 828 1T736 2Q675 2 644 2T609 1Q592 1 592 11Q592 13 594 25Q598 41 602 43T625 46Q652 46 685 49Q699 52 704 61Q706 65 742 207T813 490T848 631L654 322Q458 10 453 5Q451 4 449 3Q444 0 433 0Q418 0 415 7Q413 11 374 317L335 624L267 354Q200 88 200 79Q206 46 272 46H282Q288 41 289 37T286 19Q282 3 278 1Q274 0 267 0Q265 0 255 0T221 1T157 2Q127 2 95 1T58 0Q43 0 39 2T35 11Q35 13 38 25T43 40Q45 46 65 46Q135 46 154 86Q158 92 223 354T289 629Z"></path></g><g data-mml-node="msup" transform="translate(1051, 0)"><g data-mml-node="mi"><path data-c="1D446" d="M308 24Q367 24 416 76T466 197Q466 260 414 284Q308 311 278 321T236 341Q176 383 176 462Q176 523 208 573T273 648Q302 673 343 688T407 704H418H425Q521 704 564 640Q565 640 577 653T603 682T623 704Q624 704 627 704T632 705Q645 705 645 698T617 577T585 459T569 456Q549 456 549 465Q549 471 550 475Q550 478 551 494T553 520Q553 554 544 579T526 616T501 641Q465 662 419 662Q362 662 313 616T263 510Q263 480 278 458T319 427Q323 425 389 408T456 390Q490 379 522 342T554 242Q554 216 546 186Q541 164 528 137T492 78T426 18T332 -20Q320 -22 298 -22Q199 -22 144 33L134 44L106 13Q83 -14 78 -18T65 -22Q52 -22 52 -14Q52 -11 110 221Q112 227 130 227H143Q149 221 149 216Q149 214 148 207T144 186T142 153Q144 114 160 87T203 47T255 29T308 24Z"></path></g><g data-mml-node="mi" transform="translate(696.6, 363) scale(0.707)"><path data-c="1D45B" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path></g></g></g></g></svg></mjx-container></li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523163411.png" alt="image-20210523163411638"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523163428.png" alt="image-20210523163428035"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523163439.png" alt="image-20210523163439522"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523170011.png" alt="image-20210523170011446"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523170023.png" alt="image-20210523170023740"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523170034.png"></p>
<h2 id="Targeted-Proteomics"><a href="#Targeted-Proteomics" class="headerlink" title="Targeted Proteomics"></a>Targeted Proteomics</h2><h3 id="Comparison-between-traditional-and-MS-based-protein-quantification"><a href="#Comparison-between-traditional-and-MS-based-protein-quantification" class="headerlink" title="Comparison between traditional and MS based protein quantification"></a>Comparison between traditional and MS based protein quantification</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605151040.png" alt="image-20210605151040333"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605151108.png" alt="image-20210605151108708"></p>
<h3 id="MRM-SRM"><a href="#MRM-SRM" class="headerlink" title="MRM / SRM"></a>MRM / SRM</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605151348.png" alt="image-20210605151348627"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605151400.png" alt="image-20210605151400638"></p>
<h3 id="Mass-Spec-based-Assay-Benefits"><a href="#Mass-Spec-based-Assay-Benefits" class="headerlink" title="Mass Spec-based Assay Benefits"></a>Mass Spec-based Assay Benefits</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605151441.png" alt="image-20210605151441467"></p>
<h3 id="Weakness-of-MRM-SRM"><a href="#Weakness-of-MRM-SRM" class="headerlink" title="Weakness of MRM/SRM"></a>Weakness of MRM/SRM</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605151504.png" alt="image-20210605151504601"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605151703.png" alt="image-20210605151703176"></p>

        </div>

        
            <div class="post-copyright-info">
                <div class="article-copyright-info-container">
    <ul>
        <li>Post title：BIO312 Wang&#39;s part</li>
        <li>Post author：Yuxuan Wu</li>
        <li>Create time：2021-03-02 19:44:01</li>
        <li>
            Post link：yuxuanwu17.github.io2021/03/02/2021-03-02-BIO312-Wangs-part/
        </li>
        <li>
            Copyright Notice：All articles in this blog are licensed under <a class="license" target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by-nc-sa/4.0/deed.zh">BY-NC-SA</a> unless stating additionally.
        </li>
    </ul>
</div>

            </div>
        

        
            <div class="article-nav">
                
                    <div class="article-prev">
                        <a class="prev"
                           rel="prev"
                           href="/2021/03/10/BIO312%20Wang%20Mu&#39;s%20CW/"
                        >
                            <span class="left arrow-icon flex-center">
                              <i class="fas fa-chevron-left"></i>
                            </span>
                            <span class="title flex-center">
                                <span class="post-nav-title-item">BIO312 Wang Mu&#39;s CW</span>
                                <span class="post-nav-item">Prev posts</span>
                            </span>
                        </a>
                    </div>
                
                
                    <div class="article-next">
                        <a class="next"
                           rel="next"
                           href="/2021/03/02/2021-03-02-BIO312-lees-part/"
                        >
                            <span class="title flex-center">
                                <span class="post-nav-title-item">BIO312 Lee&#39;s part</span>
                                <span class="post-nav-item">Next posts</span>
                            </span>
                            <span class="right arrow-icon flex-center">
                              <i class="fas fa-chevron-right"></i>
                            </span>
                        </a>
                    </div>
                
            </div>
        

        
    </div>
</div>


                
            </div>

        </div>

        <div class="page-main-content-bottom">
            <footer class="footer">
    <div class="info-container">
        <div class="copyright-info info-item">
            &copy;
            
              <span>2020</span> -
            
            2021 <i class="fas fa-heart icon-animate"></i> <a href="/">Yuxuan Wu</a>
        </div>
        
            <script async  src="//busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
            <div class="website-count info-item">
                
                    <span id="busuanzi_container_site_uv">
                        Visitor Count <span id="busuanzi_value_site_uv"></span>&ensp;
                    </span>
                
                
                    <span id="busuanzi_container_site_pv">
                        Totalview <span id="busuanzi_value_site_pv"></span>
                    </span>
                
            </div>
        
        <div class="theme-info info-item">
            Powered by <a target="_blank" href="https://hexo.io">Hexo</a> | Theme <a class="theme-version" target="_blank" href="https://github.com/XPoet/hexo-theme-keep">Keep v3.3.1</a>
        </div>
    </div>
</footer>

        </div>
    </div>

    
        <div class="post-tools">
            <div class="post-tools-container">
    <ul class="tools-list">
        <!-- TOC aside toggle -->
        
            <li class="tools-item page-aside-toggle">
                <i class="fas fa-outdent"></i>
            </li>
        

        <!-- go comment -->
        
    </ul>
</div>

        </div>
    

    <div class="right-bottom-side-tools">
        <div class="side-tools-container">
    <ul class="side-tools-list">
        <li class="tools-item tool-font-adjust-plus flex-center">
            <i class="fas fa-search-plus"></i>
        </li>

        <li class="tools-item tool-font-adjust-minus flex-center">
            <i class="fas fa-search-minus"></i>
        </li>

        <li class="tools-item tool-expand-width flex-center">
            <i class="fas fa-arrows-alt-h"></i>
        </li>

        <li class="tools-item tool-dark-light-toggle flex-center">
            <i class="fas fa-moon"></i>
        </li>

        <!-- rss -->
        

        

        <li class="tools-item tool-scroll-to-bottom flex-center">
            <i class="fas fa-arrow-down"></i>
        </li>
    </ul>

    <ul class="exposed-tools-list">
        <li class="tools-item tool-toggle-show flex-center">
            <i class="fas fa-cog fa-spin"></i>
        </li>
        
            <li class="tools-item tool-scroll-to-top flex-center">
                <i class="arrow-up fas fa-arrow-up"></i>
                <span class="percent"></span>
            </li>
        
    </ul>
</div>

    </div>

    
        <aside class="page-aside">
            <div class="post-toc-wrap">
    <div class="post-toc">
        <ol class="nav"><li class="nav-item nav-level-2"><a class="nav-link" href="#Revision-protein-structure-and-methods-of-analysis"><span class="nav-number">1.</span> <span class="nav-text">Revision protein structure and methods of analysis</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Techniques-for-protein-analysis"><span class="nav-number">1.1.</span> <span class="nav-text">Techniques for protein analysis</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-seperation"><span class="nav-number">1.2.</span> <span class="nav-text">Protein seperation</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#SDS-PAGE"><span class="nav-number">1.3.</span> <span class="nav-text">SDS-PAGE:</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Isoelectric-point-pi"><span class="nav-number">1.4.</span> <span class="nav-text">Isoelectric point (pi)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Isoelectric-Focusing"><span class="nav-number">1.5.</span> <span class="nav-text">Isoelectric Focusing</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#2D-gel"><span class="nav-number">1.6.</span> <span class="nav-text">2D gel</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Strength-and-weakness-of-the-2D-gel"><span class="nav-number">1.7.</span> <span class="nav-text">Strength and weakness of the 2D gel</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Western-Blot"><span class="nav-number">1.8.</span> <span class="nav-text">Western Blot</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Salting-out-precipitation-in-protein-fraction"><span class="nav-number">1.9.</span> <span class="nav-text">Salting out (precipitation in protein fraction)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Centrifugation"><span class="nav-number">1.10.</span> <span class="nav-text">Centrifugation</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#chromatography"><span class="nav-number">1.11.</span> <span class="nav-text">chromatography</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Gel-filtration-Size-exclusion-chromatography"><span class="nav-number">1.12.</span> <span class="nav-text">Gel filtration&#x2F;Size exclusion chromatography</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Ion-exchange-chromatography"><span class="nav-number">1.13.</span> <span class="nav-text">Ion exchange chromatography</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Hydrophobic-interaction-chromatography"><span class="nav-number">1.14.</span> <span class="nav-text">Hydrophobic interaction chromatography</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-sequencing"><span class="nav-number">1.15.</span> <span class="nav-text">Protein sequencing</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Cleavage-of-polypeptides-for-analysis"><span class="nav-number">1.16.</span> <span class="nav-text">Cleavage of polypeptides for analysis</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-identification"><span class="nav-number">1.17.</span> <span class="nav-text">Protein identification</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#Edman-degradation"><span class="nav-number">1.17.1.</span> <span class="nav-text">Edman degradation</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Mass-spectrometry"><span class="nav-number">1.17.2.</span> <span class="nav-text">Mass spectrometry</span></a></li></ol></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Mass-Spectrometry"><span class="nav-number">2.</span> <span class="nav-text">Mass Spectrometry:</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Workflow"><span class="nav-number">2.1.</span> <span class="nav-text">Workflow</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Types-of-MS-data"><span class="nav-number">2.2.</span> <span class="nav-text">Types of MS data</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#3D-structure-determination"><span class="nav-number">2.3.</span> <span class="nav-text">3D structure determination</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Identification-of-proteins-in-complex-mixtures"><span class="nav-number">3.</span> <span class="nav-text">Identification of proteins in complex mixtures</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Proteomics-achievement"><span class="nav-number">3.1.</span> <span class="nav-text">Proteomics achievement</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-Identification-by-Mass-Spectrometry"><span class="nav-number">3.2.</span> <span class="nav-text">Protein Identification by Mass Spectrometry</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Mass-Spec-MS-Basics"><span class="nav-number">3.3.</span> <span class="nav-text">Mass Spec (MS) Basics</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#MALDI"><span class="nav-number">3.3.1.</span> <span class="nav-text">MALDI</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Mass-resolution"><span class="nav-number">3.4.</span> <span class="nav-text">Mass resolution</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#LC-MS-MS-for-protein-identification"><span class="nav-number">3.5.</span> <span class="nav-text">LC-MS&#x2F;MS for protein identification</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Mass-spectrometer-is-always-right"><span class="nav-number">3.6.</span> <span class="nav-text">Mass spectrometer is always right</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Advanced-proteomic-techniques"><span class="nav-number">4.</span> <span class="nav-text">Advanced proteomic techniques</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Summary-of-existing-technology"><span class="nav-number">4.1.</span> <span class="nav-text">Summary of existing technology</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#ICAT"><span class="nav-number">4.2.</span> <span class="nav-text">ICAT</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Top-Down-or-Bottom-up-analysis"><span class="nav-number">4.3.</span> <span class="nav-text">Top-Down or Bottom-up analysis</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Top-down-advantages"><span class="nav-number">4.4.</span> <span class="nav-text">Top-down advantages</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#disadvantages-for-top-down"><span class="nav-number">4.5.</span> <span class="nav-text">disadvantages for top-down</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Quantitative-Mass-Spec-based-Proteomics"><span class="nav-number">5.</span> <span class="nav-text">Quantitative Mass Spec-based Proteomics</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Outline"><span class="nav-number">5.1.</span> <span class="nav-text">Outline</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Summary-of-label-methods"><span class="nav-number">5.2.</span> <span class="nav-text">Summary of label methods</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Pros-and-Cons-for-label-free-methods"><span class="nav-number">5.3.</span> <span class="nav-text">Pros and Cons for label-free methods</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#Methods"><span class="nav-number">5.3.1.</span> <span class="nav-text">Methods</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Spectral-counting"><span class="nav-number">5.4.</span> <span class="nav-text">Spectral counting</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Peptide-Ion-Intensity-Based-Protein-Quantification"><span class="nav-number">5.5.</span> <span class="nav-text">Peptide Ion Intensity Based Protein Quantification</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#LC-MS-Chromatographic-Alignment"><span class="nav-number">5.6.</span> <span class="nav-text">LC-MS Chromatographic Alignment</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Limitations-of-LC-MS-based-Approach-to-Large-scale-Protein-Profiling"><span class="nav-number">5.7.</span> <span class="nav-text">Limitations of LC-MS-based Approach to Large-scale Protein Profiling</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Challenges-in-proteomics"><span class="nav-number">5.8.</span> <span class="nav-text">Challenges in proteomics</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Tutorial-1"><span class="nav-number">6.</span> <span class="nav-text">Tutorial 1</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Proteomics-workflow"><span class="nav-number">6.1.</span> <span class="nav-text">Proteomics workflow</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Separation-technique"><span class="nav-number">6.2.</span> <span class="nav-text">Separation technique</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Mass-spectrometry-1"><span class="nav-number">6.3.</span> <span class="nav-text">Mass spectrometry</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Technique-summary"><span class="nav-number">6.4.</span> <span class="nav-text">Technique summary</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#PMF"><span class="nav-number">6.4.1.</span> <span class="nav-text">PMF</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#TMS"><span class="nav-number">6.5.</span> <span class="nav-text">TMS</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#The-principle-of-SDS-PAGE"><span class="nav-number">6.6.</span> <span class="nav-text">The principle of SDS-PAGE</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Western-plot"><span class="nav-number">6.7.</span> <span class="nav-text">Western plot</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Tutorial-2"><span class="nav-number">7.</span> <span class="nav-text">Tutorial 2</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Labeling-vs-non-labeling"><span class="nav-number">7.1.</span> <span class="nav-text">Labeling vs non-labeling</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Label-free-approaches"><span class="nav-number">7.2.</span> <span class="nav-text">Label free approaches</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#pros-and-cons-of-label-free-protein"><span class="nav-number">7.3.</span> <span class="nav-text">pros and cons of label free protein</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Why-does-MS1-intensity-not-tell-us-peptide-abundance-directly"><span class="nav-number">7.4.</span> <span class="nav-text">Why does MS1 intensity not tell us peptide abundance directly?</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Spectral-counting-1"><span class="nav-number">7.5.</span> <span class="nav-text">Spectral counting</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#The-normalization-of-spectral-counts"><span class="nav-number">7.6.</span> <span class="nav-text">The normalization of spectral counts</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Intensity-based-label-free"><span class="nav-number">7.7.</span> <span class="nav-text">Intensity based label free</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#Method-summary"><span class="nav-number">7.7.1.</span> <span class="nav-text">Method summary:</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Advantages"><span class="nav-number">7.7.2.</span> <span class="nav-text">Advantages:</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Disadvantages"><span class="nav-number">7.7.3.</span> <span class="nav-text">Disadvantages:</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Requirements"><span class="nav-number">7.7.4.</span> <span class="nav-text">Requirements:</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Labelling-quantification"><span class="nav-number">7.7.5.</span> <span class="nav-text">Labelling quantification</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Two-approaches"><span class="nav-number">7.7.6.</span> <span class="nav-text">Two approaches</span></a></li></ol></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-5-Post-translational-modifications"><span class="nav-number">8.</span> <span class="nav-text">Lecture 5_Post-translational modifications</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Proteomic-Analysis-for-PTMs"><span class="nav-number">8.1.</span> <span class="nav-text">Proteomic Analysis for PTMs</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#MS-for-Phosphopeptides"><span class="nav-number">8.2.</span> <span class="nav-text">MS for Phosphopeptides</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Targeted-Proteomics"><span class="nav-number">9.</span> <span class="nav-text">Targeted Proteomics</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Comparison-between-traditional-and-MS-based-protein-quantification"><span class="nav-number">9.1.</span> <span class="nav-text">Comparison between traditional and MS based protein quantification</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#MRM-SRM"><span class="nav-number">9.2.</span> <span class="nav-text">MRM &#x2F; SRM</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Mass-Spec-based-Assay-Benefits"><span class="nav-number">9.3.</span> <span class="nav-text">Mass Spec-based Assay Benefits</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Weakness-of-MRM-SRM"><span class="nav-number">9.4.</span> <span class="nav-text">Weakness of MRM&#x2F;SRM</span></a></li></ol></li></ol>
    </div>
</div>
        </aside>
    

    <div class="image-viewer-container">
    <img src="">
</div>


    
        <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
          <span class="search-input-field-pre">
            <i class="fas fa-keyboard"></i>
          </span>
            <div class="search-input-container">
                <input autocomplete="off"
                       autocorrect="off"
                       autocapitalize="off"
                       placeholder="Search..."
                       spellcheck="false"
                       type="search"
                       class="search-input"
                >
            </div>
            <span class="popup-btn-close">
                <i class="fas fa-times"></i>
            </span>
        </div>
        <div id="search-result">
            <div id="no-result">
                <i class="fas fa-spinner fa-pulse fa-5x fa-fw"></i>
            </div>
        </div>
    </div>
</div>

    

</main>




<script src="/js/utils.js"></script>

<script src="/js/main.js"></script>

<script src="/js/header-shrink.js"></script>

<script src="/js/back2top.js"></script>

<script src="/js/dark-light-toggle.js"></script>



    
<script src="/js/local-search.js"></script>




    
<script src="/js/code-copy.js"></script>





<div class="post-scripts">
    
        
<script src="/js/left-side-toggle.js"></script>

<script src="/js/libs/anime.min.js"></script>

<script src="/js/toc.js"></script>

    
</div>



</body>
</html>
