<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="keywords" content="Hexo Theme Keep">
    <meta name="description" content="Welcome to my personal blogs">
    <meta name="author" content="Yuxuan Wu">
    
    <title>
        
            BIO312 Lee&#39;s part |
        
        Yuxuan Wu
    </title>
    <link rel="shortcut icon" href="/images/logo.svg">
    
<link rel="stylesheet" href="/css/style.css">

    
<link rel="stylesheet" href="/css/font-awesome.min.css">

    <script id="hexo-configurations">
    let KEEP = window.KEEP || {};
    KEEP.hexo_config = {"hostname":"example.com","root":"/","language":"en","path":"search.xml"};
    KEEP.theme_config = {"toc":{"enable":true,"number":true,"expand_all":false},"style":{"primary_color":"#0066CC","avatar":"/images/avatar.svg","favicon":"/images/logo.svg","article_img_align":"left","left_side_width":"260px","content_max_width":"920px","hover":{"shadow":true,"scale":true},"first_screen":{"enable":true,"background_img":"/images/bg.svg","description":"Keep coding, Keep hungry."},"scroll":{"progress_bar":{"enable":true},"percent":{"enable":true}}},"local_search":{"enable":true,"preload":true},"code_copy":{"enable":true,"style":"default"},"pjax":{"enable":false},"lazyload":{"enable":false},"version":"3.3.1"};
    KEEP.language_ago = {"second":"%s seconds ago","minute":"%s minutes ago","hour":"%s hours ago","day":"%s days age","week":"%s weeks age","month":"%s months age","year":"%s years age"};
  </script>
<!-- hexo injector head_end start -->
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css">

<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/hexo-math@4.0.0/dist/style.css">
<!-- hexo injector head_end end --><meta name="generator" content="Hexo 5.3.0"><style>mjx-container[jax="SVG"] {
  direction: ltr;
}

mjx-container[jax="SVG"] > svg {
  overflow: visible;
}

mjx-container[jax="SVG"] > svg a {
  fill: blue;
  stroke: blue;
}

mjx-container[jax="SVG"][display="true"] {
  display: block;
  text-align: center;
  margin: 1em 0;
}

mjx-container[jax="SVG"][justify="left"] {
  text-align: left;
}

mjx-container[jax="SVG"][justify="right"] {
  text-align: right;
}

g[data-mml-node="merror"] > g {
  fill: red;
  stroke: red;
}

g[data-mml-node="merror"] > rect[data-background] {
  fill: yellow;
  stroke: none;
}

g[data-mml-node="mtable"] > line[data-line] {
  stroke-width: 70px;
  fill: none;
}

g[data-mml-node="mtable"] > rect[data-frame] {
  stroke-width: 70px;
  fill: none;
}

g[data-mml-node="mtable"] > .mjx-dashed {
  stroke-dasharray: 140;
}

g[data-mml-node="mtable"] > .mjx-dotted {
  stroke-linecap: round;
  stroke-dasharray: 0,140;
}

g[data-mml-node="mtable"] > svg {
  overflow: visible;
}

[jax="SVG"] mjx-tool {
  display: inline-block;
  position: relative;
  width: 0;
  height: 0;
}

[jax="SVG"] mjx-tool > mjx-tip {
  position: absolute;
  top: 0;
  left: 0;
}

mjx-tool > mjx-tip {
  display: inline-block;
  padding: .2em;
  border: 1px solid #888;
  font-size: 70%;
  background-color: #F8F8F8;
  color: black;
  box-shadow: 2px 2px 5px #AAAAAA;
}

g[data-mml-node="maction"][data-toggle] {
  cursor: pointer;
}

mjx-status {
  display: block;
  position: fixed;
  left: 1em;
  bottom: 1em;
  min-width: 25%;
  padding: .2em .4em;
  border: 1px solid #888;
  font-size: 90%;
  background-color: #F8F8F8;
  color: black;
}

foreignObject[data-mjx-xml] {
  font-family: initial;
  line-height: normal;
  overflow: visible;
}

.MathJax path {
  stroke-width: 3;
}

mjx-container[display="true"] {
  overflow: auto hidden;
}

mjx-container[display="true"] + br {
  display: none;
}
</style></head>


<body>
<div class="progress-bar-container">
    
        <span class="scroll-progress-bar"></span>
    

    
</div>


<main class="page-container">

    

    <div class="page-main-content">

        <div class="page-main-content-top">
            <header class="header-wrapper">

    <div class="header-content">
        <div class="left">
            <a class="logo-title" href="/">
                Yuxuan Wu
            </a>
        </div>

        <div class="right">
            <div class="pc">
                <ul class="menu-list">
                    
                        <li class="menu-item">
                            <a class=""
                               href="/"
                            >
                                HOME
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/archives"
                            >
                                ARCHIVES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/categories"
                            >
                                CATEGORIES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/tags"
                            >
                                TAGS
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/about"
                            >
                                ABOUT
                            </a>
                        </li>
                    
                    
                        <li class="menu-item search search-popup-trigger">
                            <i class="fas fa-search"></i>
                        </li>
                    
                </ul>
            </div>
            <div class="mobile">
                
                    <div class="icon-item search search-popup-trigger"><i class="fas fa-search"></i></div>
                
                <div class="icon-item menu-bar">
                    <div class="menu-bar-middle"></div>
                </div>
            </div>
        </div>
    </div>

    <div class="header-drawer">
        <ul class="drawer-menu-list">
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/">HOME</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/archives">ARCHIVES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/categories">CATEGORIES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/tags">TAGS</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/about">ABOUT</a>
                </li>
            
        </ul>
    </div>

    <div class="window-mask"></div>

</header>


        </div>

        <div class="page-main-content-middle">

            <div class="main-content normal-code-theme">

                
                    <div class="fade-in-down-animation">
    <div class="article-content-container">

        <div class="article-title">
            <span class="title-hover-animation">BIO312 Lee&#39;s part</span>
        </div>

        
            <div class="article-header">
                <div class="avatar">
                    <img src="/images/avatar.svg">
                </div>
                <div class="info">
                    <div class="author">
                        <span class="name">Yuxuan Wu</span>
                        <span class="level">Lv12</span>
                    </div>
                    <div class="meta-info">
                        <div class="article-meta-info">
    <span class="article-date article-meta-item">
        <i class="fas fa-edit"></i> 2021-03-02 06:44:01
    </span>
    
        <span class="article-categories article-meta-item">
            <i class="fas fa-folder"></i>
            <ul>
                
                    <li>
                        <a href="/categories/Biology/">Biology</a>
                    </li>
                
            </ul>
        </span>
    
    
        <span class="article-tags article-meta-item">
            <i class="fas fa-tags"></i>
            <ul>
                
                    <li>
                        <a href="/tags/notes/">notes</a>
                    </li>
                
            </ul>
        </span>
    

    
    
        <span class="article-wordcount article-meta-item">
            <i class="fas fa-file-word"></i> <span>4.6k Words</span>
        </span>
    
    
        <span class="article-min2read article-meta-item">
            <i class="fas fa-clock"></i> <span>28 Mins</span>
        </span>
    
    
        <span class="article-pv article-meta-item">
            <i class="fas fa-eye"></i> <span id="busuanzi_value_page_pv"></span>
        </span>
    
</div>

                    </div>
                </div>
            </div>
        

        <div class="article-content markdown-body">
            <p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210521135014.png" alt="image-20210521135014278"></p>
<p>[toc]</p>
<h2 id="Amino-acids-structure"><a href="#Amino-acids-structure" class="headerlink" title="Amino acids structure"></a>Amino acids structure</h2><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210601182146.svg" alt="Amino_Acids"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210606152747.jpeg" alt="img"></p>
<h2 id="Lecture-09-Folding-in-vivo"><a href="#Lecture-09-Folding-in-vivo" class="headerlink" title="Lecture 09 Folding in vivo"></a>Lecture 09 Folding in vivo</h2><h3 id="protein-folding-in-the-cell"><a href="#protein-folding-in-the-cell" class="headerlink" title="protein folding in the cell"></a>protein folding in the cell</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210604125225.png" alt="image-20210604125225127"></p>
<h3 id="Folding-in-vitro-and-in-vivo"><a href="#Folding-in-vitro-and-in-vivo" class="headerlink" title="Folding in vitro and in vivo"></a>Folding in vitro and in vivo</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523171332.png" alt="image-20210523171332137"></p>
<h3 id="Macromolecular-crowding"><a href="#Macromolecular-crowding" class="headerlink" title="Macromolecular crowding"></a>Macromolecular crowding</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523171412.png" alt="image-20210523171412425"></p>
<h3 id="Effects-of-crowding"><a href="#Effects-of-crowding" class="headerlink" title="Effects of crowding"></a>Effects of crowding</h3><p><strong>definition</strong></p>
<p>Molecular crowdingis ageneric term for the condition where a significant volume of a solution, or cytoplasm for example, is occupied with things other than water</p>
<p>Fact:</p>
<p>-association rate (k<strong>a</strong>) increase significantly</p>
<p>-dissociation rate (k<strong>d</strong>) decrease significantly (k<strong>d</strong>=1/k<strong>a</strong>)</p>
<h3 id="Molecular-chaperones"><a href="#Molecular-chaperones" class="headerlink" title="Molecular chaperones"></a>Molecular chaperones</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523171644.png" alt="image-20210523171644917"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210523171653.png" alt="image-20210523171653069"></p>
<h2 id="Lecture-10-Folding-in-vitro"><a href="#Lecture-10-Folding-in-vitro" class="headerlink" title="Lecture 10 Folding in vitro"></a>Lecture 10 Folding in vitro</h2><h3 id="The-Levinthal-Paradox"><a href="#The-Levinthal-Paradox" class="headerlink" title="The Levinthal Paradox"></a>The Levinthal Paradox</h3><p><code>Conclusion:</code> Protein folding is not random, must have pathways.</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525215304.png" alt="image-20210525215304583"></p>
<h3 id="Equilibrium-unfolding"><a href="#Equilibrium-unfolding" class="headerlink" title="Equilibrium unfolding"></a>Equilibrium unfolding</h3><ul>
<li><p>switch off part of the interactions in the native protein under different denaturing conditions such as chemical denaturants, low pH, high salt and high temperature</p>
</li>
<li><p>understand which types of native structure can be preserved by the remaining interactions</p>
</li>
</ul>
<h3 id="Anfinsen-Experiment"><a href="#Anfinsen-Experiment" class="headerlink" title="Anfinsen Experiment"></a>Anfinsen Experiment</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210531200013.png" alt="image-20210531200013719"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525215803.png" alt="image-20210525215803427"></p>
<blockquote>
<p>Conclusion: All the information necessary for folding the peptide chain into its native structure is contained in the primary amino acid sequence of the peptide.</p>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525215922.png" alt="image-20210525215922335"></p>
<blockquote>
<p> Conclusion: The native form of a protein has the thermodynamically most stable structure.</p>
</blockquote>
<h3 id="Information-online"><a href="#Information-online" class="headerlink" title="Information online"></a>Information online</h3><p>Christian’s plan was to use the appropriate denaturing agents, namely urea and beta mercaptoethanol, to break down the secondary and tertiary structure of ribonuclease, a polypeptide that consists of 124 amino acids and which contains four disulfide bonds within its tertiary structure.</p>
<p>Urea agent is used to break down non-covalent bonds such as hydrogen bonds holding the secondary structure</p>
<p>beta-mercaptoethanol: use to reduce and break down the disulfide bonds holding the tertiary structure together. </p>
<p>In his first experiment, Christian exposed the native enzyme to excess beta mercaptoethanol and 8.0 M urea and he found that the protein was completely denatured. </p>
<p>When he removed the two agents simultaneously via dialysis, he found that the protein refolded back into its original biologically active form. </p>
<p>In his second experiment, instead of removing the two agents at the same time, he first removed the beta mercaptoethanol first and then removed the urea. What he discovered was that the final protein refolded but became scrambled and was no longer biologically active. </p>
<p>This happened because the non-covalent bonds could not form in the presence of urea and so the disulfide bonds formed incorrectly (the non-covalent bonds coded by the primary sequence of amino acids are needed to direct the correct formation of the disulfide bonds). </p>
<p>In his third experiment, he found that if he exposed the scrambled, inactive protein to trace amounts of beta mercaptoethanol in the absence of urea, the biologically active native structure eventually reformed. Why did this happen? Well the tiny amount of beta mercaptoethanol was enough to catalyze the breaking of the incorrect disulfide bonds. Eventually the protein formed the correct disulfide bridges and returned to its native form because this was thermodynamically most stable and lowest in energy form.</p>
<h3 id="quiz"><a href="#quiz" class="headerlink" title="quiz"></a>quiz</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210531200518.png" alt="image-20210531200518786"></p>
<blockquote>
<p>Experiment A: 45:23s</p>
<ul>
<li>Remove urea</li>
<li>Oxidize </li>
</ul>
<p>没有听清，需要笔记</p>
<p> <mjx-container class="MathJax" jax="SVG"><svg style="vertical-align: -0.439ex" xmlns="http://www.w3.org/2000/svg" width="3.041ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 1344 899"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g data-mml-node="math"><g data-mml-node="mi"><path data-c="1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></g><g data-mml-node="mo" transform="translate(566, 0)"><path data-c="2212" d="M84 237T84 250T98 270H679Q694 262 694 250T679 230H98Q84 237 84 250Z"></path></g></g></g></svg></mjx-container>mercaptoethanol: is used to reduce disulfide bonds and can act as a biological antioxidant by scavenging hydroxyl radicals (amongst others).</p>
<p>Ribonuclease: Ribonuclease is a type of nuclease that catalyzes the degradation of RNA into smaller components.</p>
<p>The whole process could be the reverse of previous figures</p>
<p>native protein with disulfide bond -&gt; add <mjx-container class="MathJax" jax="SVG"><svg style="vertical-align: -0.439ex" xmlns="http://www.w3.org/2000/svg" width="3.041ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 1344 899"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g data-mml-node="math"><g data-mml-node="mi"><path data-c="1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></g><g data-mml-node="mo" transform="translate(566, 0)"><path data-c="2212" d="M84 237T84 250T98 270H679Q694 262 694 250T679 230H98Q84 237 84 250Z"></path></g></g></g></svg></mjx-container>mercaptoethanol -&gt; break disulfide bond -&gt; add urine -&gt; convert the secondary or tertiary structure to primary structure -&gt; it will  become the correct configuration</p>
<p>If we remove urea in the primary sequence, the hydrogen bond could not be formed and thus disulfide bond could be influenced and break the bond randomly, the structure has already be constrained</p>
<p>Experiment B:</p>
<p>就是Anfinsen Experiment 里写的那样 (随机组合) will never become native form (先形成二硫键，然后protein的结构被束缚constrained，所以没有activity)</p>
<p>Reform the correct structure gradually (because the peptide could find its own way to reform)</p>
<p>Nobel price:”for his work on ribonuclease, especially concerning the connection between the amino acid sequence and the biologically active conformation”</p>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210531221016.png" alt="image-20210531221015959"></p>
<h3 id="Molten-Globule-State-MG"><a href="#Molten-Globule-State-MG" class="headerlink" title="Molten Globule State (MG)"></a>Molten Globule State (MG)</h3><ul>
<li>It is an intermediate of the folding transition U→MG→F</li>
<li>It is a compact globule, yet expanded over a native radius</li>
<li>Native-like secondary structure, can be measured by CD and NMR proton exchange rate</li>
<li>It has a slowly fluctuating tertiary structure which gives no detectable near UV CD signal and gives quenched fluorescence signal with broadened NMR chemical peaks</li>
<li>Non-specific assembly of secondary structure and hydrophobic interactions, which allows ANS to bind and gives an enhanced ANS fluorescence </li>
<li>MG is about a 10 % increase in size than the native state</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525220949.png" alt="image-20210525220948934"></p>
<h3 id="ANS-as-a-hydrophobicity-measuring"><a href="#ANS-as-a-hydrophobicity-measuring" class="headerlink" title="ANS as a hydrophobicity measuring"></a>ANS as a hydrophobicity measuring</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210606092851.png" alt="image-20210606092851725"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525221041.png" alt="image-20210525221041197"></p>
<h3 id="Protein-melting-thermal-shift"><a href="#Protein-melting-thermal-shift" class="headerlink" title="Protein melting (thermal shift)"></a>Protein melting (thermal shift)</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525221117.png" alt="image-20210525221117616"></p>
<p><strong>Why intensity of fluorescence goes down as temperature goes up?</strong></p>
<p>The protein would become precipitate out and become denature protein, no longer soluble (70-90 degree)</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210601081323.png" alt="image-20210601081323262"></p>
<p>ph5.5: the protein cell does like this ph, the protein already semi-denatured. So the core of the residue would be exposed. Even at room temperature, the protein already semi-denatured. So increase the PH to suitable</p>
<p>Ph8.5 In the second part (&gt;75), the scope is very shallow, suggesting (sharp) precipitation (aggregate) already. So when the temperature increased, there is not much protein to unfold, denature anymore.</p>
<p><code>Fluorescence</code>:</p>
<ul>
<li>Would bind to the hydrophobic residues</li>
<li>Normally it will not fluorescent until the protein is denatured and exposed to the environment</li>
</ul>
<p>Fluorescence will not decay, quite stable</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210601084438.png" alt="image-20210601084438550"></p>
<p>Apo-enzyme: without any ligands （control）</p>
<p>D is for ligand 1 since it has the highest affinity </p>
<p>A is apo-enzyme (without any ligand)</p>
<p>(iii) depends on the solubility of protein samples. HCL denatured the protein completel</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210601094656.png" alt="image-20210601094656903"></p>
<h3 id="Unfolded-state"><a href="#Unfolded-state" class="headerlink" title="Unfolded state"></a>Unfolded state</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210601094813.png" alt="image-20210601094813193"></p>
<h3 id="Three-classic-models-of-protein-folding"><a href="#Three-classic-models-of-protein-folding" class="headerlink" title="Three classic models of protein folding"></a>Three classic models of protein folding</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210601095026.png" alt="image-20210601095025958"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210601095404.png" alt="image-20210601095404860"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210601095253.png" alt="image-20210601095253895"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210601095323.png" alt="image-20210601095323424"></p>
<h3 id="The-folding-funnel"><a href="#The-folding-funnel" class="headerlink" title="The folding funnel"></a>The folding funnel</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210601095743.png" alt="image-20210601095743243"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210601095806.png" alt="image-20210601095806567"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210601095818.png" alt="image-20210601095818520"></p>
<h2 id="Lecture-11-SPR"><a href="#Lecture-11-SPR" class="headerlink" title="Lecture 11 SPR"></a>Lecture 11 SPR</h2><h3 id="Surface-Plasmon-Resonance"><a href="#Surface-Plasmon-Resonance" class="headerlink" title="Surface Plasmon Resonance"></a>Surface Plasmon Resonance</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210528105445.png" alt="image-20210528105445039"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210528105559.png" alt="image-20210528105559842"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210528110144.png" alt="image-20210528110144896"></p>
<h3 id="pros-amp-cons"><a href="#pros-amp-cons" class="headerlink" title="pros & cons"></a>pros &amp; cons</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210528110232.png" alt="image-20210528110232662"></p>
<h3 id="Binding-affinity"><a href="#Binding-affinity" class="headerlink" title="Binding affinity"></a>Binding affinity</h3><p>Binding affinity is the strength of the binding interaction between a single biomolecule (e.g. protein or DNA) to its ligand/binding partner (e.g. drug or inhibitor). Binding affinity is typically measured and reported by the equilibrium dissociation constant (KD), which is used to evaluate and rank order strengths of bimolecular interactions. The smaller the KD value, the greater the binding affinity of the ligand for its target. The larger the KD value, the more weakly the target molecule and ligand are attracted to and bind to one another. </p>
<p><strong>Binding affinity is influenced by non-covalent intermolecular interactions such as hydrogen bonding, electrostatic interactions, hydrophobic and Van der Waals forces between the two molecules. In addition, binding affinity between a ligand and its target molecule may be affected by the presence of other molecules.</strong></p>
<h3 id="Tutorial"><a href="#Tutorial" class="headerlink" title="Tutorial"></a>Tutorial</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210528110725.png" alt="image-20210528110725405"></p>
<p>Inhibitaor A has a low ki value</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210528112625.png" alt="image-20210528112625677"></p>
<p><code>Question (a)</code></p>
<p>The bigger the molecule binding to the chip, the higher the shift of response you will see in the signal. </p>
<p>A: 1000 response unit – antibody</p>
<p>B: small molecular inhibitor </p>
<p><code>Question (b)</code></p>
<p>For antibody, it takes a long time (configuration and takes to time to stable for binding)</p>
<p>For small molecular, immediately (can go straight to the molecule site, when you wash it, everything come out, nothing changed in the conformation of the size)</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210528113746.png" alt="image-20210528113746160"></p>
<p>a) lower kD value means higher affinity</p>
<blockquote>
<p>Which one is better in drug design ?</p>
</blockquote>
<p>A long type binding, bind very tightly and let drug stay in the body for a long time. Avoid take pill regularly (short term problem)</p>
<p>But based on the profile of the drug design, you have take care of the drug significantly</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210528114811.png" alt="image-20210528114811051   "></p>
<p>The answer should be Ms</p>
<h2 id="Lecture-12-Crystallography"><a href="#Lecture-12-Crystallography" class="headerlink" title="Lecture 12 Crystallography"></a>Lecture 12 Crystallography</h2><h3 id="Whole-process"><a href="#Whole-process" class="headerlink" title="Whole process"></a>Whole process</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524083431.png" alt="image-20210524083431460"></p>
<h3 id="X-ray-diffraction-of-a-crystal"><a href="#X-ray-diffraction-of-a-crystal" class="headerlink" title="X-ray diffraction of a crystal"></a>X-ray diffraction of a crystal</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524083522.png" alt="image-20210524083522318"></p>
<h3 id="Raw-data"><a href="#Raw-data" class="headerlink" title="Raw data"></a>Raw data</h3><p>Every atom in a unit cell contributes to every reflection in the diffraction pattern.</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524083728.png" alt="image-20210524083727937"></p>
<p>Each diffraction spot (reflection) contains information on the position of every atom!</p>
<p><strong>The degree of order in the crystal determines the quality of the diffraction data and ultimately the quality of the final atomic model</strong></p>
<p>“low resolution”</p>
<p>“high resolution”</p>
<h3 id="From-diffraction-to-electron-density-map"><a href="#From-diffraction-to-electron-density-map" class="headerlink" title="From diffraction to electron density map"></a>From diffraction to electron density map</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524084221.png" alt="image-20210524084221744"></p>
<h3 id="Solving-the-phase-problem"><a href="#Solving-the-phase-problem" class="headerlink" title="Solving the phase problem"></a>Solving the phase problem</h3><ul>
<li>The diffraction data <code>does not give the phase angle </code> that is needed to calculate the electron density map.</li>
<li>Have to <code>get the phase angle through other methods</code>. <ul>
<li>Isomorphous Replacement: <code>Insert a heavy metal atom into crystal protein</code>, and <code>locate in diffraction pattern</code> and in the cell. Use the location of metal ion to find the phase angle for the other protein atoms.</li>
</ul>
</li>
<li>Requirements:<ul>
<li>Add atom with same unit cell size.</li>
<li>Cannot disturb protein structure.</li>
</ul>
</li>
</ul>
<h3 id="Phase-problem"><a href="#Phase-problem" class="headerlink" title="Phase problem"></a>Phase problem</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524085508.png" alt="image-20210524085508442"> </p>
<h3 id="Resolution"><a href="#Resolution" class="headerlink" title="Resolution"></a>Resolution</h3><blockquote>
<p>Resolution:</p>
<p>another measure of how good your model is.</p>
</blockquote>
<ul>
<li>Resolution gives the size of the smallest molecule you can see or resolve.  </li>
<li>Dependent on the amount of data ultimately phased and used in structure determination.</li>
</ul>
<h3 id="Limitations-of-the-phase-problem"><a href="#Limitations-of-the-phase-problem" class="headerlink" title="Limitations of the phase problem"></a>Limitations of the phase problem</h3><ul>
<li><p>Crystallizing Protein:</p>
<ul>
<li>Fragile</li>
<li>Requires a crystal with shortest side 0.2 mm</li>
</ul>
</li>
<li><p>Flaws of Crystallization:</p>
<ul>
<li>Disorder in Unit Cell</li>
<li>Vibrations of molecules</li>
<li>Distortion in Crystallization</li>
</ul>
</li>
</ul>
<h3 id="Steps-of-protein-X-ray-crystallography"><a href="#Steps-of-protein-X-ray-crystallography" class="headerlink" title="Steps of protein X-ray crystallography"></a>Steps of protein X-ray crystallography</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524085954.png" alt="image-20210524085954600"></p>
<h3 id="R-factor-measuring-convergence"><a href="#R-factor-measuring-convergence" class="headerlink" title="R-factor: measuring convergence"></a>R-factor: measuring convergence</h3><ul>
<li><p>To compare the generated electron density map and your model, you have to use the R-factor (Reliability Factor)</p>
</li>
<li><p>Measurement of the agreement between the crystallographic model and the experimental X-ray diffraction ie. how well the refined structure predicts the observed data</p>
</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524090443.png" alt="image-20210524090443676"></p>
<h3 id="Occupancy"><a href="#Occupancy" class="headerlink" title="Occupancy"></a>Occupancy</h3><p>The occupancy “nj” of atom “j”: </p>
<blockquote>
<p>a measurement of the fraction of molecules in the crystal in which atom j actually occupies the position specified in the model. </p>
</blockquote>
<ul>
<li><p>If all molecules in the crystal are precisely identical, then occupancies for all atoms are 1.00. </p>
</li>
<li><p>Occupancy is necessary because occasionally two or more distinct conformations are observed for a small region like a surface side chain. For example, if the <strong>two conformations occur with equal frequency, then atoms involved receive occupancies of 0.5 in each of their two possible positions</strong>. </p>
</li>
<li><p>So, “occupancies” is an estimates of the frequency of alternative conformations, giving some additional information about the dynamics of the protein molecule. </p>
</li>
</ul>
<h3 id="Redundancy"><a href="#Redundancy" class="headerlink" title="Redundancy"></a>Redundancy</h3><p>Redundancy = (number of measured reflections) / (number of unique reflections).</p>
<ul>
<li><p>The calculation gives the <strong>average number of independent measurements of each reflection</strong> in a crystallographic data set. </p>
</li>
<li><p>Because several independent measurements are done for each reflection. </p>
</li>
<li><p>To improve accuracy in measuring reflection intensities, data collection strategies are intentionally designed to take advantage of symmetry and overlap to give redundancy of measurement. </p>
</li>
<li><p>Such statistical parameters as standard deviation are used to measure agreement among the repeated measurements.</p>
</li>
</ul>
<h3 id="Resolution-1"><a href="#Resolution-1" class="headerlink" title="Resolution"></a>Resolution</h3><ul>
<li><p>Reflect the quality of the image.</p>
</li>
<li><p>In X-ray crystallography, “2-Å model” means that the model takes into account diffraction from sets of equivalent, parallel planes of atoms spaced as closely as 2 Å in the unit cell. </p>
</li>
</ul>
<h3 id="“B-Factor-Temperature-factor-”"><a href="#“B-Factor-Temperature-factor-”" class="headerlink" title="“B-Factor (Temperature factor)”"></a>“B-Factor (Temperature factor)”</h3><ul>
<li><p>B-factor can be thought of as a measure of <code>how much an atom oscillates or vibrates</code> around the position specified in the model.</p>
</li>
<li><p>Atoms at <code>side-chain termini</code> are expected to <code>exhibit more freedom of movement</code> than main-chain atoms, and this movement amounts to spreading each atom over a small region of space. </p>
</li>
<li><p>Diffraction is affected by this variation in atomic position.</p>
</li>
<li><p>From the <code>temperature factors</code> computed during refinement, we learn which atoms in the molecule have the most freedom of movement, and we gain some insight into the dynamics of our largely static model. </p>
</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524174928.png" alt="image-20210524174928076"></p>
<h3 id="Two-types-of-solvent-ordered-and-disordered"><a href="#Two-types-of-solvent-ordered-and-disordered" class="headerlink" title="Two types of solvent: ordered and disordered"></a>Two types of solvent: ordered and disordered</h3><ul>
<li><p><code>Ordered water</code> molecules show up as discrete blobs of electron density in contact with the protein or with other ordered water molecules</p>
</li>
<li><p><code>Disordered water</code> regions show up as featureless (flat) electron density</p>
</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524174908.png" alt="image-20210524174908366"></p>
<h3 id="crystals-and-crystal-in-crystallographic-data"><a href="#crystals-and-crystal-in-crystallographic-data" class="headerlink" title="crystals and crystal in crystallographic data"></a>crystals and crystal in crystallographic data</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210531181542.png" alt="image-20210531181542011"></p>
<h3 id="Quiz"><a href="#Quiz" class="headerlink" title="Quiz"></a>Quiz</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210603173824.png" alt="image-20210603173824297"></p>
<p>左图：lousy;  may precipitate; not comfortable </p>
<p>右图：comfortable protein </p>
<h3 id=""><a href="#" class="headerlink" title=""></a></h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210603173915.png" alt="image-20210603173915437"></p>
<p>Asp开始从左到右O-N-O</p>
<p>Asparagine and glutamine side-chains can form hydrogen-bonded ladders which contribute significantly to the stability of amyloid fibrils.</p>
<h2 id="Lecture-13-14-NMR"><a href="#Lecture-13-14-NMR" class="headerlink" title="Lecture 13  14 NMR"></a>Lecture 13  14 NMR</h2><h3 id="What-is-NRM"><a href="#What-is-NRM" class="headerlink" title="What is NRM"></a>What is NRM</h3><p><strong>Nuclear magnetic resonance</strong> (<strong>NMR</strong>) is a physical phenomenon in which nuclei in a strong constant magnetic field are perturbed by a weak oscillating magnetic field (in the near field[1]) and respond by producing an electromagnetic signal with a frequency characteristic of the magnetic field at the nucleus.</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525184805.png" alt="image-20210525184805350"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525184839.png" alt="image-20210525184839551"></p>
<h3 id="used-to-identity-peptide-amide-chain"><a href="#used-to-identity-peptide-amide-chain" class="headerlink" title="used to identity peptide amide chain"></a>used to identity peptide amide chain</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605104558.png" alt="image-20210605104558426"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525191917.png" alt="image-20210525191917854"></p>
<h3 id="demo-in-class"><a href="#demo-in-class" class="headerlink" title="demo in class"></a>demo in class</h3><p><code>answer a:</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525193314.png" alt="image-20210525193314205"></p>
<p><code>answer b:</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525193910.png" alt="image-20210525193910236"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525194221.png" alt="image-20210525194221673"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525194936.png" alt="image-20210525194936354"></p>
<h3 id="Types-of-triple-resonance"><a href="#Types-of-triple-resonance" class="headerlink" title="Types of triple resonance"></a>Types of triple resonance</h3><p><code>Nomenclature: </code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526105913.png" alt="image-20210526105913534"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526110441.png" alt="image-20210526110441526"></p>
<h3 id="Protein-dynamics"><a href="#Protein-dynamics" class="headerlink" title="Protein dynamics"></a>Protein dynamics</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526111144.png" alt="image-20210526111144037"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526111414.png" alt="image-20210526111414430"></p>
<h3 id="NMR-shows-change-in-configuration"><a href="#NMR-shows-change-in-configuration" class="headerlink" title="NMR: shows change in configuration"></a>NMR: shows change in configuration</h3><p><code>mutations can effect folding and stability </code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526113844.png" alt="image-20210526113844230"></p>
<h3 id="Binding-constants"><a href="#Binding-constants" class="headerlink" title="Binding constants"></a>Binding constants</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526113918.png" alt="image-20210526113918581"></p>
<h3 id="Pros-and-cons-of-NRM-in-structural-biology"><a href="#Pros-and-cons-of-NRM-in-structural-biology" class="headerlink" title="Pros and cons of NRM in structural biology"></a>Pros and cons of NRM in structural biology</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605105612.png" alt="image-20210605105612179"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605110353.png" alt="image-20210605110352947"></p>
<h2 id="Tutorial-on-NMR"><a href="#Tutorial-on-NMR" class="headerlink" title="Tutorial on NMR"></a>Tutorial on NMR</h2><h3 id="Question-1"><a href="#Question-1" class="headerlink" title="Question 1"></a>Question 1</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526114257.png" alt="image-20210526114257882"></p>
<p><code>answer</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526114513.png" alt="image-20210526114513266"></p>
<h3 id="Question-2"><a href="#Question-2" class="headerlink" title="Question 2"></a>Question 2</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526115416.png" alt="image-20210526115416672"></p>
<p><code>answer</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526115618.png" alt="image-20210526115618544"></p>
<h3 id="Question-3"><a href="#Question-3" class="headerlink" title="Question 3"></a>Question 3</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526120145.png" alt="image-20210526120145040"></p>
<p><code>answer</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210526120446.png" alt="image-20210526120446037"></p>
<h2 id="Lecture-15-CD"><a href="#Lecture-15-CD" class="headerlink" title="Lecture 15 CD"></a>Lecture 15 CD</h2><h3 id="Circular-Polarized-Light"><a href="#Circular-Polarized-Light" class="headerlink" title="Circular Polarized Light"></a>Circular Polarized Light</h3><p>Nearly all molecules synthesized by living organisms are <code>optically active</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524212852.png" alt="image-20210524212852153"></p>
<h3 id="Circular-Polarized-Light-1"><a href="#Circular-Polarized-Light-1" class="headerlink" title="Circular Polarized Light"></a>Circular Polarized Light</h3><p><code>Ellipticity</code></p>
<p>Plane polarized light is the sum of **left **and **right **circularly polarized light. </p>
<ul>
<li><p>When plane polarized light passes through a solution containing an optically active substance the left and right circularly polarized components of the plane-polarized light are <code>absorbed by different amounts. </code></p>
</li>
<li><p>When these components are recombined they appear as elliptically polarized light. The ellipticityis defined as Q, the angle of polarization and is measured in degrees, (deg) or millidegrees, (mdeg).</p>
</li>
</ul>
<h3 id="CD-signal-of-proteins"><a href="#CD-signal-of-proteins" class="headerlink" title="CD signal of proteins"></a>CD signal of proteins</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524213120.png" alt="image-20210524213120909"></p>
<h3 id="Standard-curves"><a href="#Standard-curves" class="headerlink" title="Standard curves"></a>Standard curves</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524213353.png" alt="image-20210524213353329"></p>
<h3 id="Using-CD-to-Monitor-tertiary-structure"><a href="#Using-CD-to-Monitor-tertiary-structure" class="headerlink" title="Using CD to Monitor tertiary structure"></a>Using CD to Monitor tertiary structure</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210605113925.png" alt="image-20210605113925130"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524213608.png" alt="image-20210524213608760"></p>
<h3 id="Summary-of-CD"><a href="#Summary-of-CD" class="headerlink" title="Summary of CD"></a>Summary of CD</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524214916.png" alt="image-20210524214916555"></p>
<h3 id="Demo-in-the-class"><a href="#Demo-in-the-class" class="headerlink" title="Demo in the class"></a>Demo in the class</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525202814.png" alt="image-20210525202813881"></p>
<p>A: <mjx-container class="MathJax" jax="SVG"><svg style="vertical-align: -0.025ex" xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g data-mml-node="math"><g data-mml-node="mi"><path data-c="1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></g></g></g></svg></mjx-container> helix </p>
<p>B: y has more structure compared to x since absence of ATP, the kinase is a random coil and with the inclusion of ATP, protein turn to a secondary structure. </p>
<p>x: background high </p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525203347.PNG" alt="IMG_0420"></p>
<h2 id="Lecture-16-Modelling"><a href="#Lecture-16-Modelling" class="headerlink" title="Lecture 16 Modelling"></a>Lecture 16 Modelling</h2><h3 id="Homology-Modelling-of-Proteins"><a href="#Homology-Modelling-of-Proteins" class="headerlink" title="Homology Modelling of Proteins"></a>Homology Modelling of Proteins</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210528194258.png" alt="image-20210528194258663"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210528194450.png" alt="image-20210528194449958"></p>
<h2 id="Lecture-17-ITC"><a href="#Lecture-17-ITC" class="headerlink" title="Lecture 17 ITC"></a>Lecture 17 ITC</h2><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210530185132.png" alt="image-20210530185132466"></p>
<h3 id="Isothermal-Titration-Calorimetry"><a href="#Isothermal-Titration-Calorimetry" class="headerlink" title="Isothermal Titration Calorimetry"></a>Isothermal Titration Calorimetry</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210530190205.png" alt="image-20210530190205232"></p>
<h3 id="What-is-ITC"><a href="#What-is-ITC" class="headerlink" title="What is ITC"></a>What is ITC</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210530190231.png" alt="image-20210530190231383"></p>
<ul>
<li>ITC could quantify the number of protein (molarity and number)</li>
</ul>
<blockquote>
<p>Kb tell how tight the protein bind</p>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210530190253.png" alt="image-20210530190253415"></p>
<h3 id="ITC-“thermogram”-Thermodynamic-measurements"><a href="#ITC-“thermogram”-Thermodynamic-measurements" class="headerlink" title="ITC “thermogram”: Thermodynamic measurements"></a>ITC “thermogram”: Thermodynamic measurements</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210530190427.png" alt="image-20210530190427895"></p>
<h3 id="C-value"><a href="#C-value" class="headerlink" title="C-value"></a>C-value</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210530190650.png" alt="image-20210530190650312"></p>
<h3 id="Stoichiometry"><a href="#Stoichiometry" class="headerlink" title="Stoichiometry"></a>Stoichiometry</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210530191006.png" alt="image-20210530191005933"></p>
<h3 id="ITC-be-used-for"><a href="#ITC-be-used-for" class="headerlink" title="ITC be used for"></a>ITC be used for</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210530191404.png" alt="image-20210530191404064"></p>
<h3 id="ITC-compared-with-SPR"><a href="#ITC-compared-with-SPR" class="headerlink" title="ITC compared with SPR:"></a>ITC compared with SPR:</h3><ul>
<li>bind molecular, do not consider the steric hinderance (no such problem)</li>
<li>Direct information (1 to 1) while SPR is not direct</li>
<li>Can be reused; recover your protein in a cheaper way </li>
</ul>
<h3 id="Advantage-of-ITC"><a href="#Advantage-of-ITC" class="headerlink" title="Advantage of ITC"></a>Advantage of ITC</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210603145156.png" alt="image-20210603145155878"></p>
<h2 id="Tutorial-for-final-paper"><a href="#Tutorial-for-final-paper" class="headerlink" title="Tutorial for final paper"></a><code>Tutorial for final paper</code></h2><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210524221949.png" alt="image-20210524221949150"></p>
<p>What does <mjx-container class="MathJax" jax="SVG"><svg style="vertical-align: -0.025ex" xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g data-mml-node="math"><g data-mml-node="mi"><path data-c="1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></g></g></g></svg></mjx-container>-helix A shows a high protection at &lt;1 ms during refolding imply? </p>
<blockquote>
<p>That’s the part where molecular condense and collapse and shield from the water</p>
</blockquote>
<p>What does the low protection readout of <mjx-container class="MathJax" jax="SVG"><svg style="vertical-align: -0.439ex" xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g data-mml-node="math"><g data-mml-node="mi"><path data-c="1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></g></g></g></svg></mjx-container> strands-1,-2 and -3 &lt;1 ms imply ?</p>
<blockquote>
<p>They are completely exposed to the water.</p>
</blockquote>
<p>Sequence of the folding process？</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525083611.png" alt="image-20210525083610873"></p>
<p>A is the first one to collapse (core of the protein), highest protection level</p>
<p>Second molecule: 4</p>
<p>Third: B</p>
<p>Forth: D</p>
<p>Fifth: C </p>
<p>A-4-B-D-C-5-E -1-2-3</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525195556.png" alt="image-20210525195556857"></p>
<p>How do you use gel filtration to determine a protein is dimer or monomer? (<code>1:20</code>)</p>
<blockquote>
<p>SDS-PAGE denature the protein, the structure would be destroyed. If this protein is a dimer, it will divid into two proteins.</p>
<p>Assume we have a kinase of 20KD from SDS-PAGE (monomer). 40KD(dimer). </p>
<p>In this case we use a set of marker run at GF column. The blue line are the marker we would use. Add protein to the marker and run run the gel again. Marker would stick to the protein. If protein came up at the same location, it will be a monomer, if out in the 40, then will be the dimer </p>
<p>Note: SDS-PAGE are not used in this experiment</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210603155027.png" alt="image-20210603155027364"></p>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525201634.png" alt="image-20210525201634649"></p>
<blockquote>
<p>互联网上的答案：</p>
<p>If other techniques are available, I generally don’t recommend the use of cross-linkers  because the process is funadamentally stochastic. Unless you have a very specific cross-linker (for example, attaching to cysteines near the dimerization site) you will get a mixture of dimers and higher order oligomers. This results in false positives at high concentration and false negatives at lower concentrations.</p>
<p>Ion mobility mass spectrometry (IM-MS) requires the dimer complex to be stable in the gas phase. Although it is quick and does give you additional information beyond some of the simple techniques, it would not be my first choice just to see if it is a dimer or not.</p>
<p>The simple techniques are actually the best here. As pointed out by other posters, a native gel or size exclusion column will likely give you the correct answer. Keep in mind, though, that native gels are generally not run in strictly physiological conditions. The pH is adjusted away from neutral  to give the protein enough charge to migrate in the gel .Blue native gels may also mask charges important for complex formation. In contrast, size exclusion columns will give the correct answer in most situations.</p>
<p>For weaker complexes that may fall apart with separation based techniques, DLS ,diffusion NMR, or ITC may help and give you additional information as well (energetics with ITC and residue specific information if you have an assigned spectrum with NMR).</p>
<p>Edit: As Michal Jakob and Scott Kalbach suggest, analytical ultracentrifugation is one of the most accurate methods for measuring monomer/oligomer equilibrium. Unlike all the other techniques except cross-linking, it can also (with difficulty) be used for membrane proteins by adjusting the density of the solvent with D2O or sucrose.</p>
</blockquote>
<h3 id="Concepts-of-thermal-shift-CD"><a href="#Concepts-of-thermal-shift-CD" class="headerlink" title="Concepts of thermal shift + CD"></a>Concepts of thermal shift + CD</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525203347.PNG" alt="IMG_0420"></p>
<p>x has low structure because the background is high of fluorescence </p>
<p>Y has more structure 所以对应的是左图中的蓝线，是alpha helix</p>
<h3 id="concepts-of-SPR-kon-kof-kd"><a href="#concepts-of-SPR-kon-kof-kd" class="headerlink" title="concepts of SPR: kon; kof; kd"></a>concepts of SPR: kon; kof; kd</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210525205400.png" alt="image-20210525205400376"></p>
<p><code>sensorgram</code></p>
<p>The difference in the binding and the affinity between these two ligands. Kon, Koff, How to define Kd</p>
<p>左图1: slow binder with high affinity </p>
<p>右图：quick binder （通过陡然的上升）but with very lousy background, low affinity (通过右图的右半部分的悬崖式的断层)</p>
<p>kD depends on the calculation <mjx-container class="MathJax" jax="SVG"><svg style="vertical-align: -0.357ex" xmlns="http://www.w3.org/2000/svg" width="3.028ex" height="1.927ex" role="img" focusable="false" viewBox="0 -694 1338.2 851.8"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g data-mml-node="math"><g data-mml-node="msub"><g data-mml-node="mi"><path data-c="1D458" d="M121 647Q121 657 125 670T137 683Q138 683 209 688T282 694Q294 694 294 686Q294 679 244 477Q194 279 194 272Q213 282 223 291Q247 309 292 354T362 415Q402 442 438 442Q468 442 485 423T503 369Q503 344 496 327T477 302T456 291T438 288Q418 288 406 299T394 328Q394 353 410 369T442 390L458 393Q446 405 434 405H430Q398 402 367 380T294 316T228 255Q230 254 243 252T267 246T293 238T320 224T342 206T359 180T365 147Q365 130 360 106T354 66Q354 26 381 26Q429 26 459 145Q461 153 479 153H483Q499 153 499 144Q499 139 496 130Q455 -11 378 -11Q333 -11 305 15T277 90Q277 108 280 121T283 145Q283 167 269 183T234 206T200 217T182 220H180Q168 178 159 139T145 81T136 44T129 20T122 7T111 -2Q98 -11 83 -11Q66 -11 57 -1T48 16Q48 26 85 176T158 471L195 616Q196 629 188 632T149 637H144Q134 637 131 637T124 640T121 647Z"></path></g><g data-mml-node="TeXAtom" transform="translate(521, -150) scale(0.707)" data-mjx-texclass="ORD"><g data-mml-node="mi"><path data-c="1D45C" d="M201 -11Q126 -11 80 38T34 156Q34 221 64 279T146 380Q222 441 301 441Q333 441 341 440Q354 437 367 433T402 417T438 387T464 338T476 268Q476 161 390 75T201 -11ZM121 120Q121 70 147 48T206 26Q250 26 289 58T351 142Q360 163 374 216T388 308Q388 352 370 375Q346 405 306 405Q243 405 195 347Q158 303 140 230T121 120Z"></path></g><g data-mml-node="mi" transform="translate(485, 0)"><path data-c="1D45B" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path></g></g></g></g></g></svg></mjx-container></p>
<p><mjx-container class="MathJax" jax="SVG"><svg style="vertical-align: -1.033ex" xmlns="http://www.w3.org/2000/svg" width="9.336ex" height="3.384ex" role="img" focusable="false" viewBox="0 -1039.3 4126.3 1495.9"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g data-mml-node="math"><g data-mml-node="msub"><g data-mml-node="mi"><path data-c="1D458" d="M121 647Q121 657 125 670T137 683Q138 683 209 688T282 694Q294 694 294 686Q294 679 244 477Q194 279 194 272Q213 282 223 291Q247 309 292 354T362 415Q402 442 438 442Q468 442 485 423T503 369Q503 344 496 327T477 302T456 291T438 288Q418 288 406 299T394 328Q394 353 410 369T442 390L458 393Q446 405 434 405H430Q398 402 367 380T294 316T228 255Q230 254 243 252T267 246T293 238T320 224T342 206T359 180T365 147Q365 130 360 106T354 66Q354 26 381 26Q429 26 459 145Q461 153 479 153H483Q499 153 499 144Q499 139 496 130Q455 -11 378 -11Q333 -11 305 15T277 90Q277 108 280 121T283 145Q283 167 269 183T234 206T200 217T182 220H180Q168 178 159 139T145 81T136 44T129 20T122 7T111 -2Q98 -11 83 -11Q66 -11 57 -1T48 16Q48 26 85 176T158 471L195 616Q196 629 188 632T149 637H144Q134 637 131 637T124 640T121 647Z"></path></g><g data-mml-node="mi" transform="translate(521, -150) scale(0.707)"><path data-c="1D437" d="M287 628Q287 635 230 637Q207 637 200 638T193 647Q193 655 197 667T204 682Q206 683 403 683Q570 682 590 682T630 676Q702 659 752 597T803 431Q803 275 696 151T444 3L430 1L236 0H125H72Q48 0 41 2T33 11Q33 13 36 25Q40 41 44 43T67 46Q94 46 127 49Q141 52 146 61Q149 65 218 339T287 628ZM703 469Q703 507 692 537T666 584T629 613T590 629T555 636Q553 636 541 636T512 636T479 637H436Q392 637 386 627Q384 623 313 339T242 52Q242 48 253 48T330 47Q335 47 349 47T373 46Q499 46 581 128Q617 164 640 212T683 339T703 469Z"></path></g></g><g data-mml-node="mo" transform="translate(1434.3, 0)"><path data-c="3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path></g><g data-mml-node="mfrac" transform="translate(2490, 0)"><g data-mml-node="msub" transform="translate(220, 548.6) scale(0.707)"><g data-mml-node="mi"><path data-c="1D458" d="M121 647Q121 657 125 670T137 683Q138 683 209 688T282 694Q294 694 294 686Q294 679 244 477Q194 279 194 272Q213 282 223 291Q247 309 292 354T362 415Q402 442 438 442Q468 442 485 423T503 369Q503 344 496 327T477 302T456 291T438 288Q418 288 406 299T394 328Q394 353 410 369T442 390L458 393Q446 405 434 405H430Q398 402 367 380T294 316T228 255Q230 254 243 252T267 246T293 238T320 224T342 206T359 180T365 147Q365 130 360 106T354 66Q354 26 381 26Q429 26 459 145Q461 153 479 153H483Q499 153 499 144Q499 139 496 130Q455 -11 378 -11Q333 -11 305 15T277 90Q277 108 280 121T283 145Q283 167 269 183T234 206T200 217T182 220H180Q168 178 159 139T145 81T136 44T129 20T122 7T111 -2Q98 -11 83 -11Q66 -11 57 -1T48 16Q48 26 85 176T158 471L195 616Q196 629 188 632T149 637H144Q134 637 131 637T124 640T121 647Z"></path></g><g data-mml-node="TeXAtom" transform="translate(521, -150) scale(0.707)" data-mjx-texclass="ORD"><g data-mml-node="mi"><path data-c="1D45C" d="M201 -11Q126 -11 80 38T34 156Q34 221 64 279T146 380Q222 441 301 441Q333 441 341 440Q354 437 367 433T402 417T438 387T464 338T476 268Q476 161 390 75T201 -11ZM121 120Q121 70 147 48T206 26Q250 26 289 58T351 142Q360 163 374 216T388 308Q388 352 370 375Q346 405 306 405Q243 405 195 347Q158 303 140 230T121 120Z"></path></g><g data-mml-node="mi" transform="translate(485, 0)"><path data-c="1D453" d="M118 -162Q120 -162 124 -164T135 -167T147 -168Q160 -168 171 -155T187 -126Q197 -99 221 27T267 267T289 382V385H242Q195 385 192 387Q188 390 188 397L195 425Q197 430 203 430T250 431Q298 431 298 432Q298 434 307 482T319 540Q356 705 465 705Q502 703 526 683T550 630Q550 594 529 578T487 561Q443 561 443 603Q443 622 454 636T478 657L487 662Q471 668 457 668Q445 668 434 658T419 630Q412 601 403 552T387 469T380 433Q380 431 435 431Q480 431 487 430T498 424Q499 420 496 407T491 391Q489 386 482 386T428 385H372L349 263Q301 15 282 -47Q255 -132 212 -173Q175 -205 139 -205Q107 -205 81 -186T55 -132Q55 -95 76 -78T118 -61Q162 -61 162 -103Q162 -122 151 -136T127 -157L118 -162Z"></path></g><g data-mml-node="mi" transform="translate(1035, 0)"><path data-c="1D453" d="M118 -162Q120 -162 124 -164T135 -167T147 -168Q160 -168 171 -155T187 -126Q197 -99 221 27T267 267T289 382V385H242Q195 385 192 387Q188 390 188 397L195 425Q197 430 203 430T250 431Q298 431 298 432Q298 434 307 482T319 540Q356 705 465 705Q502 703 526 683T550 630Q550 594 529 578T487 561Q443 561 443 603Q443 622 454 636T478 657L487 662Q471 668 457 668Q445 668 434 658T419 630Q412 601 403 552T387 469T380 433Q380 431 435 431Q480 431 487 430T498 424Q499 420 496 407T491 391Q489 386 482 386T428 385H372L349 263Q301 15 282 -47Q255 -132 212 -173Q175 -205 139 -205Q107 -205 81 -186T55 -132Q55 -95 76 -78T118 -61Q162 -61 162 -103Q162 -122 151 -136T127 -157L118 -162Z"></path></g></g></g><g data-mml-node="msub" transform="translate(345, -345) scale(0.707)"><g data-mml-node="mi"><path data-c="1D458" d="M121 647Q121 657 125 670T137 683Q138 683 209 688T282 694Q294 694 294 686Q294 679 244 477Q194 279 194 272Q213 282 223 291Q247 309 292 354T362 415Q402 442 438 442Q468 442 485 423T503 369Q503 344 496 327T477 302T456 291T438 288Q418 288 406 299T394 328Q394 353 410 369T442 390L458 393Q446 405 434 405H430Q398 402 367 380T294 316T228 255Q230 254 243 252T267 246T293 238T320 224T342 206T359 180T365 147Q365 130 360 106T354 66Q354 26 381 26Q429 26 459 145Q461 153 479 153H483Q499 153 499 144Q499 139 496 130Q455 -11 378 -11Q333 -11 305 15T277 90Q277 108 280 121T283 145Q283 167 269 183T234 206T200 217T182 220H180Q168 178 159 139T145 81T136 44T129 20T122 7T111 -2Q98 -11 83 -11Q66 -11 57 -1T48 16Q48 26 85 176T158 471L195 616Q196 629 188 632T149 637H144Q134 637 131 637T124 640T121 647Z"></path></g><g data-mml-node="TeXAtom" transform="translate(521, -150) scale(0.707)" data-mjx-texclass="ORD"><g data-mml-node="mi"><path data-c="1D45C" d="M201 -11Q126 -11 80 38T34 156Q34 221 64 279T146 380Q222 441 301 441Q333 441 341 440Q354 437 367 433T402 417T438 387T464 338T476 268Q476 161 390 75T201 -11ZM121 120Q121 70 147 48T206 26Q250 26 289 58T351 142Q360 163 374 216T388 308Q388 352 370 375Q346 405 306 405Q243 405 195 347Q158 303 140 230T121 120Z"></path></g><g data-mml-node="mi" transform="translate(485, 0)"><path data-c="1D45B" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path></g></g></g><rect width="1396.3" height="60" x="120" y="220"></rect></g></g></g></svg></mjx-container></p>
<p><code>IC50、EC50、Ki、Kd、Ka、Km、Kon、Koff傻傻分不清？可能是目前为止最详尽讨论文章</code></p>
<p><a class="link" target="_blank" rel="noopener" href="https://zhuanlan.zhihu.com/p/29900912">https://zhuanlan.zhihu.com/p/29900912<i class="fas fa-external-link-alt"></i></a></p>
<h3 id="Relate-SPR-to-ITC"><a href="#Relate-SPR-to-ITC" class="headerlink" title="Relate SPR to ITC"></a>Relate SPR to ITC</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210531083723.png" alt="image-20210531083723550"></p>
<blockquote>
<p>ANSWER:</p>
<p><mjx-container class="MathJax" jax="SVG"><svg style="vertical-align: -0.339ex" xmlns="http://www.w3.org/2000/svg" width="3.359ex" height="1.885ex" role="img" focusable="false" viewBox="0 -683 1484.5 833"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g data-mml-node="math"><g data-mml-node="msub"><g data-mml-node="mi"><path data-c="1D43E" d="M285 628Q285 635 228 637Q205 637 198 638T191 647Q191 649 193 661Q199 681 203 682Q205 683 214 683H219Q260 681 355 681Q389 681 418 681T463 682T483 682Q500 682 500 674Q500 669 497 660Q496 658 496 654T495 648T493 644T490 641T486 639T479 638T470 637T456 637Q416 636 405 634T387 623L306 305Q307 305 490 449T678 597Q692 611 692 620Q692 635 667 637Q651 637 651 648Q651 650 654 662T659 677Q662 682 676 682Q680 682 711 681T791 680Q814 680 839 681T869 682Q889 682 889 672Q889 650 881 642Q878 637 862 637Q787 632 726 586Q710 576 656 534T556 455L509 418L518 396Q527 374 546 329T581 244Q656 67 661 61Q663 59 666 57Q680 47 717 46H738Q744 38 744 37T741 19Q737 6 731 0H720Q680 3 625 3Q503 3 488 0H478Q472 6 472 9T474 27Q478 40 480 43T491 46H494Q544 46 544 71Q544 75 517 141T485 216L427 354L359 301L291 248L268 155Q245 63 245 58Q245 51 253 49T303 46H334Q340 37 340 35Q340 19 333 5Q328 0 317 0Q314 0 280 1T180 2Q118 2 85 2T49 1Q31 1 31 11Q31 13 34 25Q38 41 42 43T65 46Q92 46 125 49Q139 52 144 61Q147 65 216 339T285 628Z"></path></g><g data-mml-node="mi" transform="translate(849, -150) scale(0.707)"><path data-c="1D437" d="M287 628Q287 635 230 637Q207 637 200 638T193 647Q193 655 197 667T204 682Q206 683 403 683Q570 682 590 682T630 676Q702 659 752 597T803 431Q803 275 696 151T444 3L430 1L236 0H125H72Q48 0 41 2T33 11Q33 13 36 25Q40 41 44 43T67 46Q94 46 127 49Q141 52 146 61Q149 65 218 339T287 628ZM703 469Q703 507 692 537T666 584T629 613T590 629T555 636Q553 636 541 636T512 636T479 637H436Q392 637 386 627Q384 623 313 339T242 52Q242 48 253 48T330 47Q335 47 349 47T373 46Q499 46 581 128Q617 164 640 212T683 339T703 469Z"></path></g></g></g></g></svg></mjx-container> The lower the value, the higher the affinity; so the left figure could be A</p>
<p>The right figure which has low affinity binder could be figure B or C</p>
</blockquote>
<blockquote>
<p>Comparison between ITC and SPR</p>
<p>Biophysical techniques such as Isothermal Calorimetry (ITC), and Surface Plasmon Resonance are used to thermodynamically and kinetically characterize the binding mechanism of the protein-ligand or protein-protein interactions, </p>
<p><strong>ITC</strong> measures the heat released or absorbed during the protein-ligand interactions, whereas, <strong>SPR</strong> measures the change in reflective angle of the incident light caused by surface waves called ‘Plasmon Polaritons. Since plasmon polaritons, are sensitive to binding events occurring on a surface, it proportionally affects the angle of the reflected light </p>
</blockquote>
<h3 id="Concepts-of-thermal-shift-vs-binding-affinities"><a href="#Concepts-of-thermal-shift-vs-binding-affinities" class="headerlink" title="Concepts of thermal shift vs binding affinities"></a>Concepts of thermal shift vs binding affinities</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210531085158.png" alt="image-20210531085158750"></p>
<p>有三种选择ki (a)1nm  (b)100nm (c) 1pm,找出哪个curve和(c)对应</p>
<blockquote>
<p><strong>Answer</strong></p>
<ol>
<li>1nm =1000pm</li>
<li>1pm is the lowest value of ki, which means it has the highest affinity, suggesting the structure is stable </li>
<li>Require more energy to conduct the thermal shift</li>
</ol>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210531173403.png" alt="image-20210531173403654"></p>
<blockquote>
<p>上图是对应三种结构，记得那张在CD中的图</p>
<p>Alpha-helix : 对应的是peptideII</p>
<p>Beta-sheet:对应的是peptide I</p>
<p>Coil 对应的是peptides III</p>
<p>有一张图可以对应其关系</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210531174146.png" alt="Illustrated Glossary of Organic Chemistry - Coil; random coil"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210531173704.png" alt="image-20210531173558476"></p>
</blockquote>
<hr>
<h2 id="CW1"><a href="#CW1" class="headerlink" title="CW1"></a>CW1</h2><h3 id="Requirements"><a href="#Requirements" class="headerlink" title="Requirements"></a>Requirements</h3><p>Courseworks requirements</p>
<p>Protocol</p>
<p>columns</p>
<p>Expected output</p>
<p>Which columns</p>
<p>Justification for choosing that column </p>
<p>Expected output? Diagrams</p>
<hr>
<p>How would you design a purification regime, step-by-step, to isolate the mutant enzyme X from the liver of mouse “Y”?  Explain the rationale of your purification scheme. [<strong>60 marks</strong>] </p>
<p><strong>Answer</strong>:</p>
<p><code>step1:</code>  Since the provided raw sample is from the liver of mouse “Y”, which means the liver need to be smashed into the liquid format</p>
<p><code>step2:</code> The smashed liver liquid are then processed in a high speed blender, breaking out the cell membranes and releasing the inside cell components.</p>
<p><code>step3:</code>Owing to the fact that our target is to isolate the mutant enzyme, we need to utilize some filter papers to roughly filter out those target contents. </p>
<p><code>step4: </code>Ammonium Sulfate precipitation method can be used to partially purify the mixture. According to the protocols, we need to slowly add saturated ammonium sulfate to bring the final concentration to 50% saturation, which means the total amount of ammonium sulfate should equal with the sample amount. Therefore, the ammonium sulfate should add into the samples gradually, in this case, the ideal percentage should be 10%, 20%…50%. But owing to the different condition, some bias may exist, therefore, 5 columens should be prepared to hold the precipitates under different ammonium sulfate percentage, each columns should be clearly labeled for convenience. Adding the ammonium sulfate very slowly ensures that local concentration around the site of addition does not exceed the desired salt concentration. The insoluble aggregates would be our products. </p>
<p><code>Rational of step4</code>: The whole mechanisms behind this approach is the electrolyte-nonelectrolyte interaction.  In solution, proteins form hydrogen bonds with water molecules through their exposed polar and ionic groups. When high concentrations of small, highly charged ions such as ammonium sulfate are added, these groups compete with the proteins to bind to the water molecules. This removes the water molecules from the protein and decreases its solubility, resulting in precipitation. Critical factors that affect the concentration at which a particular protein will precipitate include: the number and position of polar groups, molecular weight of the protein, pH of the solution, and temperature at which the precipitation is performed.</p>
<p><code>step5:</code> the isolated 5 samples are resuspended in the buffer to be active and soluble again. As suggested in the problems that the presence of enzyme X can be detected with radioassay by the capture of the released 14CO2. In this case, five samples are feed into substrates respectively and the sample whose generated CO2 can be detected by radioassay could be considered as the sample enzyme x. We denoted this sample as sample A. </p>
<p><code>step6:</code>SDS-PAGE should be incorporated to test the sample A’s purity</p>
<p><code>Rational of step6:</code>SDS-PAGE is an analytical technique to separate proteins based on their molecular weight. In SDS-PAGE, the use of sodium dodecyl sulfate (SDS, also known as sodium lauryl sulfate) and polyacrylamide gel largely eliminates the influence of the structure and charge, and proteins are separated solely based on polypeptide chain length. Therefore, it could be used to figure out the samples’ purity.</p>
<p><code>step7</code>:  Dialysis technique should be included to perform purification  for samples containing macromolecules, in this case, enzymes. Some unnecessary salts in the previous steps could be removed.</p>
<p><code>Rational of step7     </code>Dialysis is a separation technique that facilitates the removal of small, unwanted compounds from macromolecules in solution by selective and passive diffusion through a semi-permeable membrane. </p>
<p><code>step8</code>: ion exchange chromatography should be utilized on sample A in the following step. Owing to the fact we have no idea about the pI (isoelectric point) of the protein, we need to include both anion and cation exchanger. In other words, the samples need to conduct the experiments twice.</p>
<p>We should note that the tightly bound ones would be washed away by the high concentrated sodium chloride, the eluted samples should be collected by different tubes</p>
<p>SDS-PAGE should be conducted to figure out the inside components.</p>
<p><code>Rational of step8:</code></p>
<p>The net surface charge of proteins varies according to the surrounding pH. The pH at which a protein has no net charge is called isoelectric point (pI). Above its isoelectric point (pI), a protein will bind to a positively charged anion exchanger. Below its pI, a protein will bind to a negatively charged cation exchanger.</p>
<p>Proteins bind as they are loaded onto a column at low ionic strength. The conditions are then altered so that bound substances are desorbed differentially. Elution is usually performed by increasing salt concentration or changing pH in a gradient. Sodium chloride is usually considered as a suitable candidate to elute the bond ones gradually. By adjusting different concentration sodium chloride solution, we could obtain different protein samples. </p>
<p><code>Step9</code> Repeat the step in step5 and denoted the target sample as sample B.</p>
<p><code>step10:</code> Hydrophobic Interaction Chromatography (HIC) would be utilized on sample B to purify the enzyme X.</p>
<p><code>Rational of step10</code>: it could separate the protein based on the hydrophobicity. In this type of chromatography, hydrophobic groups such as phenyl, octyl, or butyl, are attached to the stationary column. Proteins that pass through the column that have hydrophobic amino acid side chains on their surfaces are able to interact with and bind to the hydrophobic groups on the column. Finally, buffer need to be used to elute the rest proteins.</p>
<p><code>step11</code>: Similarly,  repeat the step in step5 &amp; step6, and denoted the target sample as sample C </p>
<p><code>step12</code>: Gel filtration chromatography should also be included on sample C to filter out the proteins based on their molecules weight. The technique is ideal for the final polishing steps in a purification when sample volumes have been reduced.</p>
<p><code>Rational of step12</code> : The gel filtration chromatography is based on the molecular size and the hydrodynamic volume of the components. The molecules are separated by the differential exclusion or inclusion of solutes as they pass through the stationary phase containing heteroporous cross-linked polymeric gel or beads. Different permeation rates of the solute molecules cause them to sift in the interior of the gel particles. A column of the porous matrix is in equilibrium with the mobile phase for separation of the molecules. Large molecules are entirely excluded from the pores and come first in the effluent. Smaller molecules get distributed between the mobile phase and the outside of the sieve. Then, they pass through the column at a slower rate and appear later in the effluent.</p>
<p><code>step13 </code>: Similarly,  repeat the step in step5 &amp; step6, and denoted the target sample as sample D</p>
<p>Based on the previous 13 steps, we could successfully isolate the mutant enzyme X from the liver of mouse “Y”. For the protein identification, you could turn to 2D gel electrophoresis and mass spectrometry for more details. </p>
<p><code>step14:</code>2D Gel Electrophoresis can be used to characterize the proteins in the remaining samples. </p>
<p><code>Rational behind step14:</code> two dimensional gel electrophoresis could separate proteins based on pI in 1st dimension and operate proteins based on molecular weight in 2nd dimensions</p>
<p><code>step15</code>: Search the online databases, hoping to find the similar results.</p>
<hr>
<p>Your partially purified sample exhibits very strong enzyme X activity in 14C assay but your SDS-PAGE looks like the example shown in the figure below: </p>
<p>What strategies would you use to identify enzyme X from the mixture? Explain how you would carry out the experiment and the reasons of your strategies. [<strong>40 marks</strong>] </p>
<p><strong>Answer:</strong></p>
<p>As learnt from the lecture, there are two widely used methods in protein identification, Edman Degradation and Mass spectrometry.</p>
<p>Edman degradation is the process of purifying protein by sequentially removing one residue at a time from the amino end of a peptide. Phenyl isothiocyanate is added to create a phenylthiocarbamoyl derivative with the N-terminal to remove only one residue at a time. The N-terminal is then cleaved under less harsh acidic conditions, creating a cyclic compound of phenylthiohydantoin PTH-amino acid. This does not damage the protein and leaves two constituents of the peptide.</p>
<p>At the first step, the proteins need to be broken down to peptide fragments by the strong acid which could ensure their terminal would be blocked and cannot be sequenced. High pressure liquid chromatography (HPLC) would be employed to separate the peptides. The amino acid sequencing are then followed to conduct sequence alignment to match the known sequences in the database to identify the protein.</p>
<p>The second choice would be the mass spectrometry. Mass spectrometry measures the mass-to-charge ratio of charged particles, mass spectrometry is used for determining masses of particles and the elemental composition of a sample of molecule as well as for elucidating the chemical structure of molecules such as peptides. </p>
<p>As can be seen from the figure that, four bands, suggesting four proteins, have been got. With either Edman degradation or mass spectrometry, we could utilize the approaches like BLAST to align the sequence stored in database. Since enzyme X in this case still could not be determined, we could use the “process of elimination” to identify the other three proteins.</p>

        </div>

        
            <div class="post-copyright-info">
                <div class="article-copyright-info-container">
    <ul>
        <li>Post title：BIO312 Lee&#39;s part</li>
        <li>Post author：Yuxuan Wu</li>
        <li>Create time：2021-03-02 06:44:01</li>
        <li>
            Post link：yuxuanwu17.github.io2021/03/02/2021-03-02-BIO312-lees-part/
        </li>
        <li>
            Copyright Notice：All articles in this blog are licensed under <a class="license" target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by-nc-sa/4.0/deed.zh">BY-NC-SA</a> unless stating additionally.
        </li>
    </ul>
</div>

            </div>
        

        
            <div class="article-nav">
                
                    <div class="article-prev">
                        <a class="prev"
                           rel="prev"
                           href="/2021/03/02/2021-03-02-BIO312-Wangs-part/"
                        >
                            <span class="left arrow-icon flex-center">
                              <i class="fas fa-chevron-left"></i>
                            </span>
                            <span class="title flex-center">
                                <span class="post-nav-title-item">BIO312 Wang&#39;s part</span>
                                <span class="post-nav-item">Prev posts</span>
                            </span>
                        </a>
                    </div>
                
                
                    <div class="article-next">
                        <a class="next"
                           rel="next"
                           href="/2021/02/19/2021-02-19-(CSE201)-Advanced-Databases/"
                        >
                            <span class="title flex-center">
                                <span class="post-nav-title-item">(CSE201) Advanced Databases</span>
                                <span class="post-nav-item">Next posts</span>
                            </span>
                            <span class="right arrow-icon flex-center">
                              <i class="fas fa-chevron-right"></i>
                            </span>
                        </a>
                    </div>
                
            </div>
        

        
    </div>
</div>


                
            </div>

        </div>

        <div class="page-main-content-bottom">
            <footer class="footer">
    <div class="info-container">
        <div class="copyright-info info-item">
            &copy;
            
              <span>2020</span> -
            
            2021 <i class="fas fa-heart icon-animate"></i> <a href="/">Yuxuan Wu</a>
        </div>
        
            <script async  src="//busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
            <div class="website-count info-item">
                
                    <span id="busuanzi_container_site_uv">
                        Visitor Count <span id="busuanzi_value_site_uv"></span>&ensp;
                    </span>
                
                
                    <span id="busuanzi_container_site_pv">
                        Totalview <span id="busuanzi_value_site_pv"></span>
                    </span>
                
            </div>
        
        <div class="theme-info info-item">
            Powered by <a target="_blank" href="https://hexo.io">Hexo</a> | Theme <a class="theme-version" target="_blank" href="https://github.com/XPoet/hexo-theme-keep">Keep v3.3.1</a>
        </div>
    </div>
</footer>

        </div>
    </div>

    
        <div class="post-tools">
            <div class="post-tools-container">
    <ul class="tools-list">
        <!-- TOC aside toggle -->
        
            <li class="tools-item page-aside-toggle">
                <i class="fas fa-outdent"></i>
            </li>
        

        <!-- go comment -->
        
    </ul>
</div>

        </div>
    

    <div class="right-bottom-side-tools">
        <div class="side-tools-container">
    <ul class="side-tools-list">
        <li class="tools-item tool-font-adjust-plus flex-center">
            <i class="fas fa-search-plus"></i>
        </li>

        <li class="tools-item tool-font-adjust-minus flex-center">
            <i class="fas fa-search-minus"></i>
        </li>

        <li class="tools-item tool-expand-width flex-center">
            <i class="fas fa-arrows-alt-h"></i>
        </li>

        <li class="tools-item tool-dark-light-toggle flex-center">
            <i class="fas fa-moon"></i>
        </li>

        <!-- rss -->
        

        

        <li class="tools-item tool-scroll-to-bottom flex-center">
            <i class="fas fa-arrow-down"></i>
        </li>
    </ul>

    <ul class="exposed-tools-list">
        <li class="tools-item tool-toggle-show flex-center">
            <i class="fas fa-cog fa-spin"></i>
        </li>
        
            <li class="tools-item tool-scroll-to-top flex-center">
                <i class="arrow-up fas fa-arrow-up"></i>
                <span class="percent"></span>
            </li>
        
    </ul>
</div>

    </div>

    
        <aside class="page-aside">
            <div class="post-toc-wrap">
    <div class="post-toc">
        <ol class="nav"><li class="nav-item nav-level-2"><a class="nav-link" href="#Amino-acids-structure"><span class="nav-number">1.</span> <span class="nav-text">Amino acids structure</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-09-Folding-in-vivo"><span class="nav-number">2.</span> <span class="nav-text">Lecture 09 Folding in vivo</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#protein-folding-in-the-cell"><span class="nav-number">2.1.</span> <span class="nav-text">protein folding in the cell</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Folding-in-vitro-and-in-vivo"><span class="nav-number">2.2.</span> <span class="nav-text">Folding in vitro and in vivo</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Macromolecular-crowding"><span class="nav-number">2.3.</span> <span class="nav-text">Macromolecular crowding</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Effects-of-crowding"><span class="nav-number">2.4.</span> <span class="nav-text">Effects of crowding</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Molecular-chaperones"><span class="nav-number">2.5.</span> <span class="nav-text">Molecular chaperones</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-10-Folding-in-vitro"><span class="nav-number">3.</span> <span class="nav-text">Lecture 10 Folding in vitro</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#The-Levinthal-Paradox"><span class="nav-number">3.1.</span> <span class="nav-text">The Levinthal Paradox</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Equilibrium-unfolding"><span class="nav-number">3.2.</span> <span class="nav-text">Equilibrium unfolding</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Anfinsen-Experiment"><span class="nav-number">3.3.</span> <span class="nav-text">Anfinsen Experiment</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Information-online"><span class="nav-number">3.4.</span> <span class="nav-text">Information online</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#quiz"><span class="nav-number">3.5.</span> <span class="nav-text">quiz</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Molten-Globule-State-MG"><span class="nav-number">3.6.</span> <span class="nav-text">Molten Globule State (MG)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#ANS-as-a-hydrophobicity-measuring"><span class="nav-number">3.7.</span> <span class="nav-text">ANS as a hydrophobicity measuring</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-melting-thermal-shift"><span class="nav-number">3.8.</span> <span class="nav-text">Protein melting (thermal shift)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Unfolded-state"><span class="nav-number">3.9.</span> <span class="nav-text">Unfolded state</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Three-classic-models-of-protein-folding"><span class="nav-number">3.10.</span> <span class="nav-text">Three classic models of protein folding</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#The-folding-funnel"><span class="nav-number">3.11.</span> <span class="nav-text">The folding funnel</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-11-SPR"><span class="nav-number">4.</span> <span class="nav-text">Lecture 11 SPR</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Surface-Plasmon-Resonance"><span class="nav-number">4.1.</span> <span class="nav-text">Surface Plasmon Resonance</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#pros-amp-cons"><span class="nav-number">4.2.</span> <span class="nav-text">pros &amp; cons</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Binding-affinity"><span class="nav-number">4.3.</span> <span class="nav-text">Binding affinity</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Tutorial"><span class="nav-number">4.4.</span> <span class="nav-text">Tutorial</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-12-Crystallography"><span class="nav-number">5.</span> <span class="nav-text">Lecture 12 Crystallography</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Whole-process"><span class="nav-number">5.1.</span> <span class="nav-text">Whole process</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#X-ray-diffraction-of-a-crystal"><span class="nav-number">5.2.</span> <span class="nav-text">X-ray diffraction of a crystal</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Raw-data"><span class="nav-number">5.3.</span> <span class="nav-text">Raw data</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#From-diffraction-to-electron-density-map"><span class="nav-number">5.4.</span> <span class="nav-text">From diffraction to electron density map</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Solving-the-phase-problem"><span class="nav-number">5.5.</span> <span class="nav-text">Solving the phase problem</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Phase-problem"><span class="nav-number">5.6.</span> <span class="nav-text">Phase problem</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Resolution"><span class="nav-number">5.7.</span> <span class="nav-text">Resolution</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Limitations-of-the-phase-problem"><span class="nav-number">5.8.</span> <span class="nav-text">Limitations of the phase problem</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Steps-of-protein-X-ray-crystallography"><span class="nav-number">5.9.</span> <span class="nav-text">Steps of protein X-ray crystallography</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#R-factor-measuring-convergence"><span class="nav-number">5.10.</span> <span class="nav-text">R-factor: measuring convergence</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Occupancy"><span class="nav-number">5.11.</span> <span class="nav-text">Occupancy</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Redundancy"><span class="nav-number">5.12.</span> <span class="nav-text">Redundancy</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Resolution-1"><span class="nav-number">5.13.</span> <span class="nav-text">Resolution</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#%E2%80%9CB-Factor-Temperature-factor-%E2%80%9D"><span class="nav-number">5.14.</span> <span class="nav-text">“B-Factor (Temperature factor)”</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Two-types-of-solvent-ordered-and-disordered"><span class="nav-number">5.15.</span> <span class="nav-text">Two types of solvent: ordered and disordered</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#crystals-and-crystal-in-crystallographic-data"><span class="nav-number">5.16.</span> <span class="nav-text">crystals and crystal in crystallographic data</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Quiz"><span class="nav-number">5.17.</span> <span class="nav-text">Quiz</span></a></li><li class="nav-item nav-level-3"><a class="nav-link"><span class="nav-number">5.18.</span> <span class="nav-text"></span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-13-14-NMR"><span class="nav-number">6.</span> <span class="nav-text">Lecture 13  14 NMR</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#What-is-NRM"><span class="nav-number">6.1.</span> <span class="nav-text">What is NRM</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#used-to-identity-peptide-amide-chain"><span class="nav-number">6.2.</span> <span class="nav-text">used to identity peptide amide chain</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#demo-in-class"><span class="nav-number">6.3.</span> <span class="nav-text">demo in class</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Types-of-triple-resonance"><span class="nav-number">6.4.</span> <span class="nav-text">Types of triple resonance</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Protein-dynamics"><span class="nav-number">6.5.</span> <span class="nav-text">Protein dynamics</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#NMR-shows-change-in-configuration"><span class="nav-number">6.6.</span> <span class="nav-text">NMR: shows change in configuration</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Binding-constants"><span class="nav-number">6.7.</span> <span class="nav-text">Binding constants</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Pros-and-cons-of-NRM-in-structural-biology"><span class="nav-number">6.8.</span> <span class="nav-text">Pros and cons of NRM in structural biology</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Tutorial-on-NMR"><span class="nav-number">7.</span> <span class="nav-text">Tutorial on NMR</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Question-1"><span class="nav-number">7.1.</span> <span class="nav-text">Question 1</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Question-2"><span class="nav-number">7.2.</span> <span class="nav-text">Question 2</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Question-3"><span class="nav-number">7.3.</span> <span class="nav-text">Question 3</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-15-CD"><span class="nav-number">8.</span> <span class="nav-text">Lecture 15 CD</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Circular-Polarized-Light"><span class="nav-number">8.1.</span> <span class="nav-text">Circular Polarized Light</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Circular-Polarized-Light-1"><span class="nav-number">8.2.</span> <span class="nav-text">Circular Polarized Light</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#CD-signal-of-proteins"><span class="nav-number">8.3.</span> <span class="nav-text">CD signal of proteins</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Standard-curves"><span class="nav-number">8.4.</span> <span class="nav-text">Standard curves</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Using-CD-to-Monitor-tertiary-structure"><span class="nav-number">8.5.</span> <span class="nav-text">Using CD to Monitor tertiary structure</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Summary-of-CD"><span class="nav-number">8.6.</span> <span class="nav-text">Summary of CD</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Demo-in-the-class"><span class="nav-number">8.7.</span> <span class="nav-text">Demo in the class</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-16-Modelling"><span class="nav-number">9.</span> <span class="nav-text">Lecture 16 Modelling</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Homology-Modelling-of-Proteins"><span class="nav-number">9.1.</span> <span class="nav-text">Homology Modelling of Proteins</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Lecture-17-ITC"><span class="nav-number">10.</span> <span class="nav-text">Lecture 17 ITC</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Isothermal-Titration-Calorimetry"><span class="nav-number">10.1.</span> <span class="nav-text">Isothermal Titration Calorimetry</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#What-is-ITC"><span class="nav-number">10.2.</span> <span class="nav-text">What is ITC</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#ITC-%E2%80%9Cthermogram%E2%80%9D-Thermodynamic-measurements"><span class="nav-number">10.3.</span> <span class="nav-text">ITC “thermogram”: Thermodynamic measurements</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#C-value"><span class="nav-number">10.4.</span> <span class="nav-text">C-value</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Stoichiometry"><span class="nav-number">10.5.</span> <span class="nav-text">Stoichiometry</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#ITC-be-used-for"><span class="nav-number">10.6.</span> <span class="nav-text">ITC be used for</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#ITC-compared-with-SPR"><span class="nav-number">10.7.</span> <span class="nav-text">ITC compared with SPR:</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Advantage-of-ITC"><span class="nav-number">10.8.</span> <span class="nav-text">Advantage of ITC</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Tutorial-for-final-paper"><span class="nav-number">11.</span> <span class="nav-text">Tutorial for final paper</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Concepts-of-thermal-shift-CD"><span class="nav-number">11.1.</span> <span class="nav-text">Concepts of thermal shift + CD</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#concepts-of-SPR-kon-kof-kd"><span class="nav-number">11.2.</span> <span class="nav-text">concepts of SPR: kon; kof; kd</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Relate-SPR-to-ITC"><span class="nav-number">11.3.</span> <span class="nav-text">Relate SPR to ITC</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Concepts-of-thermal-shift-vs-binding-affinities"><span class="nav-number">11.4.</span> <span class="nav-text">Concepts of thermal shift vs binding affinities</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#CW1"><span class="nav-number">12.</span> <span class="nav-text">CW1</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Requirements"><span class="nav-number">12.1.</span> <span class="nav-text">Requirements</span></a></li></ol></li></ol>
    </div>
</div>
        </aside>
    

    <div class="image-viewer-container">
    <img src="">
</div>


    
        <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
          <span class="search-input-field-pre">
            <i class="fas fa-keyboard"></i>
          </span>
            <div class="search-input-container">
                <input autocomplete="off"
                       autocorrect="off"
                       autocapitalize="off"
                       placeholder="Search..."
                       spellcheck="false"
                       type="search"
                       class="search-input"
                >
            </div>
            <span class="popup-btn-close">
                <i class="fas fa-times"></i>
            </span>
        </div>
        <div id="search-result">
            <div id="no-result">
                <i class="fas fa-spinner fa-pulse fa-5x fa-fw"></i>
            </div>
        </div>
    </div>
</div>

    

</main>




<script src="/js/utils.js"></script>

<script src="/js/main.js"></script>

<script src="/js/header-shrink.js"></script>

<script src="/js/back2top.js"></script>

<script src="/js/dark-light-toggle.js"></script>



    
<script src="/js/local-search.js"></script>




    
<script src="/js/code-copy.js"></script>





<div class="post-scripts">
    
        
<script src="/js/left-side-toggle.js"></script>

<script src="/js/libs/anime.min.js"></script>

<script src="/js/toc.js"></script>

    
</div>



</body>
</html>
