<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="keywords" content="Hexo Theme Keep">
    <meta name="description" content="Welcome to my personal blogs">
    <meta name="author" content="Yuxuan Wu">
    
    <title>
        
            PROTEOMICS DATA ANALYSIS |
        
        Yuxuan Wu
    </title>
    <link rel="shortcut icon" href="/images/logo.svg">
    
<link rel="stylesheet" href="/css/style.css">

    
<link rel="stylesheet" href="/css/font-awesome.min.css">

    <script id="hexo-configurations">
    let KEEP = window.KEEP || {};
    KEEP.hexo_config = {"hostname":"example.com","root":"/","language":"en","path":"search.xml"};
    KEEP.theme_config = {"toc":{"enable":true,"number":true,"expand_all":false},"style":{"primary_color":"#0066CC","avatar":"/images/avatar.svg","favicon":"/images/logo.svg","article_img_align":"left","left_side_width":"260px","content_max_width":"920px","hover":{"shadow":true,"scale":true},"first_screen":{"enable":true,"background_img":"/images/bg.svg","description":"Keep coding, Keep hungry."},"scroll":{"progress_bar":{"enable":true},"percent":{"enable":true}}},"local_search":{"enable":true,"preload":true},"code_copy":{"enable":true,"style":"default"},"pjax":{"enable":false},"lazyload":{"enable":false},"version":"3.3.1"};
    KEEP.language_ago = {"second":"%s seconds ago","minute":"%s minutes ago","hour":"%s hours ago","day":"%s days age","week":"%s weeks age","month":"%s months age","year":"%s years age"};
  </script>
<!-- hexo injector head_end start -->
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css">

<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/hexo-math@4.0.0/dist/style.css">
<!-- hexo injector head_end end --><meta name="generator" content="Hexo 5.3.0"></head>


<body>
<div class="progress-bar-container">
    
        <span class="scroll-progress-bar"></span>
    

    
</div>


<main class="page-container">

    

    <div class="page-main-content">

        <div class="page-main-content-top">
            <header class="header-wrapper">

    <div class="header-content">
        <div class="left">
            <a class="logo-title" href="/">
                Yuxuan Wu
            </a>
        </div>

        <div class="right">
            <div class="pc">
                <ul class="menu-list">
                    
                        <li class="menu-item">
                            <a class=""
                               href="/"
                            >
                                HOME
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/archives"
                            >
                                ARCHIVES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/categories"
                            >
                                CATEGORIES
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/tags"
                            >
                                TAGS
                            </a>
                        </li>
                    
                        <li class="menu-item">
                            <a class=""
                               href="/about"
                            >
                                ABOUT
                            </a>
                        </li>
                    
                    
                        <li class="menu-item search search-popup-trigger">
                            <i class="fas fa-search"></i>
                        </li>
                    
                </ul>
            </div>
            <div class="mobile">
                
                    <div class="icon-item search search-popup-trigger"><i class="fas fa-search"></i></div>
                
                <div class="icon-item menu-bar">
                    <div class="menu-bar-middle"></div>
                </div>
            </div>
        </div>
    </div>

    <div class="header-drawer">
        <ul class="drawer-menu-list">
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/">HOME</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/archives">ARCHIVES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/categories">CATEGORIES</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/tags">TAGS</a>
                </li>
            
                <li class="drawer-menu-item flex-center">
                    <a class=""
                       href="/about">ABOUT</a>
                </li>
            
        </ul>
    </div>

    <div class="window-mask"></div>

</header>


        </div>

        <div class="page-main-content-middle">

            <div class="main-content normal-code-theme">

                
                    <div class="fade-in-down-animation">
    <div class="article-content-container">

        <div class="article-title">
            <span class="title-hover-animation">PROTEOMICS DATA ANALYSIS</span>
        </div>

        
            <div class="article-header">
                <div class="avatar">
                    <img src="/images/avatar.svg">
                </div>
                <div class="info">
                    <div class="author">
                        <span class="name">Yuxuan Wu</span>
                        <span class="level">Lv13</span>
                    </div>
                    <div class="meta-info">
                        <div class="article-meta-info">
    <span class="article-date article-meta-item">
        <i class="fas fa-edit"></i> 2021-05-01 01:02:21
    </span>
    
        <span class="article-categories article-meta-item">
            <i class="fas fa-folder"></i>
            <ul>
                
                    <li>
                        <a href="/categories/Bioinformatics/">Bioinformatics</a>
                    </li>
                
            </ul>
        </span>
    
    
        <span class="article-tags article-meta-item">
            <i class="fas fa-tags"></i>
            <ul>
                
                    <li>
                        <a href="/tags/R/">R</a>
                    </li>
                
                    <li>
                        | <a href="/tags/RNA-modification/">RNA modification</a>
                    </li>
                
                    <li>
                        | <a href="/tags/notes/">notes</a>
                    </li>
                
            </ul>
        </span>
    

    
    
        <span class="article-wordcount article-meta-item">
            <i class="fas fa-file-word"></i> <span>2.5k Words</span>
        </span>
    
    
        <span class="article-min2read article-meta-item">
            <i class="fas fa-clock"></i> <span>15 Mins</span>
        </span>
    
    
        <span class="article-pv article-meta-item">
            <i class="fas fa-eye"></i> <span id="busuanzi_value_page_pv"></span>
        </span>
    
</div>

                    </div>
                </div>
            </div>
        

        <div class="article-content markdown-body">
            <p>[toc]</p>
<h1 id="Workflow"><a href="#Workflow" class="headerlink" title="Workflow"></a>Workflow</h1><p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210508102044.png" alt="image-20210508102044747"></p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210508101336.png" alt="image-20210508101335927"></p>
<h2 id="Contents"><a href="#Contents" class="headerlink" title="Contents"></a>Contents</h2><ul>
<li><p>Differentially expressed proteins and representation</p>
</li>
<li><p>Gene Ontology based Functional annotation by Blast2GO</p>
</li>
<li><p>KEGG based Pathway annotation by KOALA</p>
</li>
<li><p>IntAct based Interaction networks by CytoScape</p>
</li>
</ul>
<h2 id="DEPs-amp-REPRESENTATION"><a href="#DEPs-amp-REPRESENTATION" class="headerlink" title="DEPs &amp; REPRESENTATION"></a>DEPs &amp; REPRESENTATION</h2><ul>
<li><p>DEPS</p>
</li>
<li><p>VOLCANO PLOT</p>
</li>
<li><p>HIERARCHICAL CLUSTERING</p>
</li>
</ul>
<h3 id="Terminalogies"><a href="#Terminalogies" class="headerlink" title="Terminalogies"></a>Terminalogies</h3><p><code>Fold change</code></p>
<ul>
<li><p>A measure describing how much a quantity changes between groups</p>
</li>
<li><p>Fold change (Treatment/Control) = Average (Treatment) / Average (Control)</p>
</li>
</ul>
<p><code>Significance level</code></p>
<ul>
<li><p>Student’s <em>t</em>-test</p>
</li>
<li><p>One-way Analysis of Variance (ANOVA)</p>
</li>
<li><p>False Discovery Rate (FDR)</p>
</li>
</ul>
<h3 id="DEPs-steps"><a href="#DEPs-steps" class="headerlink" title="DEPs steps"></a>DEPs steps</h3><ol>
<li>Download BIO316_L4.xlsx on LMO. This is a a real set of data from a LC-MS/MS based proteomic experiment, pre-processed by Proteome DiscovererTM, with quantitative profile for each of the identified proteins.</li>
</ol>
<blockquote>
<p>A = Control Group, B = Treatment Group, Organism = Mouse</p>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210501132800.png" alt="image-20210501132800307"></p>
<p>The explanation of each column and its function</p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210506204024.png" alt="image-20210506204024196"></p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210506204115.png" alt="image-20210506204115484"></p>
<ol start="2">
<li>Calculate the <strong>Fold Change</strong> for each protein.</li>
</ol>
<ul>
<li><p>Fold Change (B vs A) = Average(B) / Average(A)</p>
</li>
<li><p>In this case, for the first protein Q6IME9, use the Excel imbedded function:</p>
</li>
</ul>
<p><code>=AVERAGE(N2:P2)/AVERAGE(K2:M2)</code></p>
<ol start="3">
<li>Evaluate the significance (<strong>P-value</strong>) of difference between group A and B by student’s <em>t</em>-test.</li>
</ol>
<ul>
<li><p>Assumingly the expression intensities of each protein follow a normal distribution</p>
</li>
<li><p>Assumingly the variances of the two groups are equal</p>
</li>
<li><p>In this case, for the first protein Q6IME9, use the Excel imbedded function:</p>
</li>
<li><p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210501132911.png" alt="image-20210501132911381"></p>
</li>
</ul>
<blockquote>
<p> <strong>Student’s t-test:</strong></p>
<p> determine if the means of two sets of data are significantly different from each other.</p>
</blockquote>
<blockquote>
<p><strong>ANOVA</strong>:</p>
<p>provides a statistical test of whether the <strong>population means of several groups are equal</strong>, and therefore generalizes the <em>t</em>-test to <strong>more than two groups</strong>. ANOVA is useful for comparing three or more group means for statistical significance.</p>
</blockquote>
<blockquote>
<p>FDR (<strong>false discovery rate</strong>):</p>
<p>a method of conceptualizing the rate of type I errors in null hypothesis testing when conducting multiple comparisons</p>
<ul>
<li>FDR = Q-value = corrected p-value</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210508104637.png" alt="image-20210508104637032"></p>
</blockquote>
<blockquote>
<p>p-value：衡量一次检验假阳性率的指标（False positive rate） ；<br> q value：衡量错误发现率的指标（False discovery rate，简称FDR，所有检验中假阳性的概率）。即使用Q value的这个参 数预估FDR。Q value 需要利用公式从p value 校正计算后得到，所以Q value 通常又被称为adjusted p value。所以一般情况下：我们可以认为Q value = FDR = adjusted p value，即三者是一个东西，虽然有些定义上的细微区别，但是问题也不大。</p>
<p>作者：caokai001<br>链接：<a class="link"   target="_blank" rel="noopener" href="https://www.jianshu.com/p/949626b18e69" >https://www.jianshu.com/p/949626b18e69<i class="fas fa-external-link-alt"></i></a><br>来源：简书<br>著作权归作者所有。商业转载请联系作者获得授权，非商业转载请注明出处。</p>
</blockquote>
<ol start="4">
<li>Derivate the differentially expressed proteins.</li>
</ol>
<ul>
<li><p>Fold Change &gt;= 1.2 (up-regulated) or &lt;= 1/1.2 (0.833333, down-regulated)</p>
</li>
<li><p>P-value &lt; 0.05</p>
</li>
<li><p>This results in 275 proteins up-regulated and 146 down-regulated in Group B relative to A.</p>
</li>
</ul>
<blockquote>
<p>‼️There is no golden standard for Fold Change setting. Usually, for iTRAQ, it is ideal to set the Fold Change cut-off to be above 1.5 folds (e.g. 1.5 folds, 2.0 folds). 1.2 folds is the minimum threshold recognized by peers.</p>
<p>‼️Ideally, the t-test P-value should be adjusted by Benjamini-Hochberg method. But only very few DEPs will be derived with FDR &lt; 0.05. </p>
</blockquote>
<h3 id="Tips"><a href="#Tips" class="headerlink" title="Tips:"></a>Tips:</h3><ul>
<li>DEP mainly calculates two parameters: fold change &amp; p values</li>
<li>Derive the differentially expressed proteins based on the fold change &amp; p-values obtained in previous steps</li>
<li>Determine and label the gene name in either up-regulated or down-regulated</li>
<li>Fold Change &gt;= 1.2 (up-regulated) or &lt;= 1/1.2 (0.833333, down-regulated) P-value &lt; 0.05</li>
</ul>
<h3 id="Select-significant-genes-gene-expression-profiling-experiments"><a href="#Select-significant-genes-gene-expression-profiling-experiments" class="headerlink" title="Select significant genes ( gene expression profiling experiments)"></a>Select significant genes ( gene expression profiling experiments)</h3><p><code>common objective</code> :</p>
<p>identify those genes that exhibit differential expression (an increased or decreased steady state abundance) under a particular treatment relative to the expression levels of those genes in the control samples</p>
<ul>
<li>genes exhibiting a relative<code>fold change</code>greater than some arbitrary cutoff</li>
</ul>
<h3 id="GO-amp-KEGG"><a href="#GO-amp-KEGG" class="headerlink" title="GO&amp;KEGG"></a>GO&amp;KEGG</h3><p><code>KEGG</code> (<a class="link"   target="_blank" rel="noopener" href="https://www.kegg.jp/kegg/pathway.html" >https://www.kegg.jp/kegg/pathway.html<i class="fas fa-external-link-alt"></i></a>) is a **database resource **for understanding high-level functions and utilities of the biological system, such as the cell, the organism and the ecosystem, from molecular-level information, especially large-scale molecular datasets generated by genome sequencing and other high-throughput experimental technologies.</p>
<p><code>GO</code>:a major bioinformatics initiative to <code>unify</code> the representation of gene and gene product attributes across all species</p>
<ol>
<li>maintain and develop its controlled vocabulary of gene and gene product attributes</li>
<li>annotate genes and gene products, and assimilate and disseminate annotation data;</li>
<li>provide tools for easy access to all aspects of the data provided by the project</li>
<li>enable functional interpretation of experimental data using the GO</li>
</ol>
<p>Enrichment module gives you the answer of which pathways, diseases, and GO terms is statistically significant associated with the genes/proteins you just input.</p>
<h3 id="Volcano-Plot"><a href="#Volcano-Plot" class="headerlink" title="Volcano Plot"></a>Volcano Plot</h3><p>a volcano plot is a type of scatter-plot that is used to quickly ==identify changes in large data sets composed of <strong>replicate</strong> data==.</p>
<ul>
<li>plots <code>significance</code> versus <code>fold-change</code> on the y and x axes, respectively</li>
</ul>
<p>A volcano plot combines:</p>
<ul>
<li>a measure of statistical significance from a statistical test (e.g., a P-value from an ANOVA model)</li>
<li> the magnitude of the change, enabling quick visual identification of those data-points (genes, etc.) that display large magnitude changes that are also statistically significant. (Fold change)</li>
</ul>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210506132329.png" alt="image-20210506132329236"></p>
<p>values that display <code>large magnitude fold changes</code> (hence being left or right of center) as well as high statistical significance (hence being toward the top).</p>
<h3 id="Functional-annotation"><a href="#Functional-annotation" class="headerlink" title="Functional annotation"></a>Functional annotation</h3><p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210509114336.png" alt="image-20210509114336277"></p>
<h2 id="Proteomics-Data-anlysis"><a href="#Proteomics-Data-anlysis" class="headerlink" title="Proteomics Data anlysis"></a>Proteomics Data anlysis</h2><p><strong>Essay 1:</strong> Given a set of proteins, briefly describe the steps for conducting GO annotation by Blast2GO. (20 points) Interpret the significance and potential application of GO annotation. (20 points) Use your own words (a maximum of 500 words).</p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210506133508.png" alt="image-20210506133508471"></p>
<ol>
<li>To provide knowledge of roles in biological processes, molecular functions and cellular components for each target protein.</li>
<li>To provide knowledge of significantly altered biological processes, molecular functions and cellular components and associated target proteins.</li>
</ol>
<ul>
<li><p>­DAVID</p>
</li>
<li><p>­Blast2GO</p>
</li>
</ul>
<ol start="3">
<li><p>Similarity – at least 30%</p>
</li>
<li><p>GO evidence codes - to indicate how the annotation to a particular term is supported</p>
<ul>
<li>­experimental evidence</li>
<li>­phylogenetic evidence</li>
<li>­computational evidence</li>
<li>­author statements</li>
<li>­curatorial statements </li>
<li>­automatically generated annotations</li>
</ul>
</li>
</ol>
<p><strong>Essay 2:</strong> Given a set of proteins, briefly describe the steps for conducting KEGG based pathway annotation by KOALA. (20 points) Interpret the significance and potential application of KEGG annotation. (20 points) Use your own words (a maximum of 500 words).</p>
<p><code>pathway annotation</code></p>
<p><img src="https://cdn.jsdelivr.net/gh/yuxuanwu17/image-hosting/typora/20210506133602.png" alt="image-20210506133602204"></p>
<ol>
<li><p>To provide knowledge of roles in biological pathways for each target protein.</p>
</li>
<li><p>To provide knowledge of significantly altered biological pathways and associated target proteins.</p>
</li>
</ol>
<h2 id="Metabolomics-Data-Analysis"><a href="#Metabolomics-Data-Analysis" class="headerlink" title="Metabolomics Data Analysis"></a>Metabolomics Data Analysis</h2><p><strong>Problem Scenario</strong></p>
<p>The example used is compound concentration data obtained by targeted (i.e. quantitative) metabolic profiling of 1H NMR spectra of urine samples collected from 57 cancer patients. </p>
<p>There are two groups of patients </p>
<p>– <strong>Cachexia (Y)</strong> refers to the group with <code>significant skeletal muscle loss</code>; </p>
<p>– <strong>Cachexia (N)</strong> refers to the group with <code>no obvious</code> skeletal muscle <code>loss</code>. </p>
<p>Cachexia is defined as the loss of weight, muscle atrophy, fatigue, weakness and significant loss of appetite in someone who is not actively trying to lose weight. </p>
<p>Cachexia is often seen in end-stage cancer, and in that context is called “cancer cachexia”. </p>
<p>The exact mechanism behind cachexia is poorly understood, but there is probably a role for inflammatory cytokines, such as tumor necrosis factor-alpha (TNF-α) - which is also nicknamed cachexin, Interferon gamma (IFNγ), and Interleukin 6 (IL-6). </p>
<p>The goal here is, by using methods provided ==in MetaboAnalyst==, to <strong>identify metabolites</strong> that are significantly different between these two groups of cancer patients (cachexic vs. non-cachexic). <strong>These metabolites could serve as potential early-stage biomarkers for detecting cachexia and for exploring its underlying metabolic basis</strong>. The data file is ready on ICE.</p>
<hr>
<h3 id="Statistical-Analysis"><a href="#Statistical-Analysis" class="headerlink" title="Statistical Analysis"></a>Statistical Analysis</h3><p>A wide array of commonly used statistical and machine learning methods are available: </p>
<ul>
<li>univariate: <ul>
<li>fold change, </li>
<li>t-test,</li>
<li>volcano plot,</li>
<li>ANOVA, </li>
<li>correlation analysis;</li>
</ul>
</li>
<li>multivariate:<ul>
<li>principal component analysis (PCA), </li>
<li>partial least squares-discriminant analysis (PLS-DA) </li>
<li>orthogonal partial least squares-discriminant analysis (OPLS-DA)</li>
<li>high-dimensional feature selection - significance analysis of microarrays (and metabolites) (SAM) </li>
<li>empirical Bayesian analysis of microarrays (and metabolites) (EBAM)</li>
<li>clustering - dendrogram, heatmap, K-means, and self organizing map (SOM))</li>
<li>supervised classification - random forests and support vector machine (SVM).</li>
</ul>
</li>
</ul>
<hr>
<h3 id="Task-1-Data-processing-and-normalization-Lab-demo"><a href="#Task-1-Data-processing-and-normalization-Lab-demo" class="headerlink" title="Task 1: Data processing and normalization (Lab demo)"></a><strong>Task 1:</strong> Data processing and normalization (Lab demo)</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210514192847.png" alt="image-20210514192847714"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210514192918.png" alt="image-20210514192918723"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210514192954.png" alt="image-20210514192954791"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210514193154.png" alt="image-20210514193154720"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210514193222.png" alt="image-20210514193222520"></p>
<p><code>Row-wise normalization</code>: aims to normalize each sample (row) so that they are comparable to each other</p>
<p><code>column wise normalization</code> aims to make each feature much more comparable in magnitude to each other</p>
<h3 id="Task1-Data-processing-and-normalization-15-points"><a href="#Task1-Data-processing-and-normalization-15-points" class="headerlink" title="Task1 Data processing and normalization (15 points)."></a>Task1 Data processing and normalization (15 points).</h3><ol>
<li>Check the data format and upload the data to MataboAnalyst. Check the data integrity (this step will be launched automatically) and briefly describe your findings. if missing values are detected, replace the zero values with a small positive value (half of the minimum positive number detected in the data), otherwise, skip this step and proceed to the normalization step (5 points).</li>
</ol>
<blockquote>
<p>The data integrity check run automatically and the result is shown below. For lists of concentrations the data integrity check will assess the content (look for consistent formatting and the presence of two groups), determine whether the data is paired or determine if negative numbers exists. In this case, no missing values were identified in the data. Since zero values may cause some algorithms not to work properly, MetaboAnalyst will replace these values with a small positive value (the half of the minimum positive number detected in the data).</p>
</blockquote>
<p><code>notes</code></p>
<ol>
<li>Assess the content<ol>
<li> determine whether the data is paired </li>
<li> determine if negative numbers exists</li>
</ol>
</li>
</ol>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210514192705.png" alt="image-20210514192705637"></p>
<ol start="2">
<li>Normalize to a reference sample (refer to the lab notes for the general rule of reference sample selection) and choose “Log normalization” for data transformation. Compare the distribution of normalized and original concentrations (5 points). Briefly describe the outcome and the purpose of data normalization (5 points).</li>
</ol>
<blockquote>
<p>The distribution of normalized and original concentrations is shown below. A general rule of reference sample selection in normalization is to choose a sample in the control group with the fewest missing values. In this case, NETCR_005_V1 was used as the reference sample for row-wise normalization. </p>
<p>The internal data structure is transformed now to a table with each row representing a urine sample (from a patient) and each column representing a feature (a compound with a concentration). Row-wise normalization and column-wise normalization are often applied sequentially to reduce systematic variance and to improve the performance for downstream statistical analysis. Row-wise normalization aims to normalize each sample (row) so that that they are comparable to each other. In contrast to row-wise normalization, column-wise normalization aims to make each feature (column) more comparable in magnitude to each other.</p>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210514204227.png" alt="image-20210514204227302"></p>
<hr>
<h3 id="Task-2-Identification-of-significantly-different-metabolites-50-points"><a href="#Task-2-Identification-of-significantly-different-metabolites-50-points" class="headerlink" title="Task 2: Identification of significantly different metabolites (50 points)."></a><strong>Task 2:</strong> Identification of significantly different metabolites (50 points).</h3><ol>
<li>Use <strong>a Volcano Plot</strong> to <strong>compare the size of the fold change and statistical significance level of metabolites</strong> between the two sample groups. Use <strong>a fold change threshold 1.5 and adjusted P-value threshold 0.1.</strong> Save your results and briefly interpret your findings (10 points).</li>
</ol>
<h4 id="Volcano-plot"><a href="#Volcano-plot" class="headerlink" title="Volcano plot"></a>Volcano plot</h4><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210514204341.png" alt="image-20210514204341514"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210514204401.png" alt="image-20210514204401313"></p>
<blockquote>
<p> Volcano Plot is a commonly used univariate analysis method that is often first used to obtain an overview of the data or a rough ranking of potentially important features. Volcano plots are used to compare the size of the fold change to the statistical significance level. The X axis plots the fold change between the two groups (on a log scale), while the Y axis represents the p-value for a t-test of differences between samples (on a negative log scale). As a result, seven metabolites were identified as significantly different between the two groups by Volcano Plot analysis (see the table below). </p>
</blockquote>
<hr>
<h3 id="PLS-DA-lab-explanation"><a href="#PLS-DA-lab-explanation" class="headerlink" title="PLS-DA(lab explanation)"></a>PLS-DA(lab explanation)</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210515112747.png" alt="image-20210515112747153"></p>
<hr>
<ol start="2">
<li>Perform <code>Partial-Least Squares Discriminant Analysis (PLS-DA)</code> as a more sophisticated way of detecting more candidates. Save your results and interpret your findings (10 points). Compare the difference between the Volcano Plot method and <strong>PLS-DA (10 points).</strong></li>
</ol>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210514204524.png" alt="image-20210514204524882"></p>
<blockquote>
<p>In PLS-DA, Variable Importance in Projection or VIP score is <strong>a weighted sum of squares</strong> of the PLS loadings. </p>
<ul>
<li>The weights are based on the amount of explained Y-variance in each dimension.</li>
<li>VIP indicates the importance of the variable to the whole model.</li>
<li> In many studies VIP values &gt;2.0 are selected and used for further data analysis, but this cut-off depends on the number of variables used.</li>
<li> A more relaxed VIP cutoff of around 1.0 can be used if only a small number of variables (e.g. &lt;100) are involved in the study.</li>
<li>As a result, four metabolites, <strong>Acetone (VIP=2.5062), Creatine (VIP=2.3263), Succinate (VIP=2.2816) and Uracil (VIP=2.0946)</strong>, were identified as significantly different between the two groups by PLS-DA. </li>
</ul>
<p>Univariate </p>
<ul>
<li><p>Univariate analyses are often first used to obtain an overview of the data or a rough ranking of potentially important - features before applying more sophisticated data analysis tools. </p>
</li>
<li><p>Univariate analysis examines each variable separately without taking into account the effect of multiple comparisons.</p>
</li>
<li><p>As a supervised method, PLS-DA can perform both classification and feature selection. The algorithm uses cross-validation to select an optimal number of components for classification.</p>
</li>
</ul>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210514204723.png" alt="image-20210514204723166"></p>
<hr>
<h3 id="SAM-lab-explanation"><a href="#SAM-lab-explanation" class="headerlink" title="SAM (lab explanation)"></a>SAM (lab explanation)</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210515113331.png" alt="image-20210515113331667"></p>
<hr>
<ol start="3">
<li>Try another approach, Significance Analysis of Microarray (SAM), to select significant features that distinguish between cachexic and non-cachexic patients (10 points). Briefly describe how SAM works (10 points).</li>
</ol>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210514204800.png" alt="image-20210514204800554"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210514205002.png" alt="image-20210514205002317"></p>
<blockquote>
<p>SAM is designed to address False Discovery Rate (FDR) problems when running multiple tests on high-dimensional data. It first assigns a significance score to each variable based on its change relative to the standard deviation of repeated measurements. Then it chooses variables with scores greater than an adjustable threshold and compares their relative difference to the distribution estimated by random permutations of the class labels. For each threshold, a certain proportion of the variables in the permutation set will be found to be significant by chance. This number is used to calculate the FDR.</p>
</blockquote>
<h3 id="MetaboAnalyst-tips"><a href="#MetaboAnalyst-tips" class="headerlink" title="MetaboAnalyst tips"></a><strong>MetaboAnalyst</strong> tips</h3><p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210515113725.png" alt="image-20210515113725922"></p>
<p><strong>Task 3:</strong> Consensus results and functional analysis (15 points).</p>
<ol>
<li>Find metabolite candidates consistently identified by all the three approaches and interpret the results (10 points).</li>
</ol>
<blockquote>
<p>As shown in the Venn diagram, 3 metabolites were consistently identified by all the three approaches including <strong>Uracil, Acetone</strong> and <strong>Succinate.</strong> </p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210515122115.png" alt="image-20210515122115397"></p>
</blockquote>
<ol start="2">
<li>For the consensus results, identify their biological function(s) and possible roles in pathways using MetaboAnalyst Pathway Analysis (<a class="link"   target="_blank" rel="noopener" href="https://www.metaboanalyst.ca/faces/upload/PathUploadView.xhtml" >https://www.metaboanalyst.ca/faces/upload/PathUploadView.xhtml<i class="fas fa-external-link-alt"></i></a> (10 points).</li>
</ol>
<blockquote>
<p>By searching Pathway Analysis for Uracil, Acetone and Succinate, cross-links to HMDB, KEGG and PubChem are shown below.</p>
</blockquote>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210515122157.png" alt="image-20210515122157093"></p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210515122213.png" alt="img">Take <strong>Uracil</strong> as example, by checking the information from HMDB (Uracil HMDB ID: 0000300), we can find that Uracil serves as allosteric regulator and coenzyme for many important biochemical reactions. It is also involved in the biosynthesis of polysaccharides and the transportation of sugars containing aldehydes. Uracil is found to be associated with argininemia and carbamoyl phosphate synthetase deficiency, which are inborn errors of metabolism.</p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210515122255.png" alt="image-20210515122255505"></p>
<p>Besides, Uracil plays roles in Pyrimidine metabolism, beta-Alanine metabolism and Pantothenate and CoA biosynthesis processes. </p>
<p><img src="https://cdn.jsdelivr.net/gh/imgstore/typora/20210515122305.png" alt="image-20210515122305756"></p>

        </div>

        
            <div class="post-copyright-info">
                <div class="article-copyright-info-container">
    <ul>
        <li>Post title：PROTEOMICS DATA ANALYSIS</li>
        <li>Post author：Yuxuan Wu</li>
        <li>Create time：2021-05-01 01:02:21</li>
        <li>
            Post link：yuxuanwu17.github.io2021/05/01/2021-05-01-PROTEOMICS-DATA-ANALYSIS/
        </li>
        <li>
            Copyright Notice：All articles in this blog are licensed under <a class="license" target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by-nc-sa/4.0/deed.zh">BY-NC-SA</a> unless stating additionally.
        </li>
    </ul>
</div>

            </div>
        

        
            <div class="article-nav">
                
                    <div class="article-prev">
                        <a class="prev"
                           rel="prev"
                           href="/2021/05/02/2021-05-02-%E6%B3%A8%E8%A7%A3%E5%92%8C%E5%8F%8D%E5%B0%84/"
                        >
                            <span class="left arrow-icon flex-center">
                              <i class="fas fa-chevron-left"></i>
                            </span>
                            <span class="title flex-center">
                                <span class="post-nav-title-item">注解和反射</span>
                                <span class="post-nav-item">Prev posts</span>
                            </span>
                        </a>
                    </div>
                
                
                    <div class="article-next">
                        <a class="next"
                           rel="next"
                           href="/2021/04/27/2021-04-27-%E7%BD%91%E7%BB%9C%E7%BC%96%E7%A8%8B/"
                        >
                            <span class="title flex-center">
                                <span class="post-nav-title-item">网络编程</span>
                                <span class="post-nav-item">Next posts</span>
                            </span>
                            <span class="right arrow-icon flex-center">
                              <i class="fas fa-chevron-right"></i>
                            </span>
                        </a>
                    </div>
                
            </div>
        

        
    </div>
</div>


                
            </div>

        </div>

        <div class="page-main-content-bottom">
            <footer class="footer">
    <div class="info-container">
        <div class="copyright-info info-item">
            &copy;
            
              <span>2020</span> -
            
            2022 <i class="fas fa-heart icon-animate"></i> <a href="/">Yuxuan Wu</a>
        </div>
        
            <script async  src="//busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
            <div class="website-count info-item">
                
                    <span id="busuanzi_container_site_uv">
                        Visitor Count <span id="busuanzi_value_site_uv"></span>&ensp;
                    </span>
                
                
                    <span id="busuanzi_container_site_pv">
                        Totalview <span id="busuanzi_value_site_pv"></span>
                    </span>
                
            </div>
        
        <div class="theme-info info-item">
            Powered by <a target="_blank" href="https://hexo.io">Hexo</a> | Theme <a class="theme-version" target="_blank" href="https://github.com/XPoet/hexo-theme-keep">Keep v3.3.1</a>
        </div>
    </div>
</footer>

        </div>
    </div>

    
        <div class="post-tools">
            <div class="post-tools-container">
    <ul class="tools-list">
        <!-- TOC aside toggle -->
        
            <li class="tools-item page-aside-toggle">
                <i class="fas fa-outdent"></i>
            </li>
        

        <!-- go comment -->
        
    </ul>
</div>

        </div>
    

    <div class="right-bottom-side-tools">
        <div class="side-tools-container">
    <ul class="side-tools-list">
        <li class="tools-item tool-font-adjust-plus flex-center">
            <i class="fas fa-search-plus"></i>
        </li>

        <li class="tools-item tool-font-adjust-minus flex-center">
            <i class="fas fa-search-minus"></i>
        </li>

        <li class="tools-item tool-expand-width flex-center">
            <i class="fas fa-arrows-alt-h"></i>
        </li>

        <li class="tools-item tool-dark-light-toggle flex-center">
            <i class="fas fa-moon"></i>
        </li>

        <!-- rss -->
        

        

        <li class="tools-item tool-scroll-to-bottom flex-center">
            <i class="fas fa-arrow-down"></i>
        </li>
    </ul>

    <ul class="exposed-tools-list">
        <li class="tools-item tool-toggle-show flex-center">
            <i class="fas fa-cog fa-spin"></i>
        </li>
        
            <li class="tools-item tool-scroll-to-top flex-center">
                <i class="arrow-up fas fa-arrow-up"></i>
                <span class="percent"></span>
            </li>
        
    </ul>
</div>

    </div>

    
        <aside class="page-aside">
            <div class="post-toc-wrap">
    <div class="post-toc">
        <ol class="nav"><li class="nav-item nav-level-1"><a class="nav-link" href="#Workflow"><span class="nav-number">1.</span> <span class="nav-text">Workflow</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#Contents"><span class="nav-number">1.1.</span> <span class="nav-text">Contents</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#DEPs-amp-REPRESENTATION"><span class="nav-number">1.2.</span> <span class="nav-text">DEPs &amp; REPRESENTATION</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Terminalogies"><span class="nav-number">1.2.1.</span> <span class="nav-text">Terminalogies</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#DEPs-steps"><span class="nav-number">1.2.2.</span> <span class="nav-text">DEPs steps</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Tips"><span class="nav-number">1.2.3.</span> <span class="nav-text">Tips:</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Select-significant-genes-gene-expression-profiling-experiments"><span class="nav-number">1.2.4.</span> <span class="nav-text">Select significant genes ( gene expression profiling experiments)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#GO-amp-KEGG"><span class="nav-number">1.2.5.</span> <span class="nav-text">GO&amp;KEGG</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Volcano-Plot"><span class="nav-number">1.2.6.</span> <span class="nav-text">Volcano Plot</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Functional-annotation"><span class="nav-number">1.2.7.</span> <span class="nav-text">Functional annotation</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Proteomics-Data-anlysis"><span class="nav-number">1.3.</span> <span class="nav-text">Proteomics Data anlysis</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Metabolomics-Data-Analysis"><span class="nav-number">1.4.</span> <span class="nav-text">Metabolomics Data Analysis</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Statistical-Analysis"><span class="nav-number">1.4.1.</span> <span class="nav-text">Statistical Analysis</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Task-1-Data-processing-and-normalization-Lab-demo"><span class="nav-number">1.4.2.</span> <span class="nav-text">Task 1: Data processing and normalization (Lab demo)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Task1-Data-processing-and-normalization-15-points"><span class="nav-number">1.4.3.</span> <span class="nav-text">Task1 Data processing and normalization (15 points).</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Task-2-Identification-of-significantly-different-metabolites-50-points"><span class="nav-number">1.4.4.</span> <span class="nav-text">Task 2: Identification of significantly different metabolites (50 points).</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#Volcano-plot"><span class="nav-number">1.4.4.1.</span> <span class="nav-text">Volcano plot</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#PLS-DA-lab-explanation"><span class="nav-number">1.4.5.</span> <span class="nav-text">PLS-DA(lab explanation)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#SAM-lab-explanation"><span class="nav-number">1.4.6.</span> <span class="nav-text">SAM (lab explanation)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#MetaboAnalyst-tips"><span class="nav-number">1.4.7.</span> <span class="nav-text">MetaboAnalyst tips</span></a></li></ol></li></ol></li></ol>
    </div>
</div>
        </aside>
    

    <div class="image-viewer-container">
    <img src="">
</div>


    
        <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
          <span class="search-input-field-pre">
            <i class="fas fa-keyboard"></i>
          </span>
            <div class="search-input-container">
                <input autocomplete="off"
                       autocorrect="off"
                       autocapitalize="off"
                       placeholder="Search..."
                       spellcheck="false"
                       type="search"
                       class="search-input"
                >
            </div>
            <span class="popup-btn-close">
                <i class="fas fa-times"></i>
            </span>
        </div>
        <div id="search-result">
            <div id="no-result">
                <i class="fas fa-spinner fa-pulse fa-5x fa-fw"></i>
            </div>
        </div>
    </div>
</div>

    

</main>




<script src="/js/utils.js"></script>

<script src="/js/main.js"></script>

<script src="/js/header-shrink.js"></script>

<script src="/js/back2top.js"></script>

<script src="/js/dark-light-toggle.js"></script>



    
<script src="/js/local-search.js"></script>




    
<script src="/js/code-copy.js"></script>





<div class="post-scripts">
    
        
<script src="/js/left-side-toggle.js"></script>

<script src="/js/libs/anime.min.js"></script>

<script src="/js/toc.js"></script>

    
</div>



</body>
</html>
